{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>C1</th>\n",
       "      <th>C2</th>\n",
       "      <th>C3</th>\n",
       "      <th>C4</th>\n",
       "      <th>C5</th>\n",
       "      <th>C6</th>\n",
       "      <th>C7</th>\n",
       "      <th>C8</th>\n",
       "      <th>C9</th>\n",
       "      <th>C10</th>\n",
       "      <th>...</th>\n",
       "      <th>NC17</th>\n",
       "      <th>NC18</th>\n",
       "      <th>NC19</th>\n",
       "      <th>NC20</th>\n",
       "      <th>NC21</th>\n",
       "      <th>NC22</th>\n",
       "      <th>NC23</th>\n",
       "      <th>NC24</th>\n",
       "      <th>NC25</th>\n",
       "      <th>NC26</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>#symbol</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>A1BG</th>\n",
       "      <td>0.49</td>\n",
       "      <td>0.29</td>\n",
       "      <td>0.26</td>\n",
       "      <td>0.45</td>\n",
       "      <td>0.17</td>\n",
       "      <td>0.21</td>\n",
       "      <td>0.49</td>\n",
       "      <td>0.12</td>\n",
       "      <td>0.51</td>\n",
       "      <td>0.10</td>\n",
       "      <td>...</td>\n",
       "      <td>0.37</td>\n",
       "      <td>0.33</td>\n",
       "      <td>0.25</td>\n",
       "      <td>0.20</td>\n",
       "      <td>0.40</td>\n",
       "      <td>0.30</td>\n",
       "      <td>0.70</td>\n",
       "      <td>0.75</td>\n",
       "      <td>2.80</td>\n",
       "      <td>0.22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>A1CF</th>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.01</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.01</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.01</td>\n",
       "      <td>0.00</td>\n",
       "      <td>...</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>A2M</th>\n",
       "      <td>0.21</td>\n",
       "      <td>0.14</td>\n",
       "      <td>0.03</td>\n",
       "      <td>0.09</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.08</td>\n",
       "      <td>0.23</td>\n",
       "      <td>0.08</td>\n",
       "      <td>0.88</td>\n",
       "      <td>0.13</td>\n",
       "      <td>...</td>\n",
       "      <td>0.07</td>\n",
       "      <td>0.06</td>\n",
       "      <td>0.11</td>\n",
       "      <td>0.01</td>\n",
       "      <td>0.04</td>\n",
       "      <td>0.02</td>\n",
       "      <td>0.02</td>\n",
       "      <td>0.27</td>\n",
       "      <td>0.04</td>\n",
       "      <td>0.28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>A2ML1</th>\n",
       "      <td>0.04</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.02</td>\n",
       "      <td>0.07</td>\n",
       "      <td>0.05</td>\n",
       "      <td>0.04</td>\n",
       "      <td>0.03</td>\n",
       "      <td>0.01</td>\n",
       "      <td>0.02</td>\n",
       "      <td>0.01</td>\n",
       "      <td>...</td>\n",
       "      <td>0.01</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.02</td>\n",
       "      <td>0.02</td>\n",
       "      <td>0.02</td>\n",
       "      <td>0.01</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>A3GALT2</th>\n",
       "      <td>0.07</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.07</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.07</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.79</td>\n",
       "      <td>0.15</td>\n",
       "      <td>...</td>\n",
       "      <td>0.18</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.04</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.06</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ZYG11A</th>\n",
       "      <td>0.06</td>\n",
       "      <td>0.14</td>\n",
       "      <td>0.28</td>\n",
       "      <td>0.27</td>\n",
       "      <td>0.07</td>\n",
       "      <td>0.20</td>\n",
       "      <td>0.19</td>\n",
       "      <td>0.03</td>\n",
       "      <td>0.33</td>\n",
       "      <td>0.15</td>\n",
       "      <td>...</td>\n",
       "      <td>0.10</td>\n",
       "      <td>0.06</td>\n",
       "      <td>0.06</td>\n",
       "      <td>0.10</td>\n",
       "      <td>0.06</td>\n",
       "      <td>0.24</td>\n",
       "      <td>0.12</td>\n",
       "      <td>0.14</td>\n",
       "      <td>0.33</td>\n",
       "      <td>0.10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ZYG11B</th>\n",
       "      <td>8.17</td>\n",
       "      <td>8.20</td>\n",
       "      <td>3.62</td>\n",
       "      <td>7.88</td>\n",
       "      <td>5.96</td>\n",
       "      <td>7.14</td>\n",
       "      <td>7.94</td>\n",
       "      <td>7.11</td>\n",
       "      <td>7.93</td>\n",
       "      <td>4.03</td>\n",
       "      <td>...</td>\n",
       "      <td>8.82</td>\n",
       "      <td>2.65</td>\n",
       "      <td>2.93</td>\n",
       "      <td>4.58</td>\n",
       "      <td>2.82</td>\n",
       "      <td>1.58</td>\n",
       "      <td>1.57</td>\n",
       "      <td>5.05</td>\n",
       "      <td>1.07</td>\n",
       "      <td>1.25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ZYX</th>\n",
       "      <td>363.01</td>\n",
       "      <td>399.80</td>\n",
       "      <td>430.35</td>\n",
       "      <td>209.25</td>\n",
       "      <td>272.91</td>\n",
       "      <td>246.58</td>\n",
       "      <td>186.24</td>\n",
       "      <td>328.67</td>\n",
       "      <td>366.72</td>\n",
       "      <td>411.58</td>\n",
       "      <td>...</td>\n",
       "      <td>475.35</td>\n",
       "      <td>573.54</td>\n",
       "      <td>706.85</td>\n",
       "      <td>712.03</td>\n",
       "      <td>492.63</td>\n",
       "      <td>447.14</td>\n",
       "      <td>748.55</td>\n",
       "      <td>437.62</td>\n",
       "      <td>369.66</td>\n",
       "      <td>577.13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ZZEF1</th>\n",
       "      <td>19.17</td>\n",
       "      <td>15.72</td>\n",
       "      <td>13.95</td>\n",
       "      <td>14.78</td>\n",
       "      <td>8.69</td>\n",
       "      <td>10.58</td>\n",
       "      <td>17.59</td>\n",
       "      <td>10.42</td>\n",
       "      <td>14.67</td>\n",
       "      <td>8.85</td>\n",
       "      <td>...</td>\n",
       "      <td>13.38</td>\n",
       "      <td>5.01</td>\n",
       "      <td>4.60</td>\n",
       "      <td>9.06</td>\n",
       "      <td>4.07</td>\n",
       "      <td>3.04</td>\n",
       "      <td>4.55</td>\n",
       "      <td>10.34</td>\n",
       "      <td>1.83</td>\n",
       "      <td>2.08</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ZZZ3</th>\n",
       "      <td>6.05</td>\n",
       "      <td>4.12</td>\n",
       "      <td>1.81</td>\n",
       "      <td>7.15</td>\n",
       "      <td>2.70</td>\n",
       "      <td>4.64</td>\n",
       "      <td>7.82</td>\n",
       "      <td>2.39</td>\n",
       "      <td>3.64</td>\n",
       "      <td>2.25</td>\n",
       "      <td>...</td>\n",
       "      <td>3.63</td>\n",
       "      <td>1.36</td>\n",
       "      <td>1.05</td>\n",
       "      <td>1.90</td>\n",
       "      <td>2.16</td>\n",
       "      <td>1.00</td>\n",
       "      <td>1.56</td>\n",
       "      <td>3.99</td>\n",
       "      <td>0.96</td>\n",
       "      <td>0.27</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>19472 rows × 126 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "             C1      C2      C3      C4      C5      C6      C7      C8  \\\n",
       "#symbol                                                                   \n",
       "A1BG       0.49    0.29    0.26    0.45    0.17    0.21    0.49    0.12   \n",
       "A1CF       0.00    0.00    0.00    0.01    0.00    0.00    0.01    0.00   \n",
       "A2M        0.21    0.14    0.03    0.09    0.00    0.08    0.23    0.08   \n",
       "A2ML1      0.04    0.00    0.02    0.07    0.05    0.04    0.03    0.01   \n",
       "A3GALT2    0.07    0.00    0.00    0.00    0.07    0.00    0.07    0.00   \n",
       "...         ...     ...     ...     ...     ...     ...     ...     ...   \n",
       "ZYG11A     0.06    0.14    0.28    0.27    0.07    0.20    0.19    0.03   \n",
       "ZYG11B     8.17    8.20    3.62    7.88    5.96    7.14    7.94    7.11   \n",
       "ZYX      363.01  399.80  430.35  209.25  272.91  246.58  186.24  328.67   \n",
       "ZZEF1     19.17   15.72   13.95   14.78    8.69   10.58   17.59   10.42   \n",
       "ZZZ3       6.05    4.12    1.81    7.15    2.70    4.64    7.82    2.39   \n",
       "\n",
       "             C9     C10  ...    NC17    NC18    NC19    NC20    NC21    NC22  \\\n",
       "#symbol                  ...                                                   \n",
       "A1BG       0.51    0.10  ...    0.37    0.33    0.25    0.20    0.40    0.30   \n",
       "A1CF       0.01    0.00  ...    0.00    0.00    0.00    0.00    0.00    0.00   \n",
       "A2M        0.88    0.13  ...    0.07    0.06    0.11    0.01    0.04    0.02   \n",
       "A2ML1      0.02    0.01  ...    0.01    0.00    0.00    0.02    0.02    0.02   \n",
       "A3GALT2    0.79    0.15  ...    0.18    0.00    0.04    0.00    0.00    0.00   \n",
       "...         ...     ...  ...     ...     ...     ...     ...     ...     ...   \n",
       "ZYG11A     0.33    0.15  ...    0.10    0.06    0.06    0.10    0.06    0.24   \n",
       "ZYG11B     7.93    4.03  ...    8.82    2.65    2.93    4.58    2.82    1.58   \n",
       "ZYX      366.72  411.58  ...  475.35  573.54  706.85  712.03  492.63  447.14   \n",
       "ZZEF1     14.67    8.85  ...   13.38    5.01    4.60    9.06    4.07    3.04   \n",
       "ZZZ3       3.64    2.25  ...    3.63    1.36    1.05    1.90    2.16    1.00   \n",
       "\n",
       "           NC23    NC24    NC25    NC26  \n",
       "#symbol                                  \n",
       "A1BG       0.70    0.75    2.80    0.22  \n",
       "A1CF       0.00    0.00    0.00    0.00  \n",
       "A2M        0.02    0.27    0.04    0.28  \n",
       "A2ML1      0.01    0.00    0.00    0.00  \n",
       "A3GALT2    0.00    0.06    0.00    0.00  \n",
       "...         ...     ...     ...     ...  \n",
       "ZYG11A     0.12    0.14    0.33    0.10  \n",
       "ZYG11B     1.57    5.05    1.07    1.25  \n",
       "ZYX      748.55  437.62  369.66  577.13  \n",
       "ZZEF1      4.55   10.34    1.83    2.08  \n",
       "ZZZ3       1.56    3.99    0.96    0.27  \n",
       "\n",
       "[19472 rows x 126 columns]"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import pandas as pd\n",
    "import numpy as np\n",
    "import tarfile, os\n",
    "from sklearn import preprocessing\n",
    "\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt\n",
    "\n",
    "path = \"C:\\\\Users\\\\Joana\\\\Desktop\\\\Uminho\\\\2_1\\\\SIB\\\\trab\\\\dataset\"\n",
    "\n",
    "os.chdir(path)\n",
    "\n",
    "rna_seq_tmp = pd.read_csv(\"GSE157103_genes.tpm.tsv\", sep='\\t', index_col = 0)\n",
    "\n",
    "rna_seq_tmp"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>C1</th>\n",
       "      <th>C2</th>\n",
       "      <th>C3</th>\n",
       "      <th>C4</th>\n",
       "      <th>C5</th>\n",
       "      <th>C6</th>\n",
       "      <th>C7</th>\n",
       "      <th>C8</th>\n",
       "      <th>C9</th>\n",
       "      <th>C10</th>\n",
       "      <th>...</th>\n",
       "      <th>NC17</th>\n",
       "      <th>NC18</th>\n",
       "      <th>NC19</th>\n",
       "      <th>NC20</th>\n",
       "      <th>NC21</th>\n",
       "      <th>NC22</th>\n",
       "      <th>NC23</th>\n",
       "      <th>NC24</th>\n",
       "      <th>NC25</th>\n",
       "      <th>NC26</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>#symbol</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>A1BG</th>\n",
       "      <td>29.01</td>\n",
       "      <td>11.04</td>\n",
       "      <td>15.54</td>\n",
       "      <td>23.33</td>\n",
       "      <td>10.19</td>\n",
       "      <td>13.22</td>\n",
       "      <td>26.56</td>\n",
       "      <td>6.88</td>\n",
       "      <td>27.64</td>\n",
       "      <td>8.13</td>\n",
       "      <td>...</td>\n",
       "      <td>24.08</td>\n",
       "      <td>33.15</td>\n",
       "      <td>25.97</td>\n",
       "      <td>14.76</td>\n",
       "      <td>28.75</td>\n",
       "      <td>32.32</td>\n",
       "      <td>63.20</td>\n",
       "      <td>51.0</td>\n",
       "      <td>235.4</td>\n",
       "      <td>25.04</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>A1CF</th>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>1.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>1.16</td>\n",
       "      <td>0.00</td>\n",
       "      <td>1.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>...</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>1.00</td>\n",
       "      <td>1.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>A2M</th>\n",
       "      <td>17.00</td>\n",
       "      <td>7.00</td>\n",
       "      <td>2.00</td>\n",
       "      <td>6.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>7.00</td>\n",
       "      <td>17.00</td>\n",
       "      <td>7.00</td>\n",
       "      <td>64.00</td>\n",
       "      <td>15.00</td>\n",
       "      <td>...</td>\n",
       "      <td>6.00</td>\n",
       "      <td>8.00</td>\n",
       "      <td>16.00</td>\n",
       "      <td>1.00</td>\n",
       "      <td>4.00</td>\n",
       "      <td>3.00</td>\n",
       "      <td>3.00</td>\n",
       "      <td>25.0</td>\n",
       "      <td>5.0</td>\n",
       "      <td>42.00</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>A2ML1</th>\n",
       "      <td>3.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>1.06</td>\n",
       "      <td>4.06</td>\n",
       "      <td>3.07</td>\n",
       "      <td>2.96</td>\n",
       "      <td>2.01</td>\n",
       "      <td>0.93</td>\n",
       "      <td>0.99</td>\n",
       "      <td>0.93</td>\n",
       "      <td>...</td>\n",
       "      <td>1.04</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>2.01</td>\n",
       "      <td>2.00</td>\n",
       "      <td>1.94</td>\n",
       "      <td>1.02</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>A3GALT2</th>\n",
       "      <td>1.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>1.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>1.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>11.00</td>\n",
       "      <td>3.00</td>\n",
       "      <td>...</td>\n",
       "      <td>3.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>1.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>A4GALT</th>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>...</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>2.00</td>\n",
       "      <td>3.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>A4GNT</th>\n",
       "      <td>1.00</td>\n",
       "      <td>1.00</td>\n",
       "      <td>2.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>1.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>5.00</td>\n",
       "      <td>...</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>1.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>AAAS</th>\n",
       "      <td>567.00</td>\n",
       "      <td>361.00</td>\n",
       "      <td>416.00</td>\n",
       "      <td>572.00</td>\n",
       "      <td>253.00</td>\n",
       "      <td>621.00</td>\n",
       "      <td>789.00</td>\n",
       "      <td>316.00</td>\n",
       "      <td>622.00</td>\n",
       "      <td>632.00</td>\n",
       "      <td>...</td>\n",
       "      <td>543.00</td>\n",
       "      <td>557.00</td>\n",
       "      <td>604.00</td>\n",
       "      <td>503.00</td>\n",
       "      <td>898.00</td>\n",
       "      <td>919.00</td>\n",
       "      <td>913.00</td>\n",
       "      <td>934.0</td>\n",
       "      <td>907.0</td>\n",
       "      <td>314.00</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>AACS</th>\n",
       "      <td>229.00</td>\n",
       "      <td>109.00</td>\n",
       "      <td>102.00</td>\n",
       "      <td>204.00</td>\n",
       "      <td>67.00</td>\n",
       "      <td>189.00</td>\n",
       "      <td>217.00</td>\n",
       "      <td>114.00</td>\n",
       "      <td>251.00</td>\n",
       "      <td>224.00</td>\n",
       "      <td>...</td>\n",
       "      <td>155.00</td>\n",
       "      <td>152.00</td>\n",
       "      <td>127.00</td>\n",
       "      <td>139.00</td>\n",
       "      <td>180.00</td>\n",
       "      <td>192.00</td>\n",
       "      <td>133.00</td>\n",
       "      <td>222.0</td>\n",
       "      <td>113.0</td>\n",
       "      <td>38.00</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>AADAC</th>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>...</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.00</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>10 rows × 126 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "             C1      C2      C3      C4      C5      C6      C7      C8  \\\n",
       "#symbol                                                                   \n",
       "A1BG      29.01   11.04   15.54   23.33   10.19   13.22   26.56    6.88   \n",
       "A1CF       0.00    0.00    0.00    1.00    0.00    0.00    1.16    0.00   \n",
       "A2M       17.00    7.00    2.00    6.00    0.00    7.00   17.00    7.00   \n",
       "A2ML1      3.00    0.00    1.06    4.06    3.07    2.96    2.01    0.93   \n",
       "A3GALT2    1.00    0.00    0.00    0.00    1.00    0.00    1.00    0.00   \n",
       "A4GALT     0.00    0.00    0.00    0.00    0.00    0.00    0.00    0.00   \n",
       "A4GNT      1.00    1.00    2.00    0.00    0.00    1.00    0.00    0.00   \n",
       "AAAS     567.00  361.00  416.00  572.00  253.00  621.00  789.00  316.00   \n",
       "AACS     229.00  109.00  102.00  204.00   67.00  189.00  217.00  114.00   \n",
       "AADAC      0.00    0.00    0.00    0.00    0.00    0.00    0.00    0.00   \n",
       "\n",
       "             C9     C10  ...    NC17    NC18    NC19    NC20    NC21    NC22  \\\n",
       "#symbol                  ...                                                   \n",
       "A1BG      27.64    8.13  ...   24.08   33.15   25.97   14.76   28.75   32.32   \n",
       "A1CF       1.00    0.00  ...    0.00    0.00    0.00    1.00    1.00    0.00   \n",
       "A2M       64.00   15.00  ...    6.00    8.00   16.00    1.00    4.00    3.00   \n",
       "A2ML1      0.99    0.93  ...    1.04    0.00    0.00    2.01    2.00    1.94   \n",
       "A3GALT2   11.00    3.00  ...    3.00    0.00    1.00    0.00    0.00    0.00   \n",
       "A4GALT     0.00    0.00  ...    0.00    0.00    2.00    3.00    0.00    0.00   \n",
       "A4GNT      0.00    5.00  ...    0.00    0.00    0.00    1.00    0.00    0.00   \n",
       "AAAS     622.00  632.00  ...  543.00  557.00  604.00  503.00  898.00  919.00   \n",
       "AACS     251.00  224.00  ...  155.00  152.00  127.00  139.00  180.00  192.00   \n",
       "AADAC      0.00    0.00  ...    0.00    0.00    0.00    0.00    0.00    0.00   \n",
       "\n",
       "           NC23   NC24   NC25    NC26  \n",
       "#symbol                                \n",
       "A1BG      63.20   51.0  235.4   25.04  \n",
       "A1CF       0.00    0.0    0.0    0.00  \n",
       "A2M        3.00   25.0    5.0   42.00  \n",
       "A2ML1      1.02    0.0    0.0    0.00  \n",
       "A3GALT2    0.00    1.0    0.0    0.00  \n",
       "A4GALT     0.00    0.0    0.0    0.00  \n",
       "A4GNT      0.00    0.0    0.0    0.00  \n",
       "AAAS     913.00  934.0  907.0  314.00  \n",
       "AACS     133.00  222.0  113.0   38.00  \n",
       "AADAC      0.00    0.0    0.0    0.00  \n",
       "\n",
       "[10 rows x 126 columns]"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "rna_seq_ec = pd.read_csv(\"GSE157103_genes.ec.tsv\", sep='\\t', index_col = 0)\n",
    "\n",
    "rna_seq_ec.head(10)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>^DATABASE = GeoMiame</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>!Database_name = Gene Expression Omnibus (GEO)</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Database_institute = NCBI NLM NIH</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Database_web_link = http://www.ncbi.nlm.nih.gov/geo</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Database_email = geo@ncbi.nlm.nih.gov</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>^SERIES = GSE157103</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_title = Large-scale Multi-omic Analysis of COVID-19 Severity</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_geo_accession = GSE157103</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_status = Public on Aug 29 2020</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_submission_date = Aug 28 2020</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_last_update_date = Nov 30 2020</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_pubmed_id = 33096026</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_summary = Using RNA-seq and high-resolution mass spectrometry we performed a comprehensive systems analysis on 128 plasma and leukocyte samples from hospitalized patients with or without COVID-19 (n=102 and 26 respectively) and with differing degrees of disease severity. We generated abundance measurements for over 17,000 transcripts, proteins, metabolites, and lipids and compiled them with clinical data into a curated relational database. This resource offers the unique opportunity to perform systems analysis and cross-ome correlations to both molecules and patient outcomes. In total 219 molecular features were mapped with high significance to COVID-19 status and severity, including those involved in processes such as complement system activation, dysregulated lipid transport, and B cell activation. In one example, we detected a trio of covarying molecules – citrate, plasmenyl-phosphatidylcholines, and gelsolin (GSN) – that offer both pathophysiological insight and potential novel therapeutic targets. Further, our data revealed in some cases, and supported in others, that several biological processes were dysregulated in COVID-19 patients including vessel damage, platelet activation and degranulation, blood coagulation, and acute phase response. For example, we observed that the coagulation-related protein, cellular fibronectin (cFN), was highly increased within COVID-19 patients and provide new evidence that the upregulated proteoform stems from endothelial cells, consistent with endothelial injury as a major activator of the coagulation cascade. The abundance of prothrombin, which is cleaved to form thrombin during clotting, was significantly reduced and correlated with severity and might help to explain the hyper coagulative environment of SARS-CoV-2 infection. From transcriptomic analysis of leukocytes, we concluded that COVID-19 patients with acute respiratory distress syndrome (ARDS) demonstrated a phenotype that overlapped with, but was distinct from, that found in patients with non-COVID-19-ARDS. To aid in the global efforts toward elucidation of disease pathophysiology and therapeutic development, we created a web-based tool with interactive visualizations allowing for easy navigation of this systems-level compendium of biomolecule abundance in relation to COVID-19 status and severity. Finally, we leveraged these multi-omic data to predict COVID-19 patient outcomes with machine learning, which highlighted the predictive power of these expansive molecular measurements beyond the standardized clinical estimate of 10-year survival Charlson score.</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_overall_design = 126 samples were analyzed in total with 100 COVID-19 patients and 26 non-COVID-19</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_type = Expression profiling by high throughput sequencing</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_contributor = Ariel,,Jaitovitch</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_contributor = Joseph,,Balnis</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_contributor = Scott,A,Swanson</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753021</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753022</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753023</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753024</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753025</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753026</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753027</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753028</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753029</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753030</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753031</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753032</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753033</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753034</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753035</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753036</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753037</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753038</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753039</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753040</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753041</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753042</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753043</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753044</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753045</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753046</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753047</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753048</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753049</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753050</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753051</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753052</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>!Series_sample_id = GSM4753053</th>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "Empty DataFrame\n",
       "Columns: []\n",
       "Index: [!Database_name = Gene Expression Omnibus (GEO), !Database_institute = NCBI NLM NIH, !Database_web_link = http://www.ncbi.nlm.nih.gov/geo, !Database_email = geo@ncbi.nlm.nih.gov, ^SERIES = GSE157103, !Series_title = Large-scale Multi-omic Analysis of COVID-19 Severity, !Series_geo_accession = GSE157103, !Series_status = Public on Aug 29 2020, !Series_submission_date = Aug 28 2020, !Series_last_update_date = Nov 30 2020, !Series_pubmed_id = 33096026, !Series_summary = Using RNA-seq and high-resolution mass spectrometry we performed a comprehensive systems analysis on 128 plasma and leukocyte samples from hospitalized patients with or without COVID-19 (n=102 and 26 respectively) and with differing degrees of disease severity. We generated abundance measurements for over 17,000 transcripts, proteins, metabolites, and lipids and compiled them with clinical data into a curated relational database. This resource offers the unique opportunity to perform systems analysis and cross-ome correlations to both molecules and patient outcomes. In total 219 molecular features were mapped with high significance to COVID-19 status and severity, including those involved in processes such as complement system activation, dysregulated lipid transport, and B cell activation. In one example, we detected a trio of covarying molecules – citrate, plasmenyl-phosphatidylcholines, and gelsolin (GSN) – that offer both pathophysiological insight and potential novel therapeutic targets. Further, our data revealed in some cases, and supported in others, that several biological processes were dysregulated in COVID-19 patients including vessel damage, platelet activation and degranulation, blood coagulation, and acute phase response. For example, we observed that the coagulation-related protein, cellular fibronectin (cFN), was highly increased within COVID-19 patients and provide new evidence that the upregulated proteoform stems from endothelial cells, consistent with endothelial injury as a major activator of the coagulation cascade. The abundance of prothrombin, which is cleaved to form thrombin during clotting, was significantly reduced and correlated with severity and might help to explain the hyper coagulative environment of SARS-CoV-2 infection. From transcriptomic analysis of leukocytes, we concluded that COVID-19 patients with acute respiratory distress syndrome (ARDS) demonstrated a phenotype that overlapped with, but was distinct from, that found in patients with non-COVID-19-ARDS. To aid in the global efforts toward elucidation of disease pathophysiology and therapeutic development, we created a web-based tool with interactive visualizations allowing for easy navigation of this systems-level compendium of biomolecule abundance in relation to COVID-19 status and severity. Finally, we leveraged these multi-omic data to predict COVID-19 patient outcomes with machine learning, which highlighted the predictive power of these expansive molecular measurements beyond the standardized clinical estimate of 10-year survival Charlson score., !Series_overall_design = 126 samples were analyzed in total with 100 COVID-19 patients and 26 non-COVID-19, !Series_type = Expression profiling by high throughput sequencing, !Series_contributor = Ariel,,Jaitovitch, !Series_contributor = Joseph,,Balnis, !Series_contributor = Scott,A,Swanson, !Series_sample_id = GSM4753021, !Series_sample_id = GSM4753022, !Series_sample_id = GSM4753023, !Series_sample_id = GSM4753024, !Series_sample_id = GSM4753025, !Series_sample_id = GSM4753026, !Series_sample_id = GSM4753027, !Series_sample_id = GSM4753028, !Series_sample_id = GSM4753029, !Series_sample_id = GSM4753030, !Series_sample_id = GSM4753031, !Series_sample_id = GSM4753032, !Series_sample_id = GSM4753033, !Series_sample_id = GSM4753034, !Series_sample_id = GSM4753035, !Series_sample_id = GSM4753036, !Series_sample_id = GSM4753037, !Series_sample_id = GSM4753038, !Series_sample_id = GSM4753039, !Series_sample_id = GSM4753040, !Series_sample_id = GSM4753041, !Series_sample_id = GSM4753042, !Series_sample_id = GSM4753043, !Series_sample_id = GSM4753044, !Series_sample_id = GSM4753045, !Series_sample_id = GSM4753046, !Series_sample_id = GSM4753047, !Series_sample_id = GSM4753048, !Series_sample_id = GSM4753049, !Series_sample_id = GSM4753050, !Series_sample_id = GSM4753051, !Series_sample_id = GSM4753052, !Series_sample_id = GSM4753053]"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "soft = pd.read_csv(\"GSE157103_family.soft\", sep='\\t', index_col = 0)\n",
    "\n",
    "\n",
    "\n",
    "soft.head(50)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Assay Type</th>\n",
       "      <th>associated_controlled-vocabulary_terms</th>\n",
       "      <th>AvgSpotLen</th>\n",
       "      <th>Bases</th>\n",
       "      <th>BioProject</th>\n",
       "      <th>BioSample</th>\n",
       "      <th>Bytes</th>\n",
       "      <th>Cell_type</th>\n",
       "      <th>Center Name</th>\n",
       "      <th>Charlson_score</th>\n",
       "      <th>...</th>\n",
       "      <th>SAMPLE_TYPE</th>\n",
       "      <th>source_name</th>\n",
       "      <th>SRA Study</th>\n",
       "      <th>tissue_of_origin</th>\n",
       "      <th>ventilator-free_days</th>\n",
       "      <th>DM</th>\n",
       "      <th>sex</th>\n",
       "      <th>fibrinogen</th>\n",
       "      <th>apacheii</th>\n",
       "      <th>sofa</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Run</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>SRR12544419</th>\n",
       "      <td>RNA-Seq</td>\n",
       "      <td>DOID:0080600 UBERON:0003101 NCIT:C70909 CL:000...</td>\n",
       "      <td>102</td>\n",
       "      <td>1922788332</td>\n",
       "      <td>PRJNA660067</td>\n",
       "      <td>SAMN15934957</td>\n",
       "      <td>574369113</td>\n",
       "      <td>leukocyte</td>\n",
       "      <td>GEO</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>primary cells</td>\n",
       "      <td>Leukocytes from whole blood</td>\n",
       "      <td>SRP279280</td>\n",
       "      <td>whole blood</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>male</td>\n",
       "      <td>513.0</td>\n",
       "      <td>15.0</td>\n",
       "      <td>8.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>SRR12544420</th>\n",
       "      <td>RNA-Seq</td>\n",
       "      <td>DOID:0080600 UBERON:0003101 NCIT:C70909 CL:000...</td>\n",
       "      <td>102</td>\n",
       "      <td>1893842262</td>\n",
       "      <td>PRJNA660067</td>\n",
       "      <td>SAMN15934957</td>\n",
       "      <td>566318430</td>\n",
       "      <td>leukocyte</td>\n",
       "      <td>GEO</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>primary cells</td>\n",
       "      <td>Leukocytes from whole blood</td>\n",
       "      <td>SRP279280</td>\n",
       "      <td>whole blood</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>male</td>\n",
       "      <td>513.0</td>\n",
       "      <td>15.0</td>\n",
       "      <td>8.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>SRR12544421</th>\n",
       "      <td>RNA-Seq</td>\n",
       "      <td>DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C1...</td>\n",
       "      <td>102</td>\n",
       "      <td>1118235588</td>\n",
       "      <td>PRJNA660067</td>\n",
       "      <td>SAMN15934956</td>\n",
       "      <td>333238403</td>\n",
       "      <td>leukocyte</td>\n",
       "      <td>GEO</td>\n",
       "      <td>2</td>\n",
       "      <td>...</td>\n",
       "      <td>primary cells</td>\n",
       "      <td>Leukocytes from whole blood</td>\n",
       "      <td>SRP279280</td>\n",
       "      <td>whole blood</td>\n",
       "      <td>28</td>\n",
       "      <td>0.0</td>\n",
       "      <td>male</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>SRR12544422</th>\n",
       "      <td>RNA-Seq</td>\n",
       "      <td>DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C1...</td>\n",
       "      <td>102</td>\n",
       "      <td>1099963716</td>\n",
       "      <td>PRJNA660067</td>\n",
       "      <td>SAMN15934956</td>\n",
       "      <td>328145216</td>\n",
       "      <td>leukocyte</td>\n",
       "      <td>GEO</td>\n",
       "      <td>2</td>\n",
       "      <td>...</td>\n",
       "      <td>primary cells</td>\n",
       "      <td>Leukocytes from whole blood</td>\n",
       "      <td>SRP279280</td>\n",
       "      <td>whole blood</td>\n",
       "      <td>28</td>\n",
       "      <td>0.0</td>\n",
       "      <td>male</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>SRR12544423</th>\n",
       "      <td>RNA-Seq</td>\n",
       "      <td>DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C1...</td>\n",
       "      <td>102</td>\n",
       "      <td>1695555486</td>\n",
       "      <td>PRJNA660067</td>\n",
       "      <td>SAMN15934955</td>\n",
       "      <td>500465346</td>\n",
       "      <td>leukocyte</td>\n",
       "      <td>GEO</td>\n",
       "      <td>2</td>\n",
       "      <td>...</td>\n",
       "      <td>primary cells</td>\n",
       "      <td>Leukocytes from whole blood</td>\n",
       "      <td>SRP279280</td>\n",
       "      <td>whole blood</td>\n",
       "      <td>28</td>\n",
       "      <td>1.0</td>\n",
       "      <td>male</td>\n",
       "      <td>513.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>SRR12544666</th>\n",
       "      <td>RNA-Seq</td>\n",
       "      <td>DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C1...</td>\n",
       "      <td>102</td>\n",
       "      <td>2068218504</td>\n",
       "      <td>PRJNA660067</td>\n",
       "      <td>SAMN15935032</td>\n",
       "      <td>607590011</td>\n",
       "      <td>leukocyte</td>\n",
       "      <td>GEO</td>\n",
       "      <td>4</td>\n",
       "      <td>...</td>\n",
       "      <td>primary cells</td>\n",
       "      <td>Leukocytes from whole blood</td>\n",
       "      <td>SRP279280</td>\n",
       "      <td>whole blood</td>\n",
       "      <td>28</td>\n",
       "      <td>0.0</td>\n",
       "      <td>male</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>SRR12544667</th>\n",
       "      <td>RNA-Seq</td>\n",
       "      <td>DOID:4 UBERON:0003100 CL:0000738 NCIT:C12529</td>\n",
       "      <td>102</td>\n",
       "      <td>1685916894</td>\n",
       "      <td>PRJNA660067</td>\n",
       "      <td>SAMN15935031</td>\n",
       "      <td>493773643</td>\n",
       "      <td>leukocyte</td>\n",
       "      <td>GEO</td>\n",
       "      <td>2</td>\n",
       "      <td>...</td>\n",
       "      <td>primary cells</td>\n",
       "      <td>Leukocytes from whole blood</td>\n",
       "      <td>SRP279280</td>\n",
       "      <td>whole blood</td>\n",
       "      <td>28</td>\n",
       "      <td>0.0</td>\n",
       "      <td>female</td>\n",
       "      <td>261.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>SRR12544668</th>\n",
       "      <td>RNA-Seq</td>\n",
       "      <td>DOID:4 UBERON:0003100 CL:0000738 NCIT:C12529</td>\n",
       "      <td>102</td>\n",
       "      <td>1658309472</td>\n",
       "      <td>PRJNA660067</td>\n",
       "      <td>SAMN15935031</td>\n",
       "      <td>486294982</td>\n",
       "      <td>leukocyte</td>\n",
       "      <td>GEO</td>\n",
       "      <td>2</td>\n",
       "      <td>...</td>\n",
       "      <td>primary cells</td>\n",
       "      <td>Leukocytes from whole blood</td>\n",
       "      <td>SRP279280</td>\n",
       "      <td>whole blood</td>\n",
       "      <td>28</td>\n",
       "      <td>0.0</td>\n",
       "      <td>female</td>\n",
       "      <td>261.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>SRR12544669</th>\n",
       "      <td>RNA-Seq</td>\n",
       "      <td>DOID:4 UBERON:0003101 NCIT:C53511 NCIT:C70909 ...</td>\n",
       "      <td>102</td>\n",
       "      <td>2055329172</td>\n",
       "      <td>PRJNA660067</td>\n",
       "      <td>SAMN15935030</td>\n",
       "      <td>601300170</td>\n",
       "      <td>leukocyte</td>\n",
       "      <td>GEO</td>\n",
       "      <td>6</td>\n",
       "      <td>...</td>\n",
       "      <td>primary cells</td>\n",
       "      <td>Leukocytes from whole blood</td>\n",
       "      <td>SRP279280</td>\n",
       "      <td>whole blood</td>\n",
       "      <td>18</td>\n",
       "      <td>0.0</td>\n",
       "      <td>male</td>\n",
       "      <td>267.0</td>\n",
       "      <td>15.0</td>\n",
       "      <td>12.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>SRR12544670</th>\n",
       "      <td>RNA-Seq</td>\n",
       "      <td>DOID:4 UBERON:0003101 NCIT:C53511 NCIT:C70909 ...</td>\n",
       "      <td>102</td>\n",
       "      <td>2025194088</td>\n",
       "      <td>PRJNA660067</td>\n",
       "      <td>SAMN15935030</td>\n",
       "      <td>593296562</td>\n",
       "      <td>leukocyte</td>\n",
       "      <td>GEO</td>\n",
       "      <td>6</td>\n",
       "      <td>...</td>\n",
       "      <td>primary cells</td>\n",
       "      <td>Leukocytes from whole blood</td>\n",
       "      <td>SRP279280</td>\n",
       "      <td>whole blood</td>\n",
       "      <td>18</td>\n",
       "      <td>0.0</td>\n",
       "      <td>male</td>\n",
       "      <td>267.0</td>\n",
       "      <td>15.0</td>\n",
       "      <td>12.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>252 rows × 37 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "            Assay Type             associated_controlled-vocabulary_terms  \\\n",
       "Run                                                                         \n",
       "SRR12544419    RNA-Seq  DOID:0080600 UBERON:0003101 NCIT:C70909 CL:000...   \n",
       "SRR12544420    RNA-Seq  DOID:0080600 UBERON:0003101 NCIT:C70909 CL:000...   \n",
       "SRR12544421    RNA-Seq  DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C1...   \n",
       "SRR12544422    RNA-Seq  DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C1...   \n",
       "SRR12544423    RNA-Seq  DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C1...   \n",
       "...                ...                                                ...   \n",
       "SRR12544666    RNA-Seq  DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C1...   \n",
       "SRR12544667    RNA-Seq       DOID:4 UBERON:0003100 CL:0000738 NCIT:C12529   \n",
       "SRR12544668    RNA-Seq       DOID:4 UBERON:0003100 CL:0000738 NCIT:C12529   \n",
       "SRR12544669    RNA-Seq  DOID:4 UBERON:0003101 NCIT:C53511 NCIT:C70909 ...   \n",
       "SRR12544670    RNA-Seq  DOID:4 UBERON:0003101 NCIT:C53511 NCIT:C70909 ...   \n",
       "\n",
       "             AvgSpotLen       Bases   BioProject     BioSample      Bytes  \\\n",
       "Run                                                                         \n",
       "SRR12544419         102  1922788332  PRJNA660067  SAMN15934957  574369113   \n",
       "SRR12544420         102  1893842262  PRJNA660067  SAMN15934957  566318430   \n",
       "SRR12544421         102  1118235588  PRJNA660067  SAMN15934956  333238403   \n",
       "SRR12544422         102  1099963716  PRJNA660067  SAMN15934956  328145216   \n",
       "SRR12544423         102  1695555486  PRJNA660067  SAMN15934955  500465346   \n",
       "...                 ...         ...          ...           ...        ...   \n",
       "SRR12544666         102  2068218504  PRJNA660067  SAMN15935032  607590011   \n",
       "SRR12544667         102  1685916894  PRJNA660067  SAMN15935031  493773643   \n",
       "SRR12544668         102  1658309472  PRJNA660067  SAMN15935031  486294982   \n",
       "SRR12544669         102  2055329172  PRJNA660067  SAMN15935030  601300170   \n",
       "SRR12544670         102  2025194088  PRJNA660067  SAMN15935030  593296562   \n",
       "\n",
       "             Cell_type Center Name  Charlson_score  ...    SAMPLE_TYPE  \\\n",
       "Run                                                 ...                  \n",
       "SRR12544419  leukocyte         GEO               0  ...  primary cells   \n",
       "SRR12544420  leukocyte         GEO               0  ...  primary cells   \n",
       "SRR12544421  leukocyte         GEO               2  ...  primary cells   \n",
       "SRR12544422  leukocyte         GEO               2  ...  primary cells   \n",
       "SRR12544423  leukocyte         GEO               2  ...  primary cells   \n",
       "...                ...         ...             ...  ...            ...   \n",
       "SRR12544666  leukocyte         GEO               4  ...  primary cells   \n",
       "SRR12544667  leukocyte         GEO               2  ...  primary cells   \n",
       "SRR12544668  leukocyte         GEO               2  ...  primary cells   \n",
       "SRR12544669  leukocyte         GEO               6  ...  primary cells   \n",
       "SRR12544670  leukocyte         GEO               6  ...  primary cells   \n",
       "\n",
       "                             source_name  SRA Study tissue_of_origin  \\\n",
       "Run                                                                    \n",
       "SRR12544419  Leukocytes from whole blood  SRP279280      whole blood   \n",
       "SRR12544420  Leukocytes from whole blood  SRP279280      whole blood   \n",
       "SRR12544421  Leukocytes from whole blood  SRP279280      whole blood   \n",
       "SRR12544422  Leukocytes from whole blood  SRP279280      whole blood   \n",
       "SRR12544423  Leukocytes from whole blood  SRP279280      whole blood   \n",
       "...                                  ...        ...              ...   \n",
       "SRR12544666  Leukocytes from whole blood  SRP279280      whole blood   \n",
       "SRR12544667  Leukocytes from whole blood  SRP279280      whole blood   \n",
       "SRR12544668  Leukocytes from whole blood  SRP279280      whole blood   \n",
       "SRR12544669  Leukocytes from whole blood  SRP279280      whole blood   \n",
       "SRR12544670  Leukocytes from whole blood  SRP279280      whole blood   \n",
       "\n",
       "            ventilator-free_days   DM     sex fibrinogen apacheii  sofa  \n",
       "Run                                                                      \n",
       "SRR12544419                    0  0.0    male      513.0     15.0   8.0  \n",
       "SRR12544420                    0  0.0    male      513.0     15.0   8.0  \n",
       "SRR12544421                   28  0.0    male        NaN      NaN   NaN  \n",
       "SRR12544422                   28  0.0    male        NaN      NaN   NaN  \n",
       "SRR12544423                   28  1.0    male      513.0      NaN   NaN  \n",
       "...                          ...  ...     ...        ...      ...   ...  \n",
       "SRR12544666                   28  0.0    male        NaN      NaN   NaN  \n",
       "SRR12544667                   28  0.0  female      261.0      NaN   NaN  \n",
       "SRR12544668                   28  0.0  female      261.0      NaN   NaN  \n",
       "SRR12544669                   18  0.0    male      267.0     15.0  12.0  \n",
       "SRR12544670                   18  0.0    male      267.0     15.0  12.0  \n",
       "\n",
       "[252 rows x 37 columns]"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "metadados = pd.read_csv(\"SraRunTable.txt\", sep=',', index_col = 0)\n",
    "metadados"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Requirement already satisfied: GEOparse in c:\\users\\joana\\anaconda3\\lib\\site-packages (2.0.2)\n",
      "Requirement already satisfied: requests>=2.21.0 in c:\\users\\joana\\anaconda3\\lib\\site-packages (from GEOparse) (2.22.0)\n",
      "Requirement already satisfied: numpy>=1.7 in c:\\users\\joana\\anaconda3\\lib\\site-packages (from GEOparse) (1.18.1)\n",
      "Requirement already satisfied: tqdm>=4.31.1 in c:\\users\\joana\\anaconda3\\lib\\site-packages (from GEOparse) (4.42.1)\n",
      "Requirement already satisfied: pandas>=0.17 in c:\\users\\joana\\anaconda3\\lib\\site-packages (from GEOparse) (1.0.1)\n",
      "Requirement already satisfied: chardet<3.1.0,>=3.0.2 in c:\\users\\joana\\anaconda3\\lib\\site-packages (from requests>=2.21.0->GEOparse) (3.0.4)\n",
      "Requirement already satisfied: idna<2.9,>=2.5 in c:\\users\\joana\\anaconda3\\lib\\site-packages (from requests>=2.21.0->GEOparse) (2.8)\n",
      "Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in c:\\users\\joana\\anaconda3\\lib\\site-packages (from requests>=2.21.0->GEOparse) (1.25.8)\n",
      "Requirement already satisfied: certifi>=2017.4.17 in c:\\users\\joana\\anaconda3\\lib\\site-packages (from requests>=2.21.0->GEOparse) (2019.11.28)\n",
      "Requirement already satisfied: python-dateutil>=2.6.1 in c:\\users\\joana\\anaconda3\\lib\\site-packages (from pandas>=0.17->GEOparse) (2.8.1)\n",
      "Requirement already satisfied: pytz>=2017.2 in c:\\users\\joana\\anaconda3\\lib\\site-packages (from pandas>=0.17->GEOparse) (2019.3)\n",
      "Requirement already satisfied: six>=1.5 in c:\\users\\joana\\anaconda3\\lib\\site-packages (from python-dateutil>=2.6.1->pandas>=0.17->GEOparse) (1.14.0)\n",
      "Note: you may need to restart the kernel to use updated packages.\n"
     ]
    }
   ],
   "source": [
    "pip install GEOparse"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import GEOparse\n",
    "import pandas as pd\n",
    "import pylab as pl\n",
    "import seaborn as sns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "09-Dec-2020 17:53:24 DEBUG utils - Directory ./ already exists. Skipping.\n",
      "09-Dec-2020 17:53:24 INFO GEOparse - File already exist: using local version.\n",
      "09-Dec-2020 17:53:24 INFO GEOparse - Parsing ./GSE157103_family.soft.gz: \n",
      "09-Dec-2020 17:53:24 DEBUG GEOparse - DATABASE: GeoMiame\n",
      "09-Dec-2020 17:53:24 DEBUG GEOparse - SERIES: GSE157103\n",
      "09-Dec-2020 17:53:24 DEBUG GEOparse - PLATFORM: GPL24676\n",
      "09-Dec-2020 17:53:24 DEBUG GEOparse - SAMPLE: GSM4753021\n",
      "09-Dec-2020 17:53:24 DEBUG GEOparse - SAMPLE: GSM4753022\n",
      "09-Dec-2020 17:53:24 DEBUG GEOparse - SAMPLE: GSM4753023\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753024\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753025\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753026\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753027\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753028\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753029\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753030\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753031\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753032\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753033\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753034\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753035\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753036\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753037\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753038\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753039\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753040\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753041\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753042\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753043\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753044\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753045\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753046\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753047\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753048\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753049\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753050\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753051\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753052\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753053\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753054\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753055\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753056\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753057\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753058\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753059\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753060\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753061\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753062\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753063\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753064\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753065\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753066\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753067\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753068\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753069\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753070\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753071\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753072\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753073\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753074\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753075\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753076\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753077\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753078\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753079\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753080\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753081\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753082\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753083\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753084\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753085\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753086\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753087\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753088\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753089\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753090\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753091\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753092\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753093\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753094\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753095\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753096\n",
      "09-Dec-2020 17:53:25 DEBUG GEOparse - SAMPLE: GSM4753097\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753098\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753099\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753100\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753101\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753102\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753103\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753104\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753105\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753106\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753107\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753108\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753109\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753110\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753111\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753112\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753113\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753114\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753115\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753116\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753117\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753118\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753119\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753120\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753121\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753122\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753123\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753124\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753125\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753126\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753127\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753128\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753129\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753130\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753131\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753132\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753133\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753134\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753135\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753136\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753137\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753138\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753139\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753140\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753141\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753142\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753143\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753144\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753145\n",
      "09-Dec-2020 17:53:26 DEBUG GEOparse - SAMPLE: GSM4753146\n"
     ]
    }
   ],
   "source": [
    "gse157103 = GEOparse.get_GEO(\"GSE157103\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'GPL24676': <d: GPL24676>}"
      ]
     },
     "execution_count": 40,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gse157103.gpls"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'GSM4753021': <SAMPLE: GSM4753021>,\n",
       " 'GSM4753022': <SAMPLE: GSM4753022>,\n",
       " 'GSM4753023': <SAMPLE: GSM4753023>,\n",
       " 'GSM4753024': <SAMPLE: GSM4753024>,\n",
       " 'GSM4753025': <SAMPLE: GSM4753025>,\n",
       " 'GSM4753026': <SAMPLE: GSM4753026>,\n",
       " 'GSM4753027': <SAMPLE: GSM4753027>,\n",
       " 'GSM4753028': <SAMPLE: GSM4753028>,\n",
       " 'GSM4753029': <SAMPLE: GSM4753029>,\n",
       " 'GSM4753030': <SAMPLE: GSM4753030>,\n",
       " 'GSM4753031': <SAMPLE: GSM4753031>,\n",
       " 'GSM4753032': <SAMPLE: GSM4753032>,\n",
       " 'GSM4753033': <SAMPLE: GSM4753033>,\n",
       " 'GSM4753034': <SAMPLE: GSM4753034>,\n",
       " 'GSM4753035': <SAMPLE: GSM4753035>,\n",
       " 'GSM4753036': <SAMPLE: GSM4753036>,\n",
       " 'GSM4753037': <SAMPLE: GSM4753037>,\n",
       " 'GSM4753038': <SAMPLE: GSM4753038>,\n",
       " 'GSM4753039': <SAMPLE: GSM4753039>,\n",
       " 'GSM4753040': <SAMPLE: GSM4753040>,\n",
       " 'GSM4753041': <SAMPLE: GSM4753041>,\n",
       " 'GSM4753042': <SAMPLE: GSM4753042>,\n",
       " 'GSM4753043': <SAMPLE: GSM4753043>,\n",
       " 'GSM4753044': <SAMPLE: GSM4753044>,\n",
       " 'GSM4753045': <SAMPLE: GSM4753045>,\n",
       " 'GSM4753046': <SAMPLE: GSM4753046>,\n",
       " 'GSM4753047': <SAMPLE: GSM4753047>,\n",
       " 'GSM4753048': <SAMPLE: GSM4753048>,\n",
       " 'GSM4753049': <SAMPLE: GSM4753049>,\n",
       " 'GSM4753050': <SAMPLE: GSM4753050>,\n",
       " 'GSM4753051': <SAMPLE: GSM4753051>,\n",
       " 'GSM4753052': <SAMPLE: GSM4753052>,\n",
       " 'GSM4753053': <SAMPLE: GSM4753053>,\n",
       " 'GSM4753054': <SAMPLE: GSM4753054>,\n",
       " 'GSM4753055': <SAMPLE: GSM4753055>,\n",
       " 'GSM4753056': <SAMPLE: GSM4753056>,\n",
       " 'GSM4753057': <SAMPLE: GSM4753057>,\n",
       " 'GSM4753058': <SAMPLE: GSM4753058>,\n",
       " 'GSM4753059': <SAMPLE: GSM4753059>,\n",
       " 'GSM4753060': <SAMPLE: GSM4753060>,\n",
       " 'GSM4753061': <SAMPLE: GSM4753061>,\n",
       " 'GSM4753062': <SAMPLE: GSM4753062>,\n",
       " 'GSM4753063': <SAMPLE: GSM4753063>,\n",
       " 'GSM4753064': <SAMPLE: GSM4753064>,\n",
       " 'GSM4753065': <SAMPLE: GSM4753065>,\n",
       " 'GSM4753066': <SAMPLE: GSM4753066>,\n",
       " 'GSM4753067': <SAMPLE: GSM4753067>,\n",
       " 'GSM4753068': <SAMPLE: GSM4753068>,\n",
       " 'GSM4753069': <SAMPLE: GSM4753069>,\n",
       " 'GSM4753070': <SAMPLE: GSM4753070>,\n",
       " 'GSM4753071': <SAMPLE: GSM4753071>,\n",
       " 'GSM4753072': <SAMPLE: GSM4753072>,\n",
       " 'GSM4753073': <SAMPLE: GSM4753073>,\n",
       " 'GSM4753074': <SAMPLE: GSM4753074>,\n",
       " 'GSM4753075': <SAMPLE: GSM4753075>,\n",
       " 'GSM4753076': <SAMPLE: GSM4753076>,\n",
       " 'GSM4753077': <SAMPLE: GSM4753077>,\n",
       " 'GSM4753078': <SAMPLE: GSM4753078>,\n",
       " 'GSM4753079': <SAMPLE: GSM4753079>,\n",
       " 'GSM4753080': <SAMPLE: GSM4753080>,\n",
       " 'GSM4753081': <SAMPLE: GSM4753081>,\n",
       " 'GSM4753082': <SAMPLE: GSM4753082>,\n",
       " 'GSM4753083': <SAMPLE: GSM4753083>,\n",
       " 'GSM4753084': <SAMPLE: GSM4753084>,\n",
       " 'GSM4753085': <SAMPLE: GSM4753085>,\n",
       " 'GSM4753086': <SAMPLE: GSM4753086>,\n",
       " 'GSM4753087': <SAMPLE: GSM4753087>,\n",
       " 'GSM4753088': <SAMPLE: GSM4753088>,\n",
       " 'GSM4753089': <SAMPLE: GSM4753089>,\n",
       " 'GSM4753090': <SAMPLE: GSM4753090>,\n",
       " 'GSM4753091': <SAMPLE: GSM4753091>,\n",
       " 'GSM4753092': <SAMPLE: GSM4753092>,\n",
       " 'GSM4753093': <SAMPLE: GSM4753093>,\n",
       " 'GSM4753094': <SAMPLE: GSM4753094>,\n",
       " 'GSM4753095': <SAMPLE: GSM4753095>,\n",
       " 'GSM4753096': <SAMPLE: GSM4753096>,\n",
       " 'GSM4753097': <SAMPLE: GSM4753097>,\n",
       " 'GSM4753098': <SAMPLE: GSM4753098>,\n",
       " 'GSM4753099': <SAMPLE: GSM4753099>,\n",
       " 'GSM4753100': <SAMPLE: GSM4753100>,\n",
       " 'GSM4753101': <SAMPLE: GSM4753101>,\n",
       " 'GSM4753102': <SAMPLE: GSM4753102>,\n",
       " 'GSM4753103': <SAMPLE: GSM4753103>,\n",
       " 'GSM4753104': <SAMPLE: GSM4753104>,\n",
       " 'GSM4753105': <SAMPLE: GSM4753105>,\n",
       " 'GSM4753106': <SAMPLE: GSM4753106>,\n",
       " 'GSM4753107': <SAMPLE: GSM4753107>,\n",
       " 'GSM4753108': <SAMPLE: GSM4753108>,\n",
       " 'GSM4753109': <SAMPLE: GSM4753109>,\n",
       " 'GSM4753110': <SAMPLE: GSM4753110>,\n",
       " 'GSM4753111': <SAMPLE: GSM4753111>,\n",
       " 'GSM4753112': <SAMPLE: GSM4753112>,\n",
       " 'GSM4753113': <SAMPLE: GSM4753113>,\n",
       " 'GSM4753114': <SAMPLE: GSM4753114>,\n",
       " 'GSM4753115': <SAMPLE: GSM4753115>,\n",
       " 'GSM4753116': <SAMPLE: GSM4753116>,\n",
       " 'GSM4753117': <SAMPLE: GSM4753117>,\n",
       " 'GSM4753118': <SAMPLE: GSM4753118>,\n",
       " 'GSM4753119': <SAMPLE: GSM4753119>,\n",
       " 'GSM4753120': <SAMPLE: GSM4753120>,\n",
       " 'GSM4753121': <SAMPLE: GSM4753121>,\n",
       " 'GSM4753122': <SAMPLE: GSM4753122>,\n",
       " 'GSM4753123': <SAMPLE: GSM4753123>,\n",
       " 'GSM4753124': <SAMPLE: GSM4753124>,\n",
       " 'GSM4753125': <SAMPLE: GSM4753125>,\n",
       " 'GSM4753126': <SAMPLE: GSM4753126>,\n",
       " 'GSM4753127': <SAMPLE: GSM4753127>,\n",
       " 'GSM4753128': <SAMPLE: GSM4753128>,\n",
       " 'GSM4753129': <SAMPLE: GSM4753129>,\n",
       " 'GSM4753130': <SAMPLE: GSM4753130>,\n",
       " 'GSM4753131': <SAMPLE: GSM4753131>,\n",
       " 'GSM4753132': <SAMPLE: GSM4753132>,\n",
       " 'GSM4753133': <SAMPLE: GSM4753133>,\n",
       " 'GSM4753134': <SAMPLE: GSM4753134>,\n",
       " 'GSM4753135': <SAMPLE: GSM4753135>,\n",
       " 'GSM4753136': <SAMPLE: GSM4753136>,\n",
       " 'GSM4753137': <SAMPLE: GSM4753137>,\n",
       " 'GSM4753138': <SAMPLE: GSM4753138>,\n",
       " 'GSM4753139': <SAMPLE: GSM4753139>,\n",
       " 'GSM4753140': <SAMPLE: GSM4753140>,\n",
       " 'GSM4753141': <SAMPLE: GSM4753141>,\n",
       " 'GSM4753142': <SAMPLE: GSM4753142>,\n",
       " 'GSM4753143': <SAMPLE: GSM4753143>,\n",
       " 'GSM4753144': <SAMPLE: GSM4753144>,\n",
       " 'GSM4753145': <SAMPLE: GSM4753145>,\n",
       " 'GSM4753146': <SAMPLE: GSM4753146>}"
      ]
     },
     "execution_count": 23,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gse157103.gsms"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<DATABASE: GeoMiame>"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gse157103.database"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'title': ['Large-scale Multi-omic Analysis of COVID-19 Severity'],\n",
       " 'geo_accession': ['GSE157103'],\n",
       " 'status': ['Public on Aug 29 2020'],\n",
       " 'submission_date': ['Aug 28 2020'],\n",
       " 'last_update_date': ['Nov 30 2020'],\n",
       " 'pubmed_id': ['33096026'],\n",
       " 'summary': ['Using RNA-seq and high-resolution mass spectrometry we performed a comprehensive systems analysis on 128 plasma and leukocyte samples from hospitalized patients with or without COVID-19 (n=102 and 26 respectively) and with differing degrees of disease severity. We generated abundance measurements for over 17,000 transcripts, proteins, metabolites, and lipids and compiled them with clinical data into a curated relational database. This resource offers the unique opportunity to perform systems analysis and cross-ome correlations to both molecules and patient outcomes. In total 219 molecular features were mapped with high significance to COVID-19 status and severity, including those involved in processes such as complement system activation, dysregulated lipid transport, and B cell activation. In one example, we detected a trio of covarying molecules â€“ citrate, plasmenyl-phosphatidylcholines, and gelsolin (GSN) â€“ that offer both pathophysiological insight and potential novel therapeutic targets. Further, our data revealed in some cases, and supported in others, that several biological processes were dysregulated in COVID-19 patients including vessel damage, platelet activation and degranulation, blood coagulation, and acute phase response. For example, we observed that the coagulation-related protein, cellular fibronectin (cFN), was highly increased within COVID-19 patients and provide new evidence that the upregulated proteoform stems from endothelial cells, consistent with endothelial injury as a major activator of the coagulation cascade. The abundance of prothrombin, which is cleaved to form thrombin during clotting, was significantly reduced and correlated with severity and might help to explain the hyper coagulative environment of SARS-CoV-2 infection. From transcriptomic analysis of leukocytes, we concluded that COVID-19 patients with acute respiratory distress syndrome (ARDS) demonstrated a phenotype that overlapped with, but was distinct from, that found in patients with non-COVID-19-ARDS. To aid in the global efforts toward elucidation of disease pathophysiology and therapeutic development, we created a web-based tool with interactive visualizations allowing for easy navigation of this systems-level compendium of biomolecule abundance in relation to COVID-19 status and severity. Finally, we leveraged these multi-omic data to predict COVID-19 patient outcomes with machine learning, which highlighted the predictive power of these expansive molecular measurements beyond the standardized clinical estimate of 10-year survival Charlson score.'],\n",
       " 'overall_design': ['126 samples were analyzed in total with 100 COVID-19 patients and 26 non-COVID-19'],\n",
       " 'type': ['Expression profiling by high throughput sequencing'],\n",
       " 'contributor': ['Ariel,,Jaitovitch', 'Joseph,,Balnis', 'Scott,A,Swanson'],\n",
       " 'sample_id': ['GSM4753021',\n",
       "  'GSM4753022',\n",
       "  'GSM4753023',\n",
       "  'GSM4753024',\n",
       "  'GSM4753025',\n",
       "  'GSM4753026',\n",
       "  'GSM4753027',\n",
       "  'GSM4753028',\n",
       "  'GSM4753029',\n",
       "  'GSM4753030',\n",
       "  'GSM4753031',\n",
       "  'GSM4753032',\n",
       "  'GSM4753033',\n",
       "  'GSM4753034',\n",
       "  'GSM4753035',\n",
       "  'GSM4753036',\n",
       "  'GSM4753037',\n",
       "  'GSM4753038',\n",
       "  'GSM4753039',\n",
       "  'GSM4753040',\n",
       "  'GSM4753041',\n",
       "  'GSM4753042',\n",
       "  'GSM4753043',\n",
       "  'GSM4753044',\n",
       "  'GSM4753045',\n",
       "  'GSM4753046',\n",
       "  'GSM4753047',\n",
       "  'GSM4753048',\n",
       "  'GSM4753049',\n",
       "  'GSM4753050',\n",
       "  'GSM4753051',\n",
       "  'GSM4753052',\n",
       "  'GSM4753053',\n",
       "  'GSM4753054',\n",
       "  'GSM4753055',\n",
       "  'GSM4753056',\n",
       "  'GSM4753057',\n",
       "  'GSM4753058',\n",
       "  'GSM4753059',\n",
       "  'GSM4753060',\n",
       "  'GSM4753061',\n",
       "  'GSM4753062',\n",
       "  'GSM4753063',\n",
       "  'GSM4753064',\n",
       "  'GSM4753065',\n",
       "  'GSM4753066',\n",
       "  'GSM4753067',\n",
       "  'GSM4753068',\n",
       "  'GSM4753069',\n",
       "  'GSM4753070',\n",
       "  'GSM4753071',\n",
       "  'GSM4753072',\n",
       "  'GSM4753073',\n",
       "  'GSM4753074',\n",
       "  'GSM4753075',\n",
       "  'GSM4753076',\n",
       "  'GSM4753077',\n",
       "  'GSM4753078',\n",
       "  'GSM4753079',\n",
       "  'GSM4753080',\n",
       "  'GSM4753081',\n",
       "  'GSM4753082',\n",
       "  'GSM4753083',\n",
       "  'GSM4753084',\n",
       "  'GSM4753085',\n",
       "  'GSM4753086',\n",
       "  'GSM4753087',\n",
       "  'GSM4753088',\n",
       "  'GSM4753089',\n",
       "  'GSM4753090',\n",
       "  'GSM4753091',\n",
       "  'GSM4753092',\n",
       "  'GSM4753093',\n",
       "  'GSM4753094',\n",
       "  'GSM4753095',\n",
       "  'GSM4753096',\n",
       "  'GSM4753097',\n",
       "  'GSM4753098',\n",
       "  'GSM4753099',\n",
       "  'GSM4753100',\n",
       "  'GSM4753101',\n",
       "  'GSM4753102',\n",
       "  'GSM4753103',\n",
       "  'GSM4753104',\n",
       "  'GSM4753105',\n",
       "  'GSM4753106',\n",
       "  'GSM4753107',\n",
       "  'GSM4753108',\n",
       "  'GSM4753109',\n",
       "  'GSM4753110',\n",
       "  'GSM4753111',\n",
       "  'GSM4753112',\n",
       "  'GSM4753113',\n",
       "  'GSM4753114',\n",
       "  'GSM4753115',\n",
       "  'GSM4753116',\n",
       "  'GSM4753117',\n",
       "  'GSM4753118',\n",
       "  'GSM4753119',\n",
       "  'GSM4753120',\n",
       "  'GSM4753121',\n",
       "  'GSM4753122',\n",
       "  'GSM4753123',\n",
       "  'GSM4753124',\n",
       "  'GSM4753125',\n",
       "  'GSM4753126',\n",
       "  'GSM4753127',\n",
       "  'GSM4753128',\n",
       "  'GSM4753129',\n",
       "  'GSM4753130',\n",
       "  'GSM4753131',\n",
       "  'GSM4753132',\n",
       "  'GSM4753133',\n",
       "  'GSM4753134',\n",
       "  'GSM4753135',\n",
       "  'GSM4753136',\n",
       "  'GSM4753137',\n",
       "  'GSM4753138',\n",
       "  'GSM4753139',\n",
       "  'GSM4753140',\n",
       "  'GSM4753141',\n",
       "  'GSM4753142',\n",
       "  'GSM4753143',\n",
       "  'GSM4753144',\n",
       "  'GSM4753145',\n",
       "  'GSM4753146'],\n",
       " 'contact_name': ['Adolfo,Ariel,Jaitovich'],\n",
       " 'contact_email': ['jaitova@amc.edu'],\n",
       " 'contact_department': ['Molecular and Cellular Physiology'],\n",
       " 'contact_institute': ['Albany Medical Center'],\n",
       " 'contact_address': ['47 New Scotland Av'],\n",
       " 'contact_city': ['Albany'],\n",
       " 'contact_state': ['NY'],\n",
       " 'contact_zip/postal_code': ['12208'],\n",
       " 'contact_country': ['USA'],\n",
       " 'supplementary_file': ['ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE157nnn/GSE157103/suppl/GSE157103_genes.ec.tsv.gz',\n",
       "  'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE157nnn/GSE157103/suppl/GSE157103_genes.tpm.tsv.gz'],\n",
       " 'platform_id': ['GPL24676'],\n",
       " 'platform_taxid': ['9606'],\n",
       " 'sample_taxid': ['9606'],\n",
       " 'relation': ['BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA660067',\n",
       "  'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP279280']}"
      ]
     },
     "execution_count": 26,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gse157103.metadata"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>title</th>\n",
       "      <th>geo_accession</th>\n",
       "      <th>status</th>\n",
       "      <th>submission_date</th>\n",
       "      <th>last_update_date</th>\n",
       "      <th>type</th>\n",
       "      <th>channel_count</th>\n",
       "      <th>source_name_ch1</th>\n",
       "      <th>organism_ch1</th>\n",
       "      <th>taxid_ch1</th>\n",
       "      <th>...</th>\n",
       "      <th>relation</th>\n",
       "      <th>supplementary_file_1</th>\n",
       "      <th>series_id</th>\n",
       "      <th>data_row_count</th>\n",
       "      <th>characteristics_ch1.14.fibrinogen</th>\n",
       "      <th>characteristics_ch1.15.sofa</th>\n",
       "      <th>characteristics_ch1.16.associated controlled-vocabulary terms</th>\n",
       "      <th>characteristics_ch1.17.cell type</th>\n",
       "      <th>characteristics_ch1.18.tissue of origin</th>\n",
       "      <th>characteristics_ch1.19.sample type</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>GSM4753021</th>\n",
       "      <td>COVID_01_39y_male_NonICU</td>\n",
       "      <td>GSM4753021</td>\n",
       "      <td>Public on Aug 29 2020</td>\n",
       "      <td>Aug 28 2020</td>\n",
       "      <td>Aug 29 2020</td>\n",
       "      <td>SRA</td>\n",
       "      <td>1</td>\n",
       "      <td>Leukocytes from whole blood</td>\n",
       "      <td>Homo sapiens</td>\n",
       "      <td>9606</td>\n",
       "      <td>...</td>\n",
       "      <td>BioSample: https://www.ncbi.nlm.nih.gov/biosam...</td>\n",
       "      <td>NONE</td>\n",
       "      <td>GSE157103</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GSM4753022</th>\n",
       "      <td>COVID_02_63y_male_NonICU</td>\n",
       "      <td>GSM4753022</td>\n",
       "      <td>Public on Aug 29 2020</td>\n",
       "      <td>Aug 28 2020</td>\n",
       "      <td>Aug 29 2020</td>\n",
       "      <td>SRA</td>\n",
       "      <td>1</td>\n",
       "      <td>Leukocytes from whole blood</td>\n",
       "      <td>Homo sapiens</td>\n",
       "      <td>9606</td>\n",
       "      <td>...</td>\n",
       "      <td>BioSample: https://www.ncbi.nlm.nih.gov/biosam...</td>\n",
       "      <td>NONE</td>\n",
       "      <td>GSE157103</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GSM4753023</th>\n",
       "      <td>COVID_03_33y_male_NonICU</td>\n",
       "      <td>GSM4753023</td>\n",
       "      <td>Public on Aug 29 2020</td>\n",
       "      <td>Aug 28 2020</td>\n",
       "      <td>Aug 29 2020</td>\n",
       "      <td>SRA</td>\n",
       "      <td>1</td>\n",
       "      <td>Leukocytes from whole blood</td>\n",
       "      <td>Homo sapiens</td>\n",
       "      <td>9606</td>\n",
       "      <td>...</td>\n",
       "      <td>BioSample: https://www.ncbi.nlm.nih.gov/biosam...</td>\n",
       "      <td>NONE</td>\n",
       "      <td>GSE157103</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GSM4753024</th>\n",
       "      <td>COVID_04_49y_male_NonICU</td>\n",
       "      <td>GSM4753024</td>\n",
       "      <td>Public on Aug 29 2020</td>\n",
       "      <td>Aug 28 2020</td>\n",
       "      <td>Aug 29 2020</td>\n",
       "      <td>SRA</td>\n",
       "      <td>1</td>\n",
       "      <td>Leukocytes from whole blood</td>\n",
       "      <td>Homo sapiens</td>\n",
       "      <td>9606</td>\n",
       "      <td>...</td>\n",
       "      <td>BioSample: https://www.ncbi.nlm.nih.gov/biosam...</td>\n",
       "      <td>NONE</td>\n",
       "      <td>GSE157103</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GSM4753025</th>\n",
       "      <td>COVID_05_49y_male_NonICU</td>\n",
       "      <td>GSM4753025</td>\n",
       "      <td>Public on Aug 29 2020</td>\n",
       "      <td>Aug 28 2020</td>\n",
       "      <td>Aug 29 2020</td>\n",
       "      <td>SRA</td>\n",
       "      <td>1</td>\n",
       "      <td>Leukocytes from whole blood</td>\n",
       "      <td>Homo sapiens</td>\n",
       "      <td>9606</td>\n",
       "      <td>...</td>\n",
       "      <td>BioSample: https://www.ncbi.nlm.nih.gov/biosam...</td>\n",
       "      <td>NONE</td>\n",
       "      <td>GSE157103</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GSM4753142</th>\n",
       "      <td>NONCOVID_22_63y_male_NonICU</td>\n",
       "      <td>GSM4753142</td>\n",
       "      <td>Public on Aug 29 2020</td>\n",
       "      <td>Aug 28 2020</td>\n",
       "      <td>Aug 29 2020</td>\n",
       "      <td>SRA</td>\n",
       "      <td>1</td>\n",
       "      <td>Leukocytes from whole blood</td>\n",
       "      <td>Homo sapiens</td>\n",
       "      <td>9606</td>\n",
       "      <td>...</td>\n",
       "      <td>BioSample: https://www.ncbi.nlm.nih.gov/biosam...</td>\n",
       "      <td>NONE</td>\n",
       "      <td>GSE157103</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GSM4753143</th>\n",
       "      <td>NONCOVID_23_42y_female_NonICU</td>\n",
       "      <td>GSM4753143</td>\n",
       "      <td>Public on Aug 29 2020</td>\n",
       "      <td>Aug 28 2020</td>\n",
       "      <td>Aug 29 2020</td>\n",
       "      <td>SRA</td>\n",
       "      <td>1</td>\n",
       "      <td>Leukocytes from whole blood</td>\n",
       "      <td>Homo sapiens</td>\n",
       "      <td>9606</td>\n",
       "      <td>...</td>\n",
       "      <td>BioSample: https://www.ncbi.nlm.nih.gov/biosam...</td>\n",
       "      <td>NONE</td>\n",
       "      <td>GSE157103</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GSM4753144</th>\n",
       "      <td>NONCOVID_24_32y_female_NonICU</td>\n",
       "      <td>GSM4753144</td>\n",
       "      <td>Public on Aug 29 2020</td>\n",
       "      <td>Aug 28 2020</td>\n",
       "      <td>Aug 29 2020</td>\n",
       "      <td>SRA</td>\n",
       "      <td>1</td>\n",
       "      <td>Leukocytes from whole blood</td>\n",
       "      <td>Homo sapiens</td>\n",
       "      <td>9606</td>\n",
       "      <td>...</td>\n",
       "      <td>BioSample: https://www.ncbi.nlm.nih.gov/biosam...</td>\n",
       "      <td>NONE</td>\n",
       "      <td>GSE157103</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GSM4753145</th>\n",
       "      <td>NONCOVID_25_62y_male_NonICU</td>\n",
       "      <td>GSM4753145</td>\n",
       "      <td>Public on Aug 29 2020</td>\n",
       "      <td>Aug 28 2020</td>\n",
       "      <td>Aug 29 2020</td>\n",
       "      <td>SRA</td>\n",
       "      <td>1</td>\n",
       "      <td>Leukocytes from whole blood</td>\n",
       "      <td>Homo sapiens</td>\n",
       "      <td>9606</td>\n",
       "      <td>...</td>\n",
       "      <td>BioSample: https://www.ncbi.nlm.nih.gov/biosam...</td>\n",
       "      <td>NONE</td>\n",
       "      <td>GSE157103</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GSM4753146</th>\n",
       "      <td>NONCOVID_26_36y_male_ICU</td>\n",
       "      <td>GSM4753146</td>\n",
       "      <td>Public on Aug 29 2020</td>\n",
       "      <td>Aug 28 2020</td>\n",
       "      <td>Aug 29 2020</td>\n",
       "      <td>SRA</td>\n",
       "      <td>1</td>\n",
       "      <td>Leukocytes from whole blood</td>\n",
       "      <td>Homo sapiens</td>\n",
       "      <td>9606</td>\n",
       "      <td>...</td>\n",
       "      <td>BioSample: https://www.ncbi.nlm.nih.gov/biosam...</td>\n",
       "      <td>NONE</td>\n",
       "      <td>GSE157103</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>126 rows × 61 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                    title geo_accession  \\\n",
       "GSM4753021       COVID_01_39y_male_NonICU    GSM4753021   \n",
       "GSM4753022       COVID_02_63y_male_NonICU    GSM4753022   \n",
       "GSM4753023       COVID_03_33y_male_NonICU    GSM4753023   \n",
       "GSM4753024       COVID_04_49y_male_NonICU    GSM4753024   \n",
       "GSM4753025       COVID_05_49y_male_NonICU    GSM4753025   \n",
       "...                                   ...           ...   \n",
       "GSM4753142    NONCOVID_22_63y_male_NonICU    GSM4753142   \n",
       "GSM4753143  NONCOVID_23_42y_female_NonICU    GSM4753143   \n",
       "GSM4753144  NONCOVID_24_32y_female_NonICU    GSM4753144   \n",
       "GSM4753145    NONCOVID_25_62y_male_NonICU    GSM4753145   \n",
       "GSM4753146       NONCOVID_26_36y_male_ICU    GSM4753146   \n",
       "\n",
       "                           status submission_date last_update_date type  \\\n",
       "GSM4753021  Public on Aug 29 2020     Aug 28 2020      Aug 29 2020  SRA   \n",
       "GSM4753022  Public on Aug 29 2020     Aug 28 2020      Aug 29 2020  SRA   \n",
       "GSM4753023  Public on Aug 29 2020     Aug 28 2020      Aug 29 2020  SRA   \n",
       "GSM4753024  Public on Aug 29 2020     Aug 28 2020      Aug 29 2020  SRA   \n",
       "GSM4753025  Public on Aug 29 2020     Aug 28 2020      Aug 29 2020  SRA   \n",
       "...                           ...             ...              ...  ...   \n",
       "GSM4753142  Public on Aug 29 2020     Aug 28 2020      Aug 29 2020  SRA   \n",
       "GSM4753143  Public on Aug 29 2020     Aug 28 2020      Aug 29 2020  SRA   \n",
       "GSM4753144  Public on Aug 29 2020     Aug 28 2020      Aug 29 2020  SRA   \n",
       "GSM4753145  Public on Aug 29 2020     Aug 28 2020      Aug 29 2020  SRA   \n",
       "GSM4753146  Public on Aug 29 2020     Aug 28 2020      Aug 29 2020  SRA   \n",
       "\n",
       "           channel_count              source_name_ch1  organism_ch1 taxid_ch1  \\\n",
       "GSM4753021             1  Leukocytes from whole blood  Homo sapiens      9606   \n",
       "GSM4753022             1  Leukocytes from whole blood  Homo sapiens      9606   \n",
       "GSM4753023             1  Leukocytes from whole blood  Homo sapiens      9606   \n",
       "GSM4753024             1  Leukocytes from whole blood  Homo sapiens      9606   \n",
       "GSM4753025             1  Leukocytes from whole blood  Homo sapiens      9606   \n",
       "...                  ...                          ...           ...       ...   \n",
       "GSM4753142             1  Leukocytes from whole blood  Homo sapiens      9606   \n",
       "GSM4753143             1  Leukocytes from whole blood  Homo sapiens      9606   \n",
       "GSM4753144             1  Leukocytes from whole blood  Homo sapiens      9606   \n",
       "GSM4753145             1  Leukocytes from whole blood  Homo sapiens      9606   \n",
       "GSM4753146             1  Leukocytes from whole blood  Homo sapiens      9606   \n",
       "\n",
       "            ...                                           relation  \\\n",
       "GSM4753021  ...  BioSample: https://www.ncbi.nlm.nih.gov/biosam...   \n",
       "GSM4753022  ...  BioSample: https://www.ncbi.nlm.nih.gov/biosam...   \n",
       "GSM4753023  ...  BioSample: https://www.ncbi.nlm.nih.gov/biosam...   \n",
       "GSM4753024  ...  BioSample: https://www.ncbi.nlm.nih.gov/biosam...   \n",
       "GSM4753025  ...  BioSample: https://www.ncbi.nlm.nih.gov/biosam...   \n",
       "...         ...                                                ...   \n",
       "GSM4753142  ...  BioSample: https://www.ncbi.nlm.nih.gov/biosam...   \n",
       "GSM4753143  ...  BioSample: https://www.ncbi.nlm.nih.gov/biosam...   \n",
       "GSM4753144  ...  BioSample: https://www.ncbi.nlm.nih.gov/biosam...   \n",
       "GSM4753145  ...  BioSample: https://www.ncbi.nlm.nih.gov/biosam...   \n",
       "GSM4753146  ...  BioSample: https://www.ncbi.nlm.nih.gov/biosam...   \n",
       "\n",
       "           supplementary_file_1  series_id data_row_count  \\\n",
       "GSM4753021                 NONE  GSE157103              0   \n",
       "GSM4753022                 NONE  GSE157103              0   \n",
       "GSM4753023                 NONE  GSE157103              0   \n",
       "GSM4753024                 NONE  GSE157103              0   \n",
       "GSM4753025                 NONE  GSE157103              0   \n",
       "...                         ...        ...            ...   \n",
       "GSM4753142                 NONE  GSE157103              0   \n",
       "GSM4753143                 NONE  GSE157103              0   \n",
       "GSM4753144                 NONE  GSE157103              0   \n",
       "GSM4753145                 NONE  GSE157103              0   \n",
       "GSM4753146                 NONE  GSE157103              0   \n",
       "\n",
       "           characteristics_ch1.14.fibrinogen characteristics_ch1.15.sofa  \\\n",
       "GSM4753021                               NaN                         NaN   \n",
       "GSM4753022                               NaN                         NaN   \n",
       "GSM4753023                               NaN                         NaN   \n",
       "GSM4753024                               NaN                         NaN   \n",
       "GSM4753025                               NaN                         NaN   \n",
       "...                                      ...                         ...   \n",
       "GSM4753142                               NaN                         NaN   \n",
       "GSM4753143                               NaN                         NaN   \n",
       "GSM4753144                               NaN                         NaN   \n",
       "GSM4753145                               NaN                         NaN   \n",
       "GSM4753146                               NaN                         NaN   \n",
       "\n",
       "           characteristics_ch1.16.associated controlled-vocabulary terms  \\\n",
       "GSM4753021                                                NaN              \n",
       "GSM4753022                                                NaN              \n",
       "GSM4753023                                                NaN              \n",
       "GSM4753024                                                NaN              \n",
       "GSM4753025                                                NaN              \n",
       "...                                                       ...              \n",
       "GSM4753142                                                NaN              \n",
       "GSM4753143                                                NaN              \n",
       "GSM4753144                                                NaN              \n",
       "GSM4753145                                                NaN              \n",
       "GSM4753146                                                NaN              \n",
       "\n",
       "           characteristics_ch1.17.cell type  \\\n",
       "GSM4753021                              NaN   \n",
       "GSM4753022                              NaN   \n",
       "GSM4753023                              NaN   \n",
       "GSM4753024                              NaN   \n",
       "GSM4753025                              NaN   \n",
       "...                                     ...   \n",
       "GSM4753142                              NaN   \n",
       "GSM4753143                              NaN   \n",
       "GSM4753144                              NaN   \n",
       "GSM4753145                              NaN   \n",
       "GSM4753146                              NaN   \n",
       "\n",
       "           characteristics_ch1.18.tissue of origin  \\\n",
       "GSM4753021                                     NaN   \n",
       "GSM4753022                                     NaN   \n",
       "GSM4753023                                     NaN   \n",
       "GSM4753024                                     NaN   \n",
       "GSM4753025                                     NaN   \n",
       "...                                            ...   \n",
       "GSM4753142                                     NaN   \n",
       "GSM4753143                                     NaN   \n",
       "GSM4753144                                     NaN   \n",
       "GSM4753145                                     NaN   \n",
       "GSM4753146                                     NaN   \n",
       "\n",
       "           characteristics_ch1.19.sample type  \n",
       "GSM4753021                                NaN  \n",
       "GSM4753022                                NaN  \n",
       "GSM4753023                                NaN  \n",
       "GSM4753024                                NaN  \n",
       "GSM4753025                                NaN  \n",
       "...                                       ...  \n",
       "GSM4753142                                NaN  \n",
       "GSM4753143                                NaN  \n",
       "GSM4753144                                NaN  \n",
       "GSM4753145                                NaN  \n",
       "GSM4753146                                NaN  \n",
       "\n",
       "[126 rows x 61 columns]"
      ]
     },
     "execution_count": 27,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gse157103.phenotype_data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<bound method BaseGEO.get_accession of <SERIES: GSE157103 - 126 SAMPLES, 1 d(s)>>"
      ]
     },
     "execution_count": 32,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gse157103.get_accession"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>geo_accession</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>GSM4753021</th>\n",
       "      <td>GSM4753021</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GSM4753022</th>\n",
       "      <td>GSM4753022</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GSM4753023</th>\n",
       "      <td>GSM4753023</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GSM4753024</th>\n",
       "      <td>GSM4753024</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GSM4753025</th>\n",
       "      <td>GSM4753025</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GSM4753142</th>\n",
       "      <td>GSM4753142</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GSM4753143</th>\n",
       "      <td>GSM4753143</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GSM4753144</th>\n",
       "      <td>GSM4753144</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GSM4753145</th>\n",
       "      <td>GSM4753145</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GSM4753146</th>\n",
       "      <td>GSM4753146</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>126 rows × 1 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "           geo_accession\n",
       "GSM4753021    GSM4753021\n",
       "GSM4753022    GSM4753022\n",
       "GSM4753023    GSM4753023\n",
       "GSM4753024    GSM4753024\n",
       "GSM4753025    GSM4753025\n",
       "...                  ...\n",
       "GSM4753142    GSM4753142\n",
       "GSM4753143    GSM4753143\n",
       "GSM4753144    GSM4753144\n",
       "GSM4753145    GSM4753145\n",
       "GSM4753146    GSM4753146\n",
       "\n",
       "[126 rows x 1 columns]"
      ]
     },
     "execution_count": 37,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gse157103.phenotype_data[[\"geo_accession\"]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "18-Dec-2020 14:13:39 DEBUG utils - Directory ./ already exists. Skipping.\n",
      "18-Dec-2020 14:13:39 INFO GEOparse - File already exist: using local version.\n",
      "18-Dec-2020 14:13:39 INFO GEOparse - Parsing ./GSE157103_family.soft.gz: \n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - DATABASE: GeoMiame\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SERIES: GSE157103\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - PLATFORM: GPL24676\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753021\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753022\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753023\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753024\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753025\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753026\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753027\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753028\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753029\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753030\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753031\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753032\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753033\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753034\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753035\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753036\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753037\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753038\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753039\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753040\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753041\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753042\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753043\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753044\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753045\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753046\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753047\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753048\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753049\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753050\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753051\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753052\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753053\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753054\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753055\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753056\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753057\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753058\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753059\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753060\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753061\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753062\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753063\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753064\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753065\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753066\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753067\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753068\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753069\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753070\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753071\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753072\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753073\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753074\n",
      "18-Dec-2020 14:13:39 DEBUG GEOparse - SAMPLE: GSM4753075\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753076\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753077\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753078\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753079\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753080\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753081\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753082\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753083\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753084\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753085\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753086\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753087\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753088\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753089\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753090\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753091\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753092\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753093\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753094\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753095\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753096\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753097\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753098\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753099\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753100\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753101\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753102\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753103\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753104\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753105\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753106\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753107\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753108\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753109\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753110\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753111\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753112\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753113\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753114\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753115\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753116\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753117\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753118\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753119\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753120\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753121\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753122\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753123\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753124\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753125\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753126\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753127\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753128\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753129\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753130\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753131\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753132\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753133\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753134\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753135\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753136\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753137\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753138\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753139\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753140\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753141\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753142\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753143\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753144\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753145\n",
      "18-Dec-2020 14:13:40 DEBUG GEOparse - SAMPLE: GSM4753146\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "GSM example = uma amostra do dataset:\n"
     ]
    },
    {
     "ename": "NameError",
     "evalue": "name 'gse' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[1;32m<ipython-input-1-a7bb2533a42b>\u001b[0m in \u001b[0;36m<module>\u001b[1;34m\u001b[0m\n\u001b[0;32m      5\u001b[0m \u001b[0mprint\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m      6\u001b[0m \u001b[0mprint\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;34m\"GSM example = uma amostra do dataset:\"\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m----> 7\u001b[1;33m \u001b[1;32mfor\u001b[0m \u001b[0mi\u001b[0m \u001b[1;32min\u001b[0m \u001b[0mgse\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mgsms\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mitems\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m      8\u001b[0m     \u001b[0mprint\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;34m\"i\"\u001b[0m\u001b[1;33m,\u001b[0m\u001b[0mi\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m      9\u001b[0m \u001b[1;32mfor\u001b[0m \u001b[0mgsm_name\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mgsm\u001b[0m \u001b[1;32min\u001b[0m \u001b[0mgse\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mgsms\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mitems\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;31mNameError\u001b[0m: name 'gse' is not defined"
     ]
    }
   ],
   "source": [
    "import GEOparse\n",
    "\n",
    "gse157103 = GEOparse.get_GEO(geo=\"GSE157103\", destdir=\"./\")\n",
    "\n",
    "print()\n",
    "print(\"GSM example = uma amostra do dataset:\")\n",
    "for i in gse.gsms.items():\n",
    "    print(\"i\",i)\n",
    "for gsm_name, gsm in gse.gsms.items():\n",
    "    print(\"Name: \", gsm_name)\n",
    "    print(\"Metadata:\",)\n",
    "    for key, value in gsm.metadata.items():\n",
    "        print(\" - %s : %s\" % (key, \", \".join(value)))\n",
    "    print (\"Table data:\",)\n",
    "    print (gsm.table.head())\n",
    "    break\n",
    "\n",
    "print()\n",
    "print(\"GPL example = uma das plataformas do projecto:\")\n",
    "for gpl_name, gpl in gse.gpls.items():\n",
    "    print(\"Name: \", gpl_name)\n",
    "    print(\"Metadata:\",)\n",
    "    for key, value in gpl.metadata.items():\n",
    "        print(\"glp.metadata.items()\")\n",
    "        print(\" - %s : %s\" % (key, \", \".join(value)))\n",
    "    print(\"Table data:\",)\n",
    "    print(gpl.table.head())\n",
    "    break"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "13-Dec-2020 14:58:20 DEBUG utils - Directory ./ already exists. Skipping.\n",
      "13-Dec-2020 14:58:20 INFO GEOparse - File already exist: using local version.\n",
      "13-Dec-2020 14:58:20 INFO GEOparse - Parsing ./GSE157103_family.soft.gz: \n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - DATABASE: GeoMiame\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SERIES: GSE157103\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - PLATFORM: GPL24676\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753021\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753022\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753023\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753024\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753025\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753026\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753027\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753028\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753029\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753030\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753031\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753032\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753033\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753034\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753035\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753036\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753037\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753038\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753039\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753040\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753041\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753042\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753043\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753044\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753045\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753046\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753047\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753048\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753049\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753050\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753051\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753052\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753053\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753054\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753055\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753056\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753057\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753058\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753059\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753060\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753061\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753062\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753063\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753064\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753065\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753066\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753067\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753068\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753069\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753070\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753071\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753072\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753073\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753074\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753075\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753076\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753077\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753078\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753079\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753080\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753081\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753082\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753083\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753084\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753085\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753086\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753087\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753088\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753089\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753090\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753091\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753092\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753093\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753094\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753095\n",
      "13-Dec-2020 14:58:20 DEBUG GEOparse - SAMPLE: GSM4753096\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753097\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753098\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753099\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753100\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753101\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753102\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753103\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753104\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753105\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753106\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753107\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753108\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753109\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753110\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753111\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753112\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753113\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753114\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753115\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753116\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753117\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753118\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753119\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753120\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753121\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753122\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753123\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753124\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753125\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753126\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753127\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753128\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753129\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753130\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753131\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753132\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753133\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753134\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753135\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753136\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753137\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753138\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753139\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753140\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753141\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753142\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753143\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753144\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753145\n",
      "13-Dec-2020 14:58:21 DEBUG GEOparse - SAMPLE: GSM4753146\n"
     ]
    }
   ],
   "source": [
    "#check the GEO database for this accession ID\n",
    "#downloads it to specified directory (by default this is CWD)\n",
    "#The result will be loaded into GEOparse.GSE object\n",
    "import GEOparse\n",
    "gse = GEOparse.get_GEO(geo=\"GSE157103\", destdir=\"./\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "GSM4753021 -- <SAMPLE: GSM4753021>\n",
      "metadata: \n",
      " {'title': ['COVID_01_39y_male_NonICU'], 'geo_accession': ['GSM4753021'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 39', 'Sex: male', 'icu: no', 'apacheii: 15', 'charlson score: 0', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 946', 'crp (mg/l): 73.1', 'ddimer (mg/l_feu): 1.3', 'procalcitonin (ng/ml): 36', 'lactate (mmol/l): 0.9', 'fibrinogen: 513', 'sofa: 8', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C1', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934957', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033799'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_01_39y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753021']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 39', 'Sex: male', 'icu: no', 'apacheii: 15', 'charlson score: 0', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 946', 'crp (mg/l): 73.1', 'ddimer (mg/l_feu): 1.3', 'procalcitonin (ng/ml): 36', 'lactate (mmol/l): 0.9', 'fibrinogen: 513', 'sofa: 8', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C1', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934957', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033799']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753022 -- <SAMPLE: GSM4753022>\n",
      "metadata: \n",
      " {'title': ['COVID_02_63y_male_NonICU'], 'geo_accession': ['GSM4753022'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 63', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 39', 'ferritin (ng/ml): 1060', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): 1.03', 'procalcitonin (ng/ml): 0.37', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C2', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934956', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033800'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_02_63y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753022']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 63', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 39', 'ferritin (ng/ml): 1060', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): 1.03', 'procalcitonin (ng/ml): 0.37', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C2', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934956', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033800']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753023 -- <SAMPLE: GSM4753023>\n",
      "metadata: \n",
      " {'title': ['COVID_03_33y_male_NonICU'], 'geo_accession': ['GSM4753023'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 33', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 18', 'ferritin (ng/ml): 1335', 'crp (mg/l): 53.2', 'ddimer (mg/l_feu): 1.48', 'procalcitonin (ng/ml): 0.07', 'lactate (mmol/l): unknown', 'fibrinogen: 513', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C3', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934955', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033801'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_03_33y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753023']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 33', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 18', 'ferritin (ng/ml): 1335', 'crp (mg/l): 53.2', 'ddimer (mg/l_feu): 1.48', 'procalcitonin (ng/ml): 0.07', 'lactate (mmol/l): unknown', 'fibrinogen: 513', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C3', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934955', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033801']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753024 -- <SAMPLE: GSM4753024>\n",
      "metadata: \n",
      " {'title': ['COVID_04_49y_male_NonICU'], 'geo_accession': ['GSM4753024'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 49', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 39', 'ferritin (ng/ml): 583', 'crp (mg/l): 251.1', 'ddimer (mg/l_feu): 1.32', 'procalcitonin (ng/ml): 0.98', 'lactate (mmol/l): 0.87', 'fibrinogen: 949', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C4', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934954', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033802'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_04_49y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753024']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 49', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 39', 'ferritin (ng/ml): 583', 'crp (mg/l): 251.1', 'ddimer (mg/l_feu): 1.32', 'procalcitonin (ng/ml): 0.98', 'lactate (mmol/l): 0.87', 'fibrinogen: 949', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C4', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934954', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033802']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753025 -- <SAMPLE: GSM4753025>\n",
      "metadata: \n",
      " {'title': ['COVID_05_49y_male_NonICU'], 'geo_accession': ['GSM4753025'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 49', 'Sex: male', 'icu: no', 'apacheii: 19', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 23', 'dm: 0', 'hospital-free days post 45 day followup (days): 27', 'ferritin (ng/ml): 800', 'crp (mg/l): 355.8', 'ddimer (mg/l_feu): 0.69', 'procalcitonin (ng/ml): 4.92', 'lactate (mmol/l): 1.48', 'fibrinogen: 929', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C5', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934953', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033803'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_05_49y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753025']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 49', 'Sex: male', 'icu: no', 'apacheii: 19', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 23', 'dm: 0', 'hospital-free days post 45 day followup (days): 27', 'ferritin (ng/ml): 800', 'crp (mg/l): 355.8', 'ddimer (mg/l_feu): 0.69', 'procalcitonin (ng/ml): 4.92', 'lactate (mmol/l): 1.48', 'fibrinogen: 929', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C5', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934953', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033803']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753026 -- <SAMPLE: GSM4753026>\n",
      "metadata: \n",
      " {'title': ['COVID_06_:y_male_NonICU'], 'geo_accession': ['GSM4753026'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): :', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 36', 'ferritin (ng/ml): 563', 'crp (mg/l): 129.1', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 0.67', 'lactate (mmol/l): 0.86', 'fibrinogen: 769', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C6', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934952', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033804'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_06_:y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753026']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): :', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 36', 'ferritin (ng/ml): 563', 'crp (mg/l): 129.1', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 0.67', 'lactate (mmol/l): 0.86', 'fibrinogen: 769', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C6', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934952', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033804']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753027 -- <SAMPLE: GSM4753027>\n",
      "metadata: \n",
      " {'title': ['COVID_07_38y_female_NonICU'], 'geo_accession': ['GSM4753027'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 38', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 7', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 42', 'ferritin (ng/ml): 366', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): 0.87', 'procalcitonin (ng/ml): 0.06', 'lactate (mmol/l): 1.17', 'fibrinogen: 478', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C7', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934951', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033805'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_07_38y_female_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753027']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 38', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 7', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 42', 'ferritin (ng/ml): 366', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): 0.87', 'procalcitonin (ng/ml): 0.06', 'lactate (mmol/l): 1.17', 'fibrinogen: 478', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C7', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934951', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033805']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753028 -- <SAMPLE: GSM4753028>\n",
      "metadata: \n",
      " {'title': ['COVID_08_78y_male_ICU'], 'geo_accession': ['GSM4753028'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 78', 'Sex: male', 'icu: yes', 'apacheii: 43', 'charlson score: 7', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1103', 'crp (mg/l): 79.5', 'ddimer (mg/l_feu): 12.16', 'procalcitonin (ng/ml): 4.2', 'lactate (mmol/l): 1.65', 'fibrinogen: 780', 'sofa: 16', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C8', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934950', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033806'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_08_78y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753028']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 78', 'Sex: male', 'icu: yes', 'apacheii: 43', 'charlson score: 7', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1103', 'crp (mg/l): 79.5', 'ddimer (mg/l_feu): 12.16', 'procalcitonin (ng/ml): 4.2', 'lactate (mmol/l): 1.65', 'fibrinogen: 780', 'sofa: 16', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C8', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934950', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033806']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753029 -- <SAMPLE: GSM4753029>\n",
      "metadata: \n",
      " {'title': ['COVID_09_64y_female_ICU'], 'geo_accession': ['GSM4753029'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 64', 'Sex: female', 'icu: yes', 'apacheii: 31', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 680', 'crp (mg/l): 203.5', 'ddimer (mg/l_feu): 102.21', 'procalcitonin (ng/ml): 0.25', 'lactate (mmol/l): 1.14', 'fibrinogen: 317', 'sofa: 12', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C9', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934949', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033807'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_09_64y_female_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753029']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 64', 'Sex: female', 'icu: yes', 'apacheii: 31', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 680', 'crp (mg/l): 203.5', 'ddimer (mg/l_feu): 102.21', 'procalcitonin (ng/ml): 0.25', 'lactate (mmol/l): 1.14', 'fibrinogen: 317', 'sofa: 12', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C9', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934949', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033807']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753030 -- <SAMPLE: GSM4753030>\n",
      "metadata: \n",
      " {'title': ['COVID_10_62y_male_ICU'], 'geo_accession': ['GSM4753030'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 62', 'Sex: male', 'icu: yes', 'apacheii: 34', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 2', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1746', 'crp (mg/l): 197.8', 'ddimer (mg/l_feu): 11.54', 'procalcitonin (ng/ml): 1.1', 'lactate (mmol/l): 1.45', 'fibrinogen: 893', 'sofa: 15', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C10', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934948', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033808'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_10_62y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753030']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 62', 'Sex: male', 'icu: yes', 'apacheii: 34', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 2', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1746', 'crp (mg/l): 197.8', 'ddimer (mg/l_feu): 11.54', 'procalcitonin (ng/ml): 1.1', 'lactate (mmol/l): 1.45', 'fibrinogen: 893', 'sofa: 15', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C10', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934948', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033808']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753031 -- <SAMPLE: GSM4753031>\n",
      "metadata: \n",
      " {'title': ['COVID_11_52y_female_NonICU'], 'geo_accession': ['GSM4753031'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 52', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 37', 'ferritin (ng/ml): 401', 'crp (mg/l): 64.1', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 0.21', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C11', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934947', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033809'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_11_52y_female_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753031']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 52', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 37', 'ferritin (ng/ml): 401', 'crp (mg/l): 64.1', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 0.21', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C11', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934947', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033809']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753032 -- <SAMPLE: GSM4753032>\n",
      "metadata: \n",
      " {'title': ['COVID_12_50y_male_ICU'], 'geo_accession': ['GSM4753032'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 50', 'Sex: male', 'icu: yes', 'apacheii: 27', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 15', 'dm: 0', 'hospital-free days post 45 day followup (days): 22', 'ferritin (ng/ml): 2303', 'crp (mg/l): 199.8', 'ddimer (mg/l_feu): 2.21', 'procalcitonin (ng/ml): 2.61', 'lactate (mmol/l): 2.1', 'fibrinogen: 910', 'sofa: 10', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C12', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934946', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033810'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_12_50y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753032']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 50', 'Sex: male', 'icu: yes', 'apacheii: 27', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 15', 'dm: 0', 'hospital-free days post 45 day followup (days): 22', 'ferritin (ng/ml): 2303', 'crp (mg/l): 199.8', 'ddimer (mg/l_feu): 2.21', 'procalcitonin (ng/ml): 2.61', 'lactate (mmol/l): 2.1', 'fibrinogen: 910', 'sofa: 10', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C12', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934946', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033810']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753033 -- <SAMPLE: GSM4753033>\n",
      "metadata: \n",
      " {'title': ['COVID_13_37y_male_NonICU'], 'geo_accession': ['GSM4753033'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 37', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 39', 'ferritin (ng/ml): 245', 'crp (mg/l): 122.5', 'ddimer (mg/l_feu): 0.58', 'procalcitonin (ng/ml): 0.15', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C13', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934945', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033811'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_13_37y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753033']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 37', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 39', 'ferritin (ng/ml): 245', 'crp (mg/l): 122.5', 'ddimer (mg/l_feu): 0.58', 'procalcitonin (ng/ml): 0.15', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C13', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934945', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033811']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753034 -- <SAMPLE: GSM4753034>\n",
      "metadata: \n",
      " {'title': ['COVID_14_55y_male_ICU'], 'geo_accession': ['GSM4753034'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 55', 'Sex: male', 'icu: yes', 'apacheii: 23', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 20', 'ferritin (ng/ml): 1632', 'crp (mg/l): 408.8', 'ddimer (mg/l_feu): 33.77', 'procalcitonin (ng/ml): 1.46', 'lactate (mmol/l): 1.35', 'fibrinogen: 654', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C14', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934944', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033812'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_14_55y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753034']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 55', 'Sex: male', 'icu: yes', 'apacheii: 23', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 20', 'ferritin (ng/ml): 1632', 'crp (mg/l): 408.8', 'ddimer (mg/l_feu): 33.77', 'procalcitonin (ng/ml): 1.46', 'lactate (mmol/l): 1.35', 'fibrinogen: 654', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C14', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934944', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033812']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753035 -- <SAMPLE: GSM4753035>\n",
      "metadata: \n",
      " {'title': ['COVID_15_68y_male_ICU'], 'geo_accession': ['GSM4753035'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 68', 'Sex: male', 'icu: yes', 'apacheii: 27', 'charlson score: 11', 'mechanical ventilation: yes', 'ventilator-free days: 2', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1205', 'crp (mg/l): 222.2', 'ddimer (mg/l_feu): 5.27', 'procalcitonin (ng/ml): 0.08', 'lactate (mmol/l): 1.35', 'fibrinogen: 675', 'sofa: 12', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C15', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934943', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033813'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_15_68y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753035']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 68', 'Sex: male', 'icu: yes', 'apacheii: 27', 'charlson score: 11', 'mechanical ventilation: yes', 'ventilator-free days: 2', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1205', 'crp (mg/l): 222.2', 'ddimer (mg/l_feu): 5.27', 'procalcitonin (ng/ml): 0.08', 'lactate (mmol/l): 1.35', 'fibrinogen: 675', 'sofa: 12', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C15', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934943', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033813']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753036 -- <SAMPLE: GSM4753036>\n",
      "metadata: \n",
      " {'title': ['COVID_16_48y_male_NonICU'], 'geo_accession': ['GSM4753036'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 48', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 42', 'ferritin (ng/ml): 923', 'crp (mg/l): 206.1', 'ddimer (mg/l_feu): 0.35', 'procalcitonin (ng/ml): 0.07', 'lactate (mmol/l): 1.23', 'fibrinogen: 540', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C16', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934942', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033814'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_16_48y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753036']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 48', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 42', 'ferritin (ng/ml): 923', 'crp (mg/l): 206.1', 'ddimer (mg/l_feu): 0.35', 'procalcitonin (ng/ml): 0.07', 'lactate (mmol/l): 1.23', 'fibrinogen: 540', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C16', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934942', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033814']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753037 -- <SAMPLE: GSM4753037>\n",
      "metadata: \n",
      " {'title': ['COVID_17_54y_male_NonICU'], 'geo_accession': ['GSM4753037'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 54', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 33', 'ferritin (ng/ml): 1249', 'crp (mg/l): 52.3', 'ddimer (mg/l_feu): 4.2', 'procalcitonin (ng/ml): 1.02', 'lactate (mmol/l): 1.47', 'fibrinogen: 583', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C17', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934940', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033815'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_17_54y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753037']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 54', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 33', 'ferritin (ng/ml): 1249', 'crp (mg/l): 52.3', 'ddimer (mg/l_feu): 4.2', 'procalcitonin (ng/ml): 1.02', 'lactate (mmol/l): 1.47', 'fibrinogen: 583', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C17', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934940', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033815']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753038 -- <SAMPLE: GSM4753038>\n",
      "metadata: \n",
      " {'title': ['COVID_18_70y_female_NonICU'], 'geo_accession': ['GSM4753038'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 70', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 34', 'ferritin (ng/ml): 77', 'crp (mg/l): 82.2', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 0.5', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C18', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934939', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033816'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_18_70y_female_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753038']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 70', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 34', 'ferritin (ng/ml): 77', 'crp (mg/l): 82.2', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 0.5', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C18', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934939', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033816']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753039 -- <SAMPLE: GSM4753039>\n",
      "metadata: \n",
      " {'title': ['COVID_19_51y_male_NonICU'], 'geo_accession': ['GSM4753039'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 51', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 38', 'ferritin (ng/ml): 1116', 'crp (mg/l): 110.1', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 0.16', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C19', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934938', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033817'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_19_51y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753039']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 51', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 38', 'ferritin (ng/ml): 1116', 'crp (mg/l): 110.1', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 0.16', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C19', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934938', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033817']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753040 -- <SAMPLE: GSM4753040>\n",
      "metadata: \n",
      " {'title': ['COVID_20_62y_male_ICU'], 'geo_accession': ['GSM4753040'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 62', 'Sex: male', 'icu: yes', 'apacheii: 8', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 35', 'ferritin (ng/ml): 1219', 'crp (mg/l): 37.7', 'ddimer (mg/l_feu): 1.12', 'procalcitonin (ng/ml): 0.31', 'lactate (mmol/l): 1.05', 'fibrinogen: 301', 'sofa: 2', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C20', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934937', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033818'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_20_62y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753040']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 62', 'Sex: male', 'icu: yes', 'apacheii: 8', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 35', 'ferritin (ng/ml): 1219', 'crp (mg/l): 37.7', 'ddimer (mg/l_feu): 1.12', 'procalcitonin (ng/ml): 0.31', 'lactate (mmol/l): 1.05', 'fibrinogen: 301', 'sofa: 2', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C20', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934937', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033818']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753041 -- <SAMPLE: GSM4753041>\n",
      "metadata: \n",
      " {'title': ['COVID_21_66y_male_ICU'], 'geo_accession': ['GSM4753041'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 66', 'Sex: male', 'icu: yes', 'apacheii: 19', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 26', 'dm: 1', 'hospital-free days post 45 day followup (days): 32', 'ferritin (ng/ml): 107', 'crp (mg/l): 132.5', 'ddimer (mg/l_feu): 4.72', 'procalcitonin (ng/ml): 0.05', 'lactate (mmol/l): 1', 'fibrinogen: 457', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C21', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934936', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033819'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_21_66y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753041']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 66', 'Sex: male', 'icu: yes', 'apacheii: 19', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 26', 'dm: 1', 'hospital-free days post 45 day followup (days): 32', 'ferritin (ng/ml): 107', 'crp (mg/l): 132.5', 'ddimer (mg/l_feu): 4.72', 'procalcitonin (ng/ml): 0.05', 'lactate (mmol/l): 1', 'fibrinogen: 457', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C21', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934936', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033819']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753042 -- <SAMPLE: GSM4753042>\n",
      "metadata: \n",
      " {'title': ['COVID_22_43y_male_ICU'], 'geo_accession': ['GSM4753042'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 43', 'Sex: male', 'icu: yes', 'apacheii: 28', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 3438', 'crp (mg/l): 242.6', 'ddimer (mg/l_feu): 5.66', 'procalcitonin (ng/ml): 20.13', 'lactate (mmol/l): 0.92', 'fibrinogen: 949', 'sofa: 13', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C22', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934935', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033820'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_22_43y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753042']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 43', 'Sex: male', 'icu: yes', 'apacheii: 28', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 3438', 'crp (mg/l): 242.6', 'ddimer (mg/l_feu): 5.66', 'procalcitonin (ng/ml): 20.13', 'lactate (mmol/l): 0.92', 'fibrinogen: 949', 'sofa: 13', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C22', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934935', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033820']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753043 -- <SAMPLE: GSM4753043>\n",
      "metadata: \n",
      " {'title': ['COVID_23_76y_male_ICU'], 'geo_accession': ['GSM4753043'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 76', 'Sex: male', 'icu: yes', 'apacheii: 21', 'charlson score: 4', 'mechanical ventilation: yes', 'ventilator-free days: 20', 'dm: 1', 'hospital-free days post 45 day followup (days): 12', 'ferritin (ng/ml): 332', 'crp (mg/l): 107.6', 'ddimer (mg/l_feu): 21.55', 'procalcitonin (ng/ml): 0.33', 'lactate (mmol/l): 1.71', 'fibrinogen: 237', 'sofa: 12', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C23', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934934', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033821'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_23_76y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753043']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 76', 'Sex: male', 'icu: yes', 'apacheii: 21', 'charlson score: 4', 'mechanical ventilation: yes', 'ventilator-free days: 20', 'dm: 1', 'hospital-free days post 45 day followup (days): 12', 'ferritin (ng/ml): 332', 'crp (mg/l): 107.6', 'ddimer (mg/l_feu): 21.55', 'procalcitonin (ng/ml): 0.33', 'lactate (mmol/l): 1.71', 'fibrinogen: 237', 'sofa: 12', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C23', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934934', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033821']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753044 -- <SAMPLE: GSM4753044>\n",
      "metadata: \n",
      " {'title': ['COVID_24_55y_male_ICU'], 'geo_accession': ['GSM4753044'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 55', 'Sex: male', 'icu: yes', 'apacheii: 20', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 22', 'dm: 0', 'hospital-free days post 45 day followup (days): 23', 'ferritin (ng/ml): 1565', 'crp (mg/l): 64', 'ddimer (mg/l_feu): 1.76', 'procalcitonin (ng/ml): 0.73', 'lactate (mmol/l): unknown', 'fibrinogen: 365', 'sofa: 6', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C24', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934933', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033822'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_24_55y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753044']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 55', 'Sex: male', 'icu: yes', 'apacheii: 20', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 22', 'dm: 0', 'hospital-free days post 45 day followup (days): 23', 'ferritin (ng/ml): 1565', 'crp (mg/l): 64', 'ddimer (mg/l_feu): 1.76', 'procalcitonin (ng/ml): 0.73', 'lactate (mmol/l): unknown', 'fibrinogen: 365', 'sofa: 6', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C24', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934933', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033822']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753045 -- <SAMPLE: GSM4753045>\n",
      "metadata: \n",
      " {'title': ['COVID_25_55y_male_ICU'], 'geo_accession': ['GSM4753045'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 55', 'Sex: male', 'icu: yes', 'apacheii: 18', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 11', 'dm: 0', 'hospital-free days post 45 day followup (days): 26', 'ferritin (ng/ml): 355', 'crp (mg/l): 64.5', 'ddimer (mg/l_feu): 28.8', 'procalcitonin (ng/ml): 18', 'lactate (mmol/l): 0.94', 'fibrinogen: 719', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C25', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934932', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033823'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_25_55y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753045']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 55', 'Sex: male', 'icu: yes', 'apacheii: 18', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 11', 'dm: 0', 'hospital-free days post 45 day followup (days): 26', 'ferritin (ng/ml): 355', 'crp (mg/l): 64.5', 'ddimer (mg/l_feu): 28.8', 'procalcitonin (ng/ml): 18', 'lactate (mmol/l): 0.94', 'fibrinogen: 719', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C25', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934932', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033823']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753046 -- <SAMPLE: GSM4753046>\n",
      "metadata: \n",
      " {'title': ['COVID_26_41y_female_ICU'], 'geo_accession': ['GSM4753046'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 41', 'Sex: female', 'icu: yes', 'apacheii: 13', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 25', 'dm: 0', 'hospital-free days post 45 day followup (days): 27', 'ferritin (ng/ml): 318', 'crp (mg/l): 51.1', 'ddimer (mg/l_feu): 0.65', 'procalcitonin (ng/ml): 0.13', 'lactate (mmol/l): 0.8', 'fibrinogen: unknown', 'sofa: 9', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C26', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934931', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033824'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_26_41y_female_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753046']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 41', 'Sex: female', 'icu: yes', 'apacheii: 13', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 25', 'dm: 0', 'hospital-free days post 45 day followup (days): 27', 'ferritin (ng/ml): 318', 'crp (mg/l): 51.1', 'ddimer (mg/l_feu): 0.65', 'procalcitonin (ng/ml): 0.13', 'lactate (mmol/l): 0.8', 'fibrinogen: unknown', 'sofa: 9', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C26', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934931', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033824']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753047 -- <SAMPLE: GSM4753047>\n",
      "metadata: \n",
      " {'title': ['COVID_27_71y_female_ICU'], 'geo_accession': ['GSM4753047'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 71', 'Sex: female', 'icu: yes', 'apacheii: 29', 'charlson score: 6', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 882', 'crp (mg/l): 285', 'ddimer (mg/l_feu): 58.18', 'procalcitonin (ng/ml): 0.9', 'lactate (mmol/l): unknown', 'fibrinogen: 659', 'sofa: 11', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C27', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934930', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033825'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_27_71y_female_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753047']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 71', 'Sex: female', 'icu: yes', 'apacheii: 29', 'charlson score: 6', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 882', 'crp (mg/l): 285', 'ddimer (mg/l_feu): 58.18', 'procalcitonin (ng/ml): 0.9', 'lactate (mmol/l): unknown', 'fibrinogen: 659', 'sofa: 11', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C27', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934930', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033825']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753048 -- <SAMPLE: GSM4753048>\n",
      "metadata: \n",
      " {'title': ['COVID_28_63y_male_ICU'], 'geo_accession': ['GSM4753048'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 63', 'Sex: male', 'icu: yes', 'apacheii: 17', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 109', 'crp (mg/l): 40.1', 'ddimer (mg/l_feu): 2.03', 'procalcitonin (ng/ml): 0.93', 'lactate (mmol/l): unknown', 'fibrinogen: 554', 'sofa: 6', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C28', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934929', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033826'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_28_63y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753048']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 63', 'Sex: male', 'icu: yes', 'apacheii: 17', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 109', 'crp (mg/l): 40.1', 'ddimer (mg/l_feu): 2.03', 'procalcitonin (ng/ml): 0.93', 'lactate (mmol/l): unknown', 'fibrinogen: 554', 'sofa: 6', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C28', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934929', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033826']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753049 -- <SAMPLE: GSM4753049>\n",
      "metadata: \n",
      " {'title': ['COVID_29_63y_female_ICU'], 'geo_accession': ['GSM4753049'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 63', 'Sex: female', 'icu: yes', 'apacheii: 17', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 21', 'dm: 1', 'hospital-free days post 45 day followup (days): 10', 'ferritin (ng/ml): 376', 'crp (mg/l): 350.6', 'ddimer (mg/l_feu): 1.75', 'procalcitonin (ng/ml): 1.43', 'lactate (mmol/l): unknown', 'fibrinogen: 439', 'sofa: 6', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C29', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934928', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033827'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_29_63y_female_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753049']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 63', 'Sex: female', 'icu: yes', 'apacheii: 17', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 21', 'dm: 1', 'hospital-free days post 45 day followup (days): 10', 'ferritin (ng/ml): 376', 'crp (mg/l): 350.6', 'ddimer (mg/l_feu): 1.75', 'procalcitonin (ng/ml): 1.43', 'lactate (mmol/l): unknown', 'fibrinogen: 439', 'sofa: 6', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C29', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934928', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033827']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753050 -- <SAMPLE: GSM4753050>\n",
      "metadata: \n",
      " {'title': ['COVID_30_54y_male_ICU'], 'geo_accession': ['GSM4753050'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 54', 'Sex: male', 'icu: yes', 'apacheii: 25', 'charlson score: 7', 'mechanical ventilation: yes', 'ventilator-free days: 7', 'dm: 1', 'hospital-free days post 45 day followup (days): 3', 'ferritin (ng/ml): 686', 'crp (mg/l): 171.1', 'ddimer (mg/l_feu): 2.37', 'procalcitonin (ng/ml): 13', 'lactate (mmol/l): 0.87', 'fibrinogen: 635', 'sofa: 8', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C30', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934927', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033828'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_30_54y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753050']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 54', 'Sex: male', 'icu: yes', 'apacheii: 25', 'charlson score: 7', 'mechanical ventilation: yes', 'ventilator-free days: 7', 'dm: 1', 'hospital-free days post 45 day followup (days): 3', 'ferritin (ng/ml): 686', 'crp (mg/l): 171.1', 'ddimer (mg/l_feu): 2.37', 'procalcitonin (ng/ml): 13', 'lactate (mmol/l): 0.87', 'fibrinogen: 635', 'sofa: 8', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C30', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934927', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033828']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753051 -- <SAMPLE: GSM4753051>\n",
      "metadata: \n",
      " {'title': ['COVID_31_50y_male_ICU'], 'geo_accession': ['GSM4753051'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 50', 'Sex: male', 'icu: yes', 'apacheii: 24', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 374', 'crp (mg/l): 28.4', 'ddimer (mg/l_feu): 74.42', 'procalcitonin (ng/ml): 2.83', 'lactate (mmol/l): 2.85', 'fibrinogen: 140', 'sofa: 14', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C31', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934986', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033829'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_31_50y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753051']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 50', 'Sex: male', 'icu: yes', 'apacheii: 24', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 374', 'crp (mg/l): 28.4', 'ddimer (mg/l_feu): 74.42', 'procalcitonin (ng/ml): 2.83', 'lactate (mmol/l): 2.85', 'fibrinogen: 140', 'sofa: 14', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C31', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934986', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033829']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753052 -- <SAMPLE: GSM4753052>\n",
      "metadata: \n",
      " {'title': ['COVID_32_72y_male_ICU'], 'geo_accession': ['GSM4753052'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 72', 'Sex: male', 'icu: yes', 'apacheii: 21', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 7', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1294', 'crp (mg/l): 311.5', 'ddimer (mg/l_feu): 1.32', 'procalcitonin (ng/ml): 2.17', 'lactate (mmol/l): 1.05', 'fibrinogen: 513', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C32', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934985', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033830'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_32_72y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753052']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 72', 'Sex: male', 'icu: yes', 'apacheii: 21', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 7', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1294', 'crp (mg/l): 311.5', 'ddimer (mg/l_feu): 1.32', 'procalcitonin (ng/ml): 2.17', 'lactate (mmol/l): 1.05', 'fibrinogen: 513', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C32', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934985', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033830']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753053 -- <SAMPLE: GSM4753053>\n",
      "metadata: \n",
      " {'title': ['COVID_33_81y_male_NonICU'], 'geo_accession': ['GSM4753053'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 81', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 40', 'ferritin (ng/ml): 1013', 'crp (mg/l): 222.4', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 0.5', 'lactate (mmol/l): 1.39', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C33', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934984', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033831'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_33_81y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753053']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 81', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 40', 'ferritin (ng/ml): 1013', 'crp (mg/l): 222.4', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 0.5', 'lactate (mmol/l): 1.39', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C33', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934984', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033831']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753054 -- <SAMPLE: GSM4753054>\n",
      "metadata: \n",
      " {'title': ['COVID_34_64y_female_NonICU'], 'geo_accession': ['GSM4753054'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 64', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 36', 'ferritin (ng/ml): 563', 'crp (mg/l): 47.8', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 1.01', 'fibrinogen: 467', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C34', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934983', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033832'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_34_64y_female_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753054']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 64', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 36', 'ferritin (ng/ml): 563', 'crp (mg/l): 47.8', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 1.01', 'fibrinogen: 467', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C34', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934983', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033832']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753055 -- <SAMPLE: GSM4753055>\n",
      "metadata: \n",
      " {'title': ['COVID_35_58y_female_NonICU'], 'geo_accession': ['GSM4753055'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 58', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 3', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 35', 'ferritin (ng/ml): 4859', 'crp (mg/l): 129.6', 'ddimer (mg/l_feu): 0.5', 'procalcitonin (ng/ml): 0.39', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C35', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934982', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033833'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_35_58y_female_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753055']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 58', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 3', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 35', 'ferritin (ng/ml): 4859', 'crp (mg/l): 129.6', 'ddimer (mg/l_feu): 0.5', 'procalcitonin (ng/ml): 0.39', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C35', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934982', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033833']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753056 -- <SAMPLE: GSM4753056>\n",
      "metadata: \n",
      " {'title': ['COVID_36_68y_male_NonICU'], 'geo_accession': ['GSM4753056'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 68', 'Sex: male', 'icu: no', 'apacheii: 26', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1689', 'crp (mg/l): 203.7', 'ddimer (mg/l_feu): 13.37', 'procalcitonin (ng/ml): 1.2', 'lactate (mmol/l): 1.24', 'fibrinogen: 752', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C36', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934981', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033834'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_36_68y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753056']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 68', 'Sex: male', 'icu: no', 'apacheii: 26', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1689', 'crp (mg/l): 203.7', 'ddimer (mg/l_feu): 13.37', 'procalcitonin (ng/ml): 1.2', 'lactate (mmol/l): 1.24', 'fibrinogen: 752', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C36', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934981', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033834']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753057 -- <SAMPLE: GSM4753057>\n",
      "metadata: \n",
      " {'title': ['COVID_37_87y_male_NonICU'], 'geo_accession': ['GSM4753057'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 87', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 43', 'ferritin (ng/ml): 248', 'crp (mg/l): 158.7', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C37', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934980', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033835'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_37_87y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753057']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 87', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 43', 'ferritin (ng/ml): 248', 'crp (mg/l): 158.7', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C37', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934980', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033835']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753058 -- <SAMPLE: GSM4753058>\n",
      "metadata: \n",
      " {'title': ['COVID_38_68y_male_ICU'], 'geo_accession': ['GSM4753058'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 68', 'Sex: male', 'icu: yes', 'apacheii: 7', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 16', 'ferritin (ng/ml): 2212', 'crp (mg/l): 133.6', 'ddimer (mg/l_feu): 7.48', 'procalcitonin (ng/ml): 0.33', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: 2', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C38', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934979', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033836'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_38_68y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753058']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 68', 'Sex: male', 'icu: yes', 'apacheii: 7', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 16', 'ferritin (ng/ml): 2212', 'crp (mg/l): 133.6', 'ddimer (mg/l_feu): 7.48', 'procalcitonin (ng/ml): 0.33', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: 2', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C38', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934979', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033836']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753059 -- <SAMPLE: GSM4753059>\n",
      "metadata: \n",
      " {'title': ['COVID_39_80y_female_ICU'], 'geo_accession': ['GSM4753059'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 80', 'Sex: female', 'icu: yes', 'apacheii: 33', 'charlson score: 7', 'mechanical ventilation: yes', 'ventilator-free days: 12', 'dm: 1', 'hospital-free days post 45 day followup (days): 3', 'ferritin (ng/ml): 747', 'crp (mg/l): 247', 'ddimer (mg/l_feu): 18.71', 'procalcitonin (ng/ml): 86.39', 'lactate (mmol/l): unknown', 'fibrinogen: 659', 'sofa: 10', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C39', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934978', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033837'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_39_80y_female_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753059']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 80', 'Sex: female', 'icu: yes', 'apacheii: 33', 'charlson score: 7', 'mechanical ventilation: yes', 'ventilator-free days: 12', 'dm: 1', 'hospital-free days post 45 day followup (days): 3', 'ferritin (ng/ml): 747', 'crp (mg/l): 247', 'ddimer (mg/l_feu): 18.71', 'procalcitonin (ng/ml): 86.39', 'lactate (mmol/l): unknown', 'fibrinogen: 659', 'sofa: 10', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C39', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934978', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033837']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753060 -- <SAMPLE: GSM4753060>\n",
      "metadata: \n",
      " {'title': ['COVID_40_66y_male_ICU'], 'geo_accession': ['GSM4753060'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 66', 'Sex: male', 'icu: yes', 'apacheii: 18', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 22', 'dm: 1', 'hospital-free days post 45 day followup (days): 29', 'ferritin (ng/ml): 566', 'crp (mg/l): 267.2', 'ddimer (mg/l_feu): 16.18', 'procalcitonin (ng/ml): 1.7', 'lactate (mmol/l): 1.65', 'fibrinogen: unknown', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C40', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934977', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033838'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_40_66y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753060']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 66', 'Sex: male', 'icu: yes', 'apacheii: 18', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 22', 'dm: 1', 'hospital-free days post 45 day followup (days): 29', 'ferritin (ng/ml): 566', 'crp (mg/l): 267.2', 'ddimer (mg/l_feu): 16.18', 'procalcitonin (ng/ml): 1.7', 'lactate (mmol/l): 1.65', 'fibrinogen: unknown', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C40', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934977', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033838']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753061 -- <SAMPLE: GSM4753061>\n",
      "metadata: \n",
      " {'title': ['COVID_41_74y_male_ICU'], 'geo_accession': ['GSM4753061'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 74', 'Sex: male', 'icu: yes', 'apacheii: 30', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 15', 'dm: 0', 'hospital-free days post 45 day followup (days): 20', 'ferritin (ng/ml): 755', 'crp (mg/l): 252', 'ddimer (mg/l_feu): 34.04', 'procalcitonin (ng/ml): 1.33', 'lactate (mmol/l): 1.69', 'fibrinogen: 268', 'sofa: 8', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C41', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934976', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033839'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_41_74y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753061']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 74', 'Sex: male', 'icu: yes', 'apacheii: 30', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 15', 'dm: 0', 'hospital-free days post 45 day followup (days): 20', 'ferritin (ng/ml): 755', 'crp (mg/l): 252', 'ddimer (mg/l_feu): 34.04', 'procalcitonin (ng/ml): 1.33', 'lactate (mmol/l): 1.69', 'fibrinogen: 268', 'sofa: 8', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C41', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934976', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033839']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753062 -- <SAMPLE: GSM4753062>\n",
      "metadata: \n",
      " {'title': ['COVID_42_21y_female_ICU'], 'geo_accession': ['GSM4753062'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 21', 'Sex: female', 'icu: yes', 'apacheii: 22', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 11', 'dm: 1', 'hospital-free days post 45 day followup (days): 28', 'ferritin (ng/ml): 383', 'crp (mg/l): 33.6', 'ddimer (mg/l_feu): 1.37', 'procalcitonin (ng/ml): 2', 'lactate (mmol/l): unknown', 'fibrinogen: 513', 'sofa: 13', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C42', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934975', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033840'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_42_21y_female_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753062']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 21', 'Sex: female', 'icu: yes', 'apacheii: 22', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 11', 'dm: 1', 'hospital-free days post 45 day followup (days): 28', 'ferritin (ng/ml): 383', 'crp (mg/l): 33.6', 'ddimer (mg/l_feu): 1.37', 'procalcitonin (ng/ml): 2', 'lactate (mmol/l): unknown', 'fibrinogen: 513', 'sofa: 13', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C42', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934975', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033840']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753063 -- <SAMPLE: GSM4753063>\n",
      "metadata: \n",
      " {'title': ['COVID_43_83y_female_ICU'], 'geo_accession': ['GSM4753063'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 83', 'Sex: female', 'icu: yes', 'apacheii: 25', 'charlson score: 4', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1114', 'crp (mg/l): 164.3', 'ddimer (mg/l_feu): 18.83', 'procalcitonin (ng/ml): 0.6', 'lactate (mmol/l): unknown', 'fibrinogen: 526', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C43', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934974', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033841'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_43_83y_female_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753063']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 83', 'Sex: female', 'icu: yes', 'apacheii: 25', 'charlson score: 4', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1114', 'crp (mg/l): 164.3', 'ddimer (mg/l_feu): 18.83', 'procalcitonin (ng/ml): 0.6', 'lactate (mmol/l): unknown', 'fibrinogen: 526', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C43', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934974', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033841']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753064 -- <SAMPLE: GSM4753064>\n",
      "metadata: \n",
      " {'title': ['COVID_44_46y_male_ICU'], 'geo_accession': ['GSM4753064'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 46', 'Sex: male', 'icu: yes', 'apacheii: 23', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 822', 'crp (mg/l): 235', 'ddimer (mg/l_feu): 12.81', 'procalcitonin (ng/ml): 2.89', 'lactate (mmol/l): 0.58', 'fibrinogen: 703', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C44', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934973', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033842'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_44_46y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753064']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 46', 'Sex: male', 'icu: yes', 'apacheii: 23', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 822', 'crp (mg/l): 235', 'ddimer (mg/l_feu): 12.81', 'procalcitonin (ng/ml): 2.89', 'lactate (mmol/l): 0.58', 'fibrinogen: 703', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C44', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934973', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033842']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753065 -- <SAMPLE: GSM4753065>\n",
      "metadata: \n",
      " {'title': ['COVID_45_62y_female_ICU'], 'geo_accession': ['GSM4753065'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 62', 'Sex: female', 'icu: yes', 'apacheii: 14', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 21', 'dm: 0', 'hospital-free days post 45 day followup (days): 20', 'ferritin (ng/ml): 505', 'crp (mg/l): 147.3', 'ddimer (mg/l_feu): 104.42', 'procalcitonin (ng/ml): 0.19', 'lactate (mmol/l): 1.53', 'fibrinogen: 479', 'sofa: 6', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C45', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934972', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033843'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_45_62y_female_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753065']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 62', 'Sex: female', 'icu: yes', 'apacheii: 14', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 21', 'dm: 0', 'hospital-free days post 45 day followup (days): 20', 'ferritin (ng/ml): 505', 'crp (mg/l): 147.3', 'ddimer (mg/l_feu): 104.42', 'procalcitonin (ng/ml): 0.19', 'lactate (mmol/l): 1.53', 'fibrinogen: 479', 'sofa: 6', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C45', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934972', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033843']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753066 -- <SAMPLE: GSM4753066>\n",
      "metadata: \n",
      " {'title': ['COVID_46_62y_male_ICU'], 'geo_accession': ['GSM4753066'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 62', 'Sex: male', 'icu: yes', 'apacheii: 15', 'charlson score: 4', 'mechanical ventilation: yes', 'ventilator-free days: 26', 'dm: 1', 'hospital-free days post 45 day followup (days): 39', 'ferritin (ng/ml): 378', 'crp (mg/l): 15.9', 'ddimer (mg/l_feu): 19.99', 'procalcitonin (ng/ml): 2.29', 'lactate (mmol/l): 0.87', 'fibrinogen: 378', 'sofa: 4', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C46', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934971', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033844'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_46_62y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753066']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 62', 'Sex: male', 'icu: yes', 'apacheii: 15', 'charlson score: 4', 'mechanical ventilation: yes', 'ventilator-free days: 26', 'dm: 1', 'hospital-free days post 45 day followup (days): 39', 'ferritin (ng/ml): 378', 'crp (mg/l): 15.9', 'ddimer (mg/l_feu): 19.99', 'procalcitonin (ng/ml): 2.29', 'lactate (mmol/l): 0.87', 'fibrinogen: 378', 'sofa: 4', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C46', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934971', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033844']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753067 -- <SAMPLE: GSM4753067>\n",
      "metadata: \n",
      " {'title': ['COVID_47_78y_male_ICU'], 'geo_accession': ['GSM4753067'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 78', 'Sex: male', 'icu: yes', 'apacheii: 21', 'charlson score: 5', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1102', 'crp (mg/l): 166.3', 'ddimer (mg/l_feu): 56.27', 'procalcitonin (ng/ml): 0.43', 'lactate (mmol/l): 1.03', 'fibrinogen: 438', 'sofa: 8', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C47', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934970', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033845'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_47_78y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753067']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 78', 'Sex: male', 'icu: yes', 'apacheii: 21', 'charlson score: 5', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1102', 'crp (mg/l): 166.3', 'ddimer (mg/l_feu): 56.27', 'procalcitonin (ng/ml): 0.43', 'lactate (mmol/l): 1.03', 'fibrinogen: 438', 'sofa: 8', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C47', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934970', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033845']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753068 -- <SAMPLE: GSM4753068>\n",
      "metadata: \n",
      " {'title': ['COVID_48_72y_female_ICU'], 'geo_accession': ['GSM4753068'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 72', 'Sex: female', 'icu: yes', 'apacheii: 33', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 301', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): 51.7', 'procalcitonin (ng/ml): 1.01', 'lactate (mmol/l): 1.3', 'fibrinogen: unknown', 'sofa: 9', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C48', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934969', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033846'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_48_72y_female_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753068']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 72', 'Sex: female', 'icu: yes', 'apacheii: 33', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 301', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): 51.7', 'procalcitonin (ng/ml): 1.01', 'lactate (mmol/l): 1.3', 'fibrinogen: unknown', 'sofa: 9', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C48', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934969', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033846']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753069 -- <SAMPLE: GSM4753069>\n",
      "metadata: \n",
      " {'title': ['COVID_49_73y_male_ICU'], 'geo_accession': ['GSM4753069'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 73', 'Sex: male', 'icu: yes', 'apacheii: 18', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1320', 'crp (mg/l): 260.6', 'ddimer (mg/l_feu): 102.75', 'procalcitonin (ng/ml): 0.26', 'lactate (mmol/l): 1.3', 'fibrinogen: 404', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C49', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934968', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033847'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_49_73y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753069']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 73', 'Sex: male', 'icu: yes', 'apacheii: 18', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1320', 'crp (mg/l): 260.6', 'ddimer (mg/l_feu): 102.75', 'procalcitonin (ng/ml): 0.26', 'lactate (mmol/l): 1.3', 'fibrinogen: 404', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C49', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934968', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033847']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753070 -- <SAMPLE: GSM4753070>\n",
      "metadata: \n",
      " {'title': ['COVID_50_37y_male_ICU'], 'geo_accession': ['GSM4753070'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 37', 'Sex: male', 'icu: yes', 'apacheii: 34', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 4609', 'crp (mg/l): 71', 'ddimer (mg/l_feu): 18.88', 'procalcitonin (ng/ml): 2.31', 'lactate (mmol/l): 1.32', 'fibrinogen: 305', 'sofa: 18', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C50', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934967', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033848'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_50_37y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753070']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 37', 'Sex: male', 'icu: yes', 'apacheii: 34', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 4609', 'crp (mg/l): 71', 'ddimer (mg/l_feu): 18.88', 'procalcitonin (ng/ml): 2.31', 'lactate (mmol/l): 1.32', 'fibrinogen: 305', 'sofa: 18', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C50', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934967', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033848']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753071 -- <SAMPLE: GSM4753071>\n",
      "metadata: \n",
      " {'title': ['COVID_51_58y_female_NonICU'], 'geo_accession': ['GSM4753071'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 58', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 6', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 38', 'ferritin (ng/ml): 1859', 'crp (mg/l): 299.8', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 1.46', 'lactate (mmol/l): 1.28', 'fibrinogen: 910', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C51', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934966', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033849'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_51_58y_female_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753071']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 58', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 6', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 38', 'ferritin (ng/ml): 1859', 'crp (mg/l): 299.8', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 1.46', 'lactate (mmol/l): 1.28', 'fibrinogen: 910', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C51', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934966', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033849']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753072 -- <SAMPLE: GSM4753072>\n",
      "metadata: \n",
      " {'title': ['COVID_52_71y_male_NonICU'], 'geo_accession': ['GSM4753072'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 71', 'Sex: male', 'icu: no', 'apacheii: 27', 'charlson score: 7', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): 1.67', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: 458', 'sofa: 12', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C52', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934965', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033850'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_52_71y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753072']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 71', 'Sex: male', 'icu: no', 'apacheii: 27', 'charlson score: 7', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): 1.67', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: 458', 'sofa: 12', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C52', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934965', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033850']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753073 -- <SAMPLE: GSM4753073>\n",
      "metadata: \n",
      " {'title': ['COVID_53_35y_female_NonICU'], 'geo_accession': ['GSM4753073'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 35', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 41', 'ferritin (ng/ml): 61', 'crp (mg/l): 38', 'ddimer (mg/l_feu): 1.35', 'procalcitonin (ng/ml): 0.17', 'lactate (mmol/l): 0.9', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C53', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934964', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033851'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_53_35y_female_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753073']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 35', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 41', 'ferritin (ng/ml): 61', 'crp (mg/l): 38', 'ddimer (mg/l_feu): 1.35', 'procalcitonin (ng/ml): 0.17', 'lactate (mmol/l): 0.9', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C53', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934964', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033851']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753074 -- <SAMPLE: GSM4753074>\n",
      "metadata: \n",
      " {'title': ['COVID_55_62y_female_ICU'], 'geo_accession': ['GSM4753074'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 62', 'Sex: female', 'icu: yes', 'apacheii: 39', 'charlson score: 7', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1035', 'crp (mg/l): 256.5', 'ddimer (mg/l_feu): 22.5', 'procalcitonin (ng/ml): 13.91', 'lactate (mmol/l): 1.52', 'fibrinogen: 654', 'sofa: 12', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C55', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934963', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033852'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_55_62y_female_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753074']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 62', 'Sex: female', 'icu: yes', 'apacheii: 39', 'charlson score: 7', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1035', 'crp (mg/l): 256.5', 'ddimer (mg/l_feu): 22.5', 'procalcitonin (ng/ml): 13.91', 'lactate (mmol/l): 1.52', 'fibrinogen: 654', 'sofa: 12', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C55', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934963', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033852']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753075 -- <SAMPLE: GSM4753075>\n",
      "metadata: \n",
      " {'title': ['COVID_56_33y_female_NonICU'], 'geo_accession': ['GSM4753075'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 33', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 39', 'ferritin (ng/ml): 91', 'crp (mg/l): 154.8', 'ddimer (mg/l_feu): 1.28', 'procalcitonin (ng/ml): 0.2', 'lactate (mmol/l): unknown', 'fibrinogen: 568', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C56', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934962', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033853'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_56_33y_female_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753075']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 33', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 39', 'ferritin (ng/ml): 91', 'crp (mg/l): 154.8', 'ddimer (mg/l_feu): 1.28', 'procalcitonin (ng/ml): 0.2', 'lactate (mmol/l): unknown', 'fibrinogen: 568', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C56', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934962', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033853']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753076 -- <SAMPLE: GSM4753076>\n",
      "metadata: \n",
      " {'title': ['COVID_57_30y_female_NonICU'], 'geo_accession': ['GSM4753076'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 30', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 43', 'ferritin (ng/ml): 32', 'crp (mg/l): 19.4', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 0.66', 'fibrinogen: 420', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C57', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934958', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033854'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_57_30y_female_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753076']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 30', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 43', 'ferritin (ng/ml): 32', 'crp (mg/l): 19.4', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 0.66', 'fibrinogen: 420', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C57', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934958', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033854']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753077 -- <SAMPLE: GSM4753077>\n",
      "metadata: \n",
      " {'title': ['COVID_58_62y_male_NonICU'], 'geo_accession': ['GSM4753077'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 62', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 6', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 44', 'ferritin (ng/ml): 228', 'crp (mg/l): 141.3', 'ddimer (mg/l_feu): 1.79', 'procalcitonin (ng/ml): 3.22', 'lactate (mmol/l): 0.77', 'fibrinogen: 703', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C58', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935042', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033855'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_58_62y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753077']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 62', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 6', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 44', 'ferritin (ng/ml): 228', 'crp (mg/l): 141.3', 'ddimer (mg/l_feu): 1.79', 'procalcitonin (ng/ml): 3.22', 'lactate (mmol/l): 0.77', 'fibrinogen: 703', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C58', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935042', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033855']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753078 -- <SAMPLE: GSM4753078>\n",
      "metadata: \n",
      " {'title': ['COVID_59_55y_male_NonICU'], 'geo_accession': ['GSM4753078'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 55', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 43', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C59', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934926', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033856'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_59_55y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753078']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 55', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 43', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C59', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934926', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033856']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753079 -- <SAMPLE: GSM4753079>\n",
      "metadata: \n",
      " {'title': ['COVID_60_49y_male_NonICU'], 'geo_accession': ['GSM4753079'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 49', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 43', 'ferritin (ng/ml): 825', 'crp (mg/l): 13.7', 'ddimer (mg/l_feu): 2.21', 'procalcitonin (ng/ml): 0.57', 'lactate (mmol/l): 0.65', 'fibrinogen: 874', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C60', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934925', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033857'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_60_49y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753079']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 49', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 43', 'ferritin (ng/ml): 825', 'crp (mg/l): 13.7', 'ddimer (mg/l_feu): 2.21', 'procalcitonin (ng/ml): 0.57', 'lactate (mmol/l): 0.65', 'fibrinogen: 874', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C60', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934925', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033857']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753080 -- <SAMPLE: GSM4753080>\n",
      "metadata: \n",
      " {'title': ['COVID_61_54y_female_NonICU'], 'geo_accession': ['GSM4753080'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 54', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 38', 'ferritin (ng/ml): 206', 'crp (mg/l): 114', 'ddimer (mg/l_feu): 0.63', 'procalcitonin (ng/ml): 0.07', 'lactate (mmol/l): unknown', 'fibrinogen: 617', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C61', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934924', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033858'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_61_54y_female_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753080']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 54', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 38', 'ferritin (ng/ml): 206', 'crp (mg/l): 114', 'ddimer (mg/l_feu): 0.63', 'procalcitonin (ng/ml): 0.07', 'lactate (mmol/l): unknown', 'fibrinogen: 617', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C61', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934924', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033858']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753081 -- <SAMPLE: GSM4753081>\n",
      "metadata: \n",
      " {'title': ['COVID_62_78y_female_ICU'], 'geo_accession': ['GSM4753081'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 78', 'Sex: female', 'icu: yes', 'apacheii: 22', 'charlson score: 9', 'mechanical ventilation: no', 'ventilator-free days: 0', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 5508', 'crp (mg/l): 138.6', 'ddimer (mg/l_feu): 0.86', 'procalcitonin (ng/ml): 1.02', 'lactate (mmol/l): 1.17', 'fibrinogen: 449', 'sofa: 6', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C62', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934961', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033859'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_62_78y_female_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753081']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 78', 'Sex: female', 'icu: yes', 'apacheii: 22', 'charlson score: 9', 'mechanical ventilation: no', 'ventilator-free days: 0', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 5508', 'crp (mg/l): 138.6', 'ddimer (mg/l_feu): 0.86', 'procalcitonin (ng/ml): 1.02', 'lactate (mmol/l): 1.17', 'fibrinogen: 449', 'sofa: 6', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C62', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934961', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033859']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753082 -- <SAMPLE: GSM4753082>\n",
      "metadata: \n",
      " {'title': ['COVID_63_39y_female_ICU'], 'geo_accession': ['GSM4753082'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 39', 'Sex: female', 'icu: yes', 'apacheii: 14', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 18', 'dm: 0', 'hospital-free days post 45 day followup (days): 20', 'ferritin (ng/ml): 77', 'crp (mg/l): 340.6', 'ddimer (mg/l_feu): 10.48', 'procalcitonin (ng/ml): 0.08', 'lactate (mmol/l): 1.23', 'fibrinogen: 797', 'sofa: 8', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C63', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934960', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033860'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_63_39y_female_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753082']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 39', 'Sex: female', 'icu: yes', 'apacheii: 14', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 18', 'dm: 0', 'hospital-free days post 45 day followup (days): 20', 'ferritin (ng/ml): 77', 'crp (mg/l): 340.6', 'ddimer (mg/l_feu): 10.48', 'procalcitonin (ng/ml): 0.08', 'lactate (mmol/l): 1.23', 'fibrinogen: 797', 'sofa: 8', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C63', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934960', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033860']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753083 -- <SAMPLE: GSM4753083>\n",
      "metadata: \n",
      " {'title': ['COVID_64_65y_male_ICU'], 'geo_accession': ['GSM4753083'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 65', 'Sex: male', 'icu: yes', 'apacheii: 23', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 9', 'dm: 0', 'hospital-free days post 45 day followup (days): 16', 'ferritin (ng/ml): 484', 'crp (mg/l): 8.7', 'ddimer (mg/l_feu): 1.3', 'procalcitonin (ng/ml): 0.79', 'lactate (mmol/l): 0.73', 'fibrinogen: 414', 'sofa: 6', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C64', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934959', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033861'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_64_65y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753083']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 65', 'Sex: male', 'icu: yes', 'apacheii: 23', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 9', 'dm: 0', 'hospital-free days post 45 day followup (days): 16', 'ferritin (ng/ml): 484', 'crp (mg/l): 8.7', 'ddimer (mg/l_feu): 1.3', 'procalcitonin (ng/ml): 0.79', 'lactate (mmol/l): 0.73', 'fibrinogen: 414', 'sofa: 6', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C64', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934959', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033861']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753084 -- <SAMPLE: GSM4753084>\n",
      "metadata: \n",
      " {'title': ['COVID_65_84y_male_NonICU'], 'geo_accession': ['GSM4753084'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 84', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 33', 'ferritin (ng/ml): 220', 'crp (mg/l): 128', 'ddimer (mg/l_feu): 0.4', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 1.47', 'fibrinogen: 797', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C65', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934923', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033862'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_65_84y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753084']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 84', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 33', 'ferritin (ng/ml): 220', 'crp (mg/l): 128', 'ddimer (mg/l_feu): 0.4', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 1.47', 'fibrinogen: 797', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C65', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934923', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033862']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753085 -- <SAMPLE: GSM4753085>\n",
      "metadata: \n",
      " {'title': ['COVID_66_66y_female_NonICU'], 'geo_accession': ['GSM4753085'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 66', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 6', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 22', 'ferritin (ng/ml): 1397', 'crp (mg/l): 8.5', 'ddimer (mg/l_feu): 2.22', 'procalcitonin (ng/ml): 0.16', 'lactate (mmol/l): unknown', 'fibrinogen: 478', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C66', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934922', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033863'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_66_66y_female_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753085']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 66', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 6', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 22', 'ferritin (ng/ml): 1397', 'crp (mg/l): 8.5', 'ddimer (mg/l_feu): 2.22', 'procalcitonin (ng/ml): 0.16', 'lactate (mmol/l): unknown', 'fibrinogen: 478', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C66', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934922', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033863']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753086 -- <SAMPLE: GSM4753086>\n",
      "metadata: \n",
      " {'title': ['COVID_67_57y_male_ICU'], 'geo_accession': ['GSM4753086'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 57', 'Sex: male', 'icu: yes', 'apacheii: 20', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 24', 'dm: 0', 'hospital-free days post 45 day followup (days): 37', 'ferritin (ng/ml): 708', 'crp (mg/l): 191.7', 'ddimer (mg/l_feu): 6.03', 'procalcitonin (ng/ml): 0.19', 'lactate (mmol/l): 0.74', 'fibrinogen: 526', 'sofa: 8', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C67', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934921', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033864'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_67_57y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753086']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 57', 'Sex: male', 'icu: yes', 'apacheii: 20', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 24', 'dm: 0', 'hospital-free days post 45 day followup (days): 37', 'ferritin (ng/ml): 708', 'crp (mg/l): 191.7', 'ddimer (mg/l_feu): 6.03', 'procalcitonin (ng/ml): 0.19', 'lactate (mmol/l): 0.74', 'fibrinogen: 526', 'sofa: 8', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C67', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934921', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033864']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753087 -- <SAMPLE: GSM4753087>\n",
      "metadata: \n",
      " {'title': ['COVID_68_79y_male_ICU'], 'geo_accession': ['GSM4753087'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 79', 'Sex: male', 'icu: yes', 'apacheii: 10', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 17', 'ferritin (ng/ml): 1953', 'crp (mg/l): 269.1', 'ddimer (mg/l_feu): 0.59', 'procalcitonin (ng/ml): 1.31', 'lactate (mmol/l): 0.66', 'fibrinogen: 727', 'sofa: 3', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C68', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934920', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033865'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_68_79y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753087']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 79', 'Sex: male', 'icu: yes', 'apacheii: 10', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 17', 'ferritin (ng/ml): 1953', 'crp (mg/l): 269.1', 'ddimer (mg/l_feu): 0.59', 'procalcitonin (ng/ml): 1.31', 'lactate (mmol/l): 0.66', 'fibrinogen: 727', 'sofa: 3', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C68', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934920', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033865']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753088 -- <SAMPLE: GSM4753088>\n",
      "metadata: \n",
      " {'title': ['COVID_69_77y_female_NonICU'], 'geo_accession': ['GSM4753088'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 77', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 42', 'ferritin (ng/ml): 206', 'crp (mg/l): 124.2', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: 501', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C69', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934919', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033866'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_69_77y_female_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753088']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 77', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 42', 'ferritin (ng/ml): 206', 'crp (mg/l): 124.2', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: 501', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C69', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934919', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033866']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753089 -- <SAMPLE: GSM4753089>\n",
      "metadata: \n",
      " {'title': ['COVID_70_81y_male_NonICU'], 'geo_accession': ['GSM4753089'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 81', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 29', 'ferritin (ng/ml): 900', 'crp (mg/l): 127.7', 'ddimer (mg/l_feu): 0.54', 'procalcitonin (ng/ml): 3.65', 'lactate (mmol/l): 1.05', 'fibrinogen: 489', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C70', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934918', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033867'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_70_81y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753089']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 81', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 29', 'ferritin (ng/ml): 900', 'crp (mg/l): 127.7', 'ddimer (mg/l_feu): 0.54', 'procalcitonin (ng/ml): 3.65', 'lactate (mmol/l): 1.05', 'fibrinogen: 489', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C70', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934918', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033867']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753090 -- <SAMPLE: GSM4753090>\n",
      "metadata: \n",
      " {'title': ['COVID_71_37y_male_ICU'], 'geo_accession': ['GSM4753090'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 37', 'Sex: male', 'icu: yes', 'apacheii: 28', 'charlson score: 0', 'mechanical ventilation: yes', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 21', 'ferritin (ng/ml): 1356', 'crp (mg/l): 128.4', 'ddimer (mg/l_feu): 2.21', 'procalcitonin (ng/ml): 4.97', 'lactate (mmol/l): 0.8', 'fibrinogen: 680', 'sofa: 11', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C71', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934917', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033868'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_71_37y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753090']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 37', 'Sex: male', 'icu: yes', 'apacheii: 28', 'charlson score: 0', 'mechanical ventilation: yes', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 21', 'ferritin (ng/ml): 1356', 'crp (mg/l): 128.4', 'ddimer (mg/l_feu): 2.21', 'procalcitonin (ng/ml): 4.97', 'lactate (mmol/l): 0.8', 'fibrinogen: 680', 'sofa: 11', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C71', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934917', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033868']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753091 -- <SAMPLE: GSM4753091>\n",
      "metadata: \n",
      " {'title': ['COVID_72_50y_female_NonICU'], 'geo_accession': ['GSM4753091'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 50', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 39', 'ferritin (ng/ml): 14', 'crp (mg/l): 23.6', 'ddimer (mg/l_feu): 0.61', 'procalcitonin (ng/ml): 0.5', 'lactate (mmol/l): unknown', 'fibrinogen: 406', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C72', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934916', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033869'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_72_50y_female_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753091']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 50', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 39', 'ferritin (ng/ml): 14', 'crp (mg/l): 23.6', 'ddimer (mg/l_feu): 0.61', 'procalcitonin (ng/ml): 0.5', 'lactate (mmol/l): unknown', 'fibrinogen: 406', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C72', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934916', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033869']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753092 -- <SAMPLE: GSM4753092>\n",
      "metadata: \n",
      " {'title': ['COVID_73_82y_male_NonICU'], 'geo_accession': ['GSM4753092'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 82', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 31', 'ferritin (ng/ml): 629', 'crp (mg/l): 14.3', 'ddimer (mg/l_feu): 0.92', 'procalcitonin (ng/ml): 0.05', 'lactate (mmol/l): unknown', 'fibrinogen: 362', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C73', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935011', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033870'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_73_82y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753092']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 82', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 31', 'ferritin (ng/ml): 629', 'crp (mg/l): 14.3', 'ddimer (mg/l_feu): 0.92', 'procalcitonin (ng/ml): 0.05', 'lactate (mmol/l): unknown', 'fibrinogen: 362', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C73', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935011', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033870']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753093 -- <SAMPLE: GSM4753093>\n",
      "metadata: \n",
      " {'title': ['COVID_74_55y_female_ICU'], 'geo_accession': ['GSM4753093'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 55', 'Sex: female', 'icu: yes', 'apacheii: 22', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 18', 'dm: 1', 'hospital-free days post 45 day followup (days): 30', 'ferritin (ng/ml): 186', 'crp (mg/l): 25.3', 'ddimer (mg/l_feu): 6.53', 'procalcitonin (ng/ml): 1.88', 'lactate (mmol/l): 1.33', 'fibrinogen: 312', 'sofa: 11', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C74', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935010', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033871'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_74_55y_female_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753093']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 55', 'Sex: female', 'icu: yes', 'apacheii: 22', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 18', 'dm: 1', 'hospital-free days post 45 day followup (days): 30', 'ferritin (ng/ml): 186', 'crp (mg/l): 25.3', 'ddimer (mg/l_feu): 6.53', 'procalcitonin (ng/ml): 1.88', 'lactate (mmol/l): 1.33', 'fibrinogen: 312', 'sofa: 11', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C74', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935010', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033871']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753094 -- <SAMPLE: GSM4753094>\n",
      "metadata: \n",
      " {'title': ['COVID_75_55y_male_NonICU'], 'geo_accession': ['GSM4753094'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 55', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 39', 'ferritin (ng/ml): 5971', 'crp (mg/l): 112.8', 'ddimer (mg/l_feu): 0.66', 'procalcitonin (ng/ml): 0.21', 'lactate (mmol/l): unknown', 'fibrinogen: 501', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C75', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935009', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033872'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_75_55y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753094']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 55', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 39', 'ferritin (ng/ml): 5971', 'crp (mg/l): 112.8', 'ddimer (mg/l_feu): 0.66', 'procalcitonin (ng/ml): 0.21', 'lactate (mmol/l): unknown', 'fibrinogen: 501', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C75', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935009', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033872']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753095 -- <SAMPLE: GSM4753095>\n",
      "metadata: \n",
      " {'title': ['COVID_76_73y_female_ICU'], 'geo_accession': ['GSM4753095'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 73', 'Sex: female', 'icu: yes', 'apacheii: 35', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 685', 'crp (mg/l): 326.6', 'ddimer (mg/l_feu): 21.78', 'procalcitonin (ng/ml): 16.51', 'lactate (mmol/l): unknown', 'fibrinogen: 490', 'sofa: 19', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C76', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935008', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033873'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_76_73y_female_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753095']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 73', 'Sex: female', 'icu: yes', 'apacheii: 35', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 685', 'crp (mg/l): 326.6', 'ddimer (mg/l_feu): 21.78', 'procalcitonin (ng/ml): 16.51', 'lactate (mmol/l): unknown', 'fibrinogen: 490', 'sofa: 19', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C76', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935008', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033873']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753096 -- <SAMPLE: GSM4753096>\n",
      "metadata: \n",
      " {'title': ['COVID_77_55y_female_ICU'], 'geo_accession': ['GSM4753096'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 55', 'Sex: female', 'icu: yes', 'apacheii: 6', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 26', 'ferritin (ng/ml): 1223', 'crp (mg/l): 84.5', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: 2', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C77', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935007', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033874'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_77_55y_female_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753096']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 55', 'Sex: female', 'icu: yes', 'apacheii: 6', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 26', 'ferritin (ng/ml): 1223', 'crp (mg/l): 84.5', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: 2', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C77', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935007', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033874']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753097 -- <SAMPLE: GSM4753097>\n",
      "metadata: \n",
      " {'title': ['COVID_78_80y_male_NonICU'], 'geo_accession': ['GSM4753097'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 80', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 31', 'ferritin (ng/ml): 635', 'crp (mg/l): 430.5', 'ddimer (mg/l_feu): 2.56', 'procalcitonin (ng/ml): 0.36', 'lactate (mmol/l): 1.26', 'fibrinogen: 639', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C78', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935006', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033875'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_78_80y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753097']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 80', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 31', 'ferritin (ng/ml): 635', 'crp (mg/l): 430.5', 'ddimer (mg/l_feu): 2.56', 'procalcitonin (ng/ml): 0.36', 'lactate (mmol/l): 1.26', 'fibrinogen: 639', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C78', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935006', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033875']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753098 -- <SAMPLE: GSM4753098>\n",
      "metadata: \n",
      " {'title': ['COVID_79_27y_male_NonICU'], 'geo_accession': ['GSM4753098'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 27', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 40', 'ferritin (ng/ml): 112', 'crp (mg/l): 1', 'ddimer (mg/l_feu): 0.8', 'procalcitonin (ng/ml): 0.13', 'lactate (mmol/l): unknown', 'fibrinogen: 215', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C79', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935005', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033876'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " title ['COVID_79_27y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753098']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 27', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 40', 'ferritin (ng/ml): 112', 'crp (mg/l): 1', 'ddimer (mg/l_feu): 0.8', 'procalcitonin (ng/ml): 0.13', 'lactate (mmol/l): unknown', 'fibrinogen: 215', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C79', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935005', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033876']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753099 -- <SAMPLE: GSM4753099>\n",
      "metadata: \n",
      " {'title': ['COVID_80_71y_male_ICU'], 'geo_accession': ['GSM4753099'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 71', 'Sex: male', 'icu: yes', 'apacheii: 22', 'charlson score: 7', 'mechanical ventilation: no', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1200', 'crp (mg/l): 122.5', 'ddimer (mg/l_feu): 1.7', 'procalcitonin (ng/ml): 0.7', 'lactate (mmol/l): 1.88', 'fibrinogen: 489', 'sofa: 4', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C80', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935004', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033877'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_80_71y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753099']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 71', 'Sex: male', 'icu: yes', 'apacheii: 22', 'charlson score: 7', 'mechanical ventilation: no', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1200', 'crp (mg/l): 122.5', 'ddimer (mg/l_feu): 1.7', 'procalcitonin (ng/ml): 0.7', 'lactate (mmol/l): 1.88', 'fibrinogen: 489', 'sofa: 4', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C80', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935004', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033877']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753100 -- <SAMPLE: GSM4753100>\n",
      "metadata: \n",
      " {'title': ['COVID_82_67y_male_NonICU'], 'geo_accession': ['GSM4753100'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 67', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 41', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C82', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935003', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033878'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_82_67y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753100']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 67', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 41', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C82', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935003', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033878']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753101 -- <SAMPLE: GSM4753101>\n",
      "metadata: \n",
      " {'title': ['COVID_83_85y_female_NonICU'], 'geo_accession': ['GSM4753101'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 85', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 133', 'crp (mg/l): 97.6', 'ddimer (mg/l_feu): 0.85', 'procalcitonin (ng/ml): 0.5', 'lactate (mmol/l): 0.84', 'fibrinogen: 296', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C83', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935002', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033879'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_83_85y_female_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753101']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 85', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 133', 'crp (mg/l): 97.6', 'ddimer (mg/l_feu): 0.85', 'procalcitonin (ng/ml): 0.5', 'lactate (mmol/l): 0.84', 'fibrinogen: 296', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C83', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935002', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033879']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753102 -- <SAMPLE: GSM4753102>\n",
      "metadata: \n",
      " {'title': ['COVID_84_75y_female_NonICU'], 'geo_accession': ['GSM4753102'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 75', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 6', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 19', 'ferritin (ng/ml): 840', 'crp (mg/l): 153.5', 'ddimer (mg/l_feu): 4.2', 'procalcitonin (ng/ml): 2.66', 'lactate (mmol/l): 0.83', 'fibrinogen: 292', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C84', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935001', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033880'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_84_75y_female_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753102']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 75', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 6', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 19', 'ferritin (ng/ml): 840', 'crp (mg/l): 153.5', 'ddimer (mg/l_feu): 4.2', 'procalcitonin (ng/ml): 2.66', 'lactate (mmol/l): 0.83', 'fibrinogen: 292', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C84', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935001', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033880']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753103 -- <SAMPLE: GSM4753103>\n",
      "metadata: \n",
      " {'title': ['COVID_85_62y_male_ICU'], 'geo_accession': ['GSM4753103'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 62', 'Sex: male', 'icu: yes', 'apacheii: 12', 'charlson score: 3', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 22', 'ferritin (ng/ml): 423', 'crp (mg/l): 54.9', 'ddimer (mg/l_feu): 4.83', 'procalcitonin (ng/ml): 0.11', 'lactate (mmol/l): 0.5', 'fibrinogen: 396', 'sofa: 3', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C85', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935000', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033881'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_85_62y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753103']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 62', 'Sex: male', 'icu: yes', 'apacheii: 12', 'charlson score: 3', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 22', 'ferritin (ng/ml): 423', 'crp (mg/l): 54.9', 'ddimer (mg/l_feu): 4.83', 'procalcitonin (ng/ml): 0.11', 'lactate (mmol/l): 0.5', 'fibrinogen: 396', 'sofa: 3', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C85', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935000', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033881']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753104 -- <SAMPLE: GSM4753104>\n",
      "metadata: \n",
      " {'title': ['COVID_86_52y_female_NonICU'], 'geo_accession': ['GSM4753104'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 52', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 42', 'ferritin (ng/ml): 352', 'crp (mg/l): 145.7', 'ddimer (mg/l_feu): 0.22', 'procalcitonin (ng/ml): 0.05', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C86', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934999', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033882'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_86_52y_female_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753104']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 52', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 42', 'ferritin (ng/ml): 352', 'crp (mg/l): 145.7', 'ddimer (mg/l_feu): 0.22', 'procalcitonin (ng/ml): 0.05', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C86', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934999', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033882']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753105 -- <SAMPLE: GSM4753105>\n",
      "metadata: \n",
      " {'title': ['COVID_87_61y_male_ICU'], 'geo_accession': ['GSM4753105'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 61', 'Sex: male', 'icu: yes', 'apacheii: 7', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 39', 'ferritin (ng/ml): 170', 'crp (mg/l): 2.8', 'ddimer (mg/l_feu): 1.55', 'procalcitonin (ng/ml): 0.5', 'lactate (mmol/l): 1.13', 'fibrinogen: 208', 'sofa: 2', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C87', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934998', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033883'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_87_61y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753105']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 61', 'Sex: male', 'icu: yes', 'apacheii: 7', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 39', 'ferritin (ng/ml): 170', 'crp (mg/l): 2.8', 'ddimer (mg/l_feu): 1.55', 'procalcitonin (ng/ml): 0.5', 'lactate (mmol/l): 1.13', 'fibrinogen: 208', 'sofa: 2', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C87', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934998', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033883']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753106 -- <SAMPLE: GSM4753106>\n",
      "metadata: \n",
      " {'title': ['COVID_89_90y_female_NonICU'], 'geo_accession': ['GSM4753106'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): >89', 'Sex: female', 'icu: no', 'apacheii: 17', 'charlson score: 7', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 33', 'ferritin (ng/ml): 38', 'crp (mg/l): 4.7', 'ddimer (mg/l_feu): 0.96', 'procalcitonin (ng/ml): 0.5', 'lactate (mmol/l): 1.09', 'fibrinogen: 292', 'sofa: 3', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C89', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934997', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033884'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_89_90y_female_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753106']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): >89', 'Sex: female', 'icu: no', 'apacheii: 17', 'charlson score: 7', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 33', 'ferritin (ng/ml): 38', 'crp (mg/l): 4.7', 'ddimer (mg/l_feu): 0.96', 'procalcitonin (ng/ml): 0.5', 'lactate (mmol/l): 1.09', 'fibrinogen: 292', 'sofa: 3', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C89', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934997', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033884']\n",
      "gsm.metadata.items \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753107 -- <SAMPLE: GSM4753107>\n",
      "metadata: \n",
      " {'title': ['COVID_90_86y_female_NonICU'], 'geo_accession': ['GSM4753107'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 86', 'Sex: female', 'icu: no', 'apacheii: 11', 'charlson score: 8', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 24', 'ferritin (ng/ml): 288', 'crp (mg/l): 43.7', 'ddimer (mg/l_feu): 15.48', 'procalcitonin (ng/ml): 0.06', 'lactate (mmol/l): unknown', 'fibrinogen: 635', 'sofa: 2', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C90', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934996', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033885'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_90_86y_female_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753107']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 86', 'Sex: female', 'icu: no', 'apacheii: 11', 'charlson score: 8', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 24', 'ferritin (ng/ml): 288', 'crp (mg/l): 43.7', 'ddimer (mg/l_feu): 15.48', 'procalcitonin (ng/ml): 0.06', 'lactate (mmol/l): unknown', 'fibrinogen: 635', 'sofa: 2', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C90', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934996', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033885']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753108 -- <SAMPLE: GSM4753108>\n",
      "metadata: \n",
      " {'title': ['COVID_91_29y_female_NonICU'], 'geo_accession': ['GSM4753108'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 29', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 40', 'ferritin (ng/ml): unknown', 'crp (mg/l): 1.8', 'ddimer (mg/l_feu): 0.28', 'procalcitonin (ng/ml): 2.5', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C91', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934995', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033886'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_91_29y_female_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753108']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 29', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 40', 'ferritin (ng/ml): unknown', 'crp (mg/l): 1.8', 'ddimer (mg/l_feu): 0.28', 'procalcitonin (ng/ml): 2.5', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C91', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934995', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033886']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753109 -- <SAMPLE: GSM4753109>\n",
      "metadata: \n",
      " {'title': ['COVID_92_82y_female_ICU'], 'geo_accession': ['GSM4753109'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 82', 'Sex: female', 'icu: yes', 'apacheii: 23', 'charlson score: 7', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 111', 'crp (mg/l): 26.6', 'ddimer (mg/l_feu): 1.22', 'procalcitonin (ng/ml): 0.1', 'lactate (mmol/l): 0.97', 'fibrinogen: 489', 'sofa: 8', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C92', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934994', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033887'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_92_82y_female_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753109']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 82', 'Sex: female', 'icu: yes', 'apacheii: 23', 'charlson score: 7', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 111', 'crp (mg/l): 26.6', 'ddimer (mg/l_feu): 1.22', 'procalcitonin (ng/ml): 0.1', 'lactate (mmol/l): 0.97', 'fibrinogen: 489', 'sofa: 8', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C92', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934994', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033887']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753110 -- <SAMPLE: GSM4753110>\n",
      "metadata: \n",
      " {'title': ['COVID_93_81y_female_ICU'], 'geo_accession': ['GSM4753110'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 81', 'Sex: female', 'icu: yes', 'apacheii: 14', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 0', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 388', 'crp (mg/l): 17.2', 'ddimer (mg/l_feu): 1.25', 'procalcitonin (ng/ml): 0.07', 'lactate (mmol/l): 1.88', 'fibrinogen: 289', 'sofa: 6', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C93', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934993', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033888'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_93_81y_female_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753110']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 81', 'Sex: female', 'icu: yes', 'apacheii: 14', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 0', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 388', 'crp (mg/l): 17.2', 'ddimer (mg/l_feu): 1.25', 'procalcitonin (ng/ml): 0.07', 'lactate (mmol/l): 1.88', 'fibrinogen: 289', 'sofa: 6', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C93', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934993', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033888']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753111 -- <SAMPLE: GSM4753111>\n",
      "metadata: \n",
      " {'title': ['COVID_94_24y_female_NonICU'], 'geo_accession': ['GSM4753111'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 24', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 43', 'ferritin (ng/ml): 52', 'crp (mg/l): 39', 'ddimer (mg/l_feu): 0.67', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: 412', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C94', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935041', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033889'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_94_24y_female_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753111']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 24', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 43', 'ferritin (ng/ml): 52', 'crp (mg/l): 39', 'ddimer (mg/l_feu): 0.67', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: 412', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C94', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935041', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033889']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753112 -- <SAMPLE: GSM4753112>\n",
      "metadata: \n",
      " {'title': ['COVID_95_49y_male_NonICU'], 'geo_accession': ['GSM4753112'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 49', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 33', 'ferritin (ng/ml): 386', 'crp (mg/l): 193.6', 'ddimer (mg/l_feu): 1.3', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: 752', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C95', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935040', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033890'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_95_49y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753112']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 49', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 33', 'ferritin (ng/ml): 386', 'crp (mg/l): 193.6', 'ddimer (mg/l_feu): 1.3', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: 752', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C95', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935040', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033890']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753113 -- <SAMPLE: GSM4753113>\n",
      "metadata: \n",
      " {'title': ['COVID_96_51y_male_NonICU'], 'geo_accession': ['GSM4753113'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 51', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 3', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 41', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C96', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935039', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033891'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_96_51y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753113']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 51', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 3', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 41', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C96', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935039', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033891']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753114 -- <SAMPLE: GSM4753114>\n",
      "metadata: \n",
      " {'title': ['COVID_97_76y_male_ICU'], 'geo_accession': ['GSM4753114'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 76', 'Sex: male', 'icu: yes', 'apacheii: 18', 'charlson score: 9', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 735', 'crp (mg/l): 185', 'ddimer (mg/l_feu): 2.06', 'procalcitonin (ng/ml): 1.06', 'lactate (mmol/l): 1.22', 'fibrinogen: 585', 'sofa: 10', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C97', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935038', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033892'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_97_76y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753114']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 76', 'Sex: male', 'icu: yes', 'apacheii: 18', 'charlson score: 9', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 735', 'crp (mg/l): 185', 'ddimer (mg/l_feu): 2.06', 'procalcitonin (ng/ml): 1.06', 'lactate (mmol/l): 1.22', 'fibrinogen: 585', 'sofa: 10', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C97', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935038', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033892']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753115 -- <SAMPLE: GSM4753115>\n",
      "metadata: \n",
      " {'title': ['COVID_98_81y_male_NonICU'], 'geo_accession': ['GSM4753115'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 81', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 8', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 15', 'ferritin (ng/ml): 582', 'crp (mg/l): 108.6', 'ddimer (mg/l_feu): 1.54', 'procalcitonin (ng/ml): 0.36', 'lactate (mmol/l): 3.28', 'fibrinogen: 468', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C98', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935037', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033893'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_98_81y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753115']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 81', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 8', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 15', 'ferritin (ng/ml): 582', 'crp (mg/l): 108.6', 'ddimer (mg/l_feu): 1.54', 'procalcitonin (ng/ml): 0.36', 'lactate (mmol/l): 3.28', 'fibrinogen: 468', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C98', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935037', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033893']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753116 -- <SAMPLE: GSM4753116>\n",
      "metadata: \n",
      " {'title': ['COVID_99_71y_male_ICU'], 'geo_accession': ['GSM4753116'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 71', 'Sex: male', 'icu: yes', 'apacheii: 12', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1231', 'crp (mg/l): 305.5', 'ddimer (mg/l_feu): 3.59', 'procalcitonin (ng/ml): 1.5', 'lactate (mmol/l): 1.18', 'fibrinogen: 949', 'sofa: 4', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C99', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935036', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033894'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_99_71y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753116']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 71', 'Sex: male', 'icu: yes', 'apacheii: 12', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1231', 'crp (mg/l): 305.5', 'ddimer (mg/l_feu): 3.59', 'procalcitonin (ng/ml): 1.5', 'lactate (mmol/l): 1.18', 'fibrinogen: 949', 'sofa: 4', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C99', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935036', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033894']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753117 -- <SAMPLE: GSM4753117>\n",
      "metadata: \n",
      " {'title': ['COVID_100_74y_female_NonICU'], 'geo_accession': ['GSM4753117'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 74', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 6', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 35', 'ferritin (ng/ml): 304', 'crp (mg/l): 136.4', 'ddimer (mg/l_feu): 1.67', 'procalcitonin (ng/ml): 0.05', 'lactate (mmol/l): unknown', 'fibrinogen: 583', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C100', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935035', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033895'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_100_74y_female_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753117']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 74', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 6', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 35', 'ferritin (ng/ml): 304', 'crp (mg/l): 136.4', 'ddimer (mg/l_feu): 1.67', 'procalcitonin (ng/ml): 0.05', 'lactate (mmol/l): unknown', 'fibrinogen: 583', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C100', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935035', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033895']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753118 -- <SAMPLE: GSM4753118>\n",
      "metadata: \n",
      " {'title': ['COVID_101_58y_male_ICU'], 'geo_accession': ['GSM4753118'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 58', 'Sex: male', 'icu: yes', 'apacheii: 14', 'charlson score: 3', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 37', 'ferritin (ng/ml): 602', 'crp (mg/l): 100.7', 'ddimer (mg/l_feu): 1.72', 'procalcitonin (ng/ml): 0.2', 'lactate (mmol/l): 2.43', 'fibrinogen: 479', 'sofa: 6', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C101', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935034', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033896'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_101_58y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753118']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 58', 'Sex: male', 'icu: yes', 'apacheii: 14', 'charlson score: 3', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 37', 'ferritin (ng/ml): 602', 'crp (mg/l): 100.7', 'ddimer (mg/l_feu): 1.72', 'procalcitonin (ng/ml): 0.2', 'lactate (mmol/l): 2.43', 'fibrinogen: 479', 'sofa: 6', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C101', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935034', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033896']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753119 -- <SAMPLE: GSM4753119>\n",
      "metadata: \n",
      " {'title': ['COVID_102_84y_male_NonICU'], 'geo_accession': ['GSM4753119'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 84', 'Sex: male', 'icu: no', 'apacheii: 23', 'charlson score: 7', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: unknown', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C102', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935033', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033897'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_102_84y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753119']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 84', 'Sex: male', 'icu: no', 'apacheii: 23', 'charlson score: 7', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: unknown', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C102', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935033', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033897']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753120 -- <SAMPLE: GSM4753120>\n",
      "metadata: \n",
      " {'title': ['COVID_103_83y_male_NonICU'], 'geo_accession': ['GSM4753120'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 83', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 669', 'crp (mg/l): 214.2', 'ddimer (mg/l_feu): 0.75', 'procalcitonin (ng/ml): 0.17', 'lactate (mmol/l): 1.58', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C103', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935032', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033898'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_103_83y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753120']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 83', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 669', 'crp (mg/l): 214.2', 'ddimer (mg/l_feu): 0.75', 'procalcitonin (ng/ml): 0.17', 'lactate (mmol/l): 1.58', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['C103', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935032', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033898']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753121 -- <SAMPLE: GSM4753121>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_01_54y_female_NonICU'], 'geo_accession': ['GSM4753121'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 54', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 35', 'ferritin (ng/ml): 18', 'crp (mg/l): 20', 'ddimer (mg/l_feu): 0.46', 'procalcitonin (ng/ml): 0.5', 'lactate (mmol/l): unknown', 'fibrinogen: 261', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC1', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935031', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033899'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_01_54y_female_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753121']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 54', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 35', 'ferritin (ng/ml): 18', 'crp (mg/l): 20', 'ddimer (mg/l_feu): 0.46', 'procalcitonin (ng/ml): 0.5', 'lactate (mmol/l): unknown', 'fibrinogen: 261', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['NC1', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935031', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033899']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753122 -- <SAMPLE: GSM4753122>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_02_65y_male_ICU'], 'geo_accession': ['GSM4753122'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 65', 'Sex: male', 'icu: yes', 'apacheii: 15', 'charlson score: 6', 'mechanical ventilation: yes', 'ventilator-free days: 18', 'dm: 0', 'hospital-free days post 45 day followup (days): 29', 'ferritin (ng/ml): 297', 'crp (mg/l): 47.8', 'ddimer (mg/l_feu): 17.11', 'procalcitonin (ng/ml): 0.41', 'lactate (mmol/l): 3.68', 'fibrinogen: 267', 'sofa: 12', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC2', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935030', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033900'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_02_65y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753122']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 65', 'Sex: male', 'icu: yes', 'apacheii: 15', 'charlson score: 6', 'mechanical ventilation: yes', 'ventilator-free days: 18', 'dm: 0', 'hospital-free days post 45 day followup (days): 29', 'ferritin (ng/ml): 297', 'crp (mg/l): 47.8', 'ddimer (mg/l_feu): 17.11', 'procalcitonin (ng/ml): 0.41', 'lactate (mmol/l): 3.68', 'fibrinogen: 267', 'sofa: 12', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['NC2', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935030', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033900']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753123 -- <SAMPLE: GSM4753123>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_03_65y_male_ICU'], 'geo_accession': ['GSM4753123'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 65', 'Sex: male', 'icu: yes', 'apacheii: 24', 'charlson score: 7', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 809', 'crp (mg/l): 226.3', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 1.69', 'lactate (mmol/l): 1.88', 'fibrinogen: unknown', 'sofa: 9', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC3', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935029', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033901'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_03_65y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753123']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 65', 'Sex: male', 'icu: yes', 'apacheii: 24', 'charlson score: 7', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 809', 'crp (mg/l): 226.3', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 1.69', 'lactate (mmol/l): 1.88', 'fibrinogen: unknown', 'sofa: 9', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['NC3', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935029', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033901']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753124 -- <SAMPLE: GSM4753124>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_04_90y_male_NonICU'], 'geo_accession': ['GSM4753124'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): >89', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 41', 'ferritin (ng/ml): 58', 'crp (mg/l): 9.3', 'ddimer (mg/l_feu): 34.4', 'procalcitonin (ng/ml): 0.43', 'lactate (mmol/l): 1.51', 'fibrinogen: 258', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC4', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935028', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033902'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_04_90y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753124']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): >89', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 41', 'ferritin (ng/ml): 58', 'crp (mg/l): 9.3', 'ddimer (mg/l_feu): 34.4', 'procalcitonin (ng/ml): 0.43', 'lactate (mmol/l): 1.51', 'fibrinogen: 258', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['NC4', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935028', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033902']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753125 -- <SAMPLE: GSM4753125>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_05_83y_female_NonICU'], 'geo_accession': ['GSM4753125'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 83', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 43', 'ferritin (ng/ml): 129', 'crp (mg/l): 104.3', 'ddimer (mg/l_feu): 1.92', 'procalcitonin (ng/ml): 11.37', 'lactate (mmol/l): 1.27', 'fibrinogen: 253', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC5', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935027', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033903'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_05_83y_female_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753125']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 83', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 43', 'ferritin (ng/ml): 129', 'crp (mg/l): 104.3', 'ddimer (mg/l_feu): 1.92', 'procalcitonin (ng/ml): 11.37', 'lactate (mmol/l): 1.27', 'fibrinogen: 253', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['NC5', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935027', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033903']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753126 -- <SAMPLE: GSM4753126>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_06_75y_female_ICU'], 'geo_accession': ['GSM4753126'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 75', 'Sex: female', 'icu: yes', 'apacheii: 12', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 580', 'crp (mg/l): 200.2', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 14.36', 'lactate (mmol/l): 2', 'fibrinogen: unknown', 'sofa: 2', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC6', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935026', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033904'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_06_75y_female_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753126']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 75', 'Sex: female', 'icu: yes', 'apacheii: 12', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 580', 'crp (mg/l): 200.2', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 14.36', 'lactate (mmol/l): 2', 'fibrinogen: unknown', 'sofa: 2', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['NC6', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935026', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033904']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753127 -- <SAMPLE: GSM4753127>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_07_50y_male_ICU'], 'geo_accession': ['GSM4753127'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 50', 'Sex: male', 'icu: yes', 'apacheii: 26', 'charlson score: 7', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 42', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 3.02', 'fibrinogen: unknown', 'sofa: 11', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC7', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935025', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033905'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_07_50y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753127']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 50', 'Sex: male', 'icu: yes', 'apacheii: 26', 'charlson score: 7', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 42', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 3.02', 'fibrinogen: unknown', 'sofa: 11', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['NC7', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935025', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033905']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753128 -- <SAMPLE: GSM4753128>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_08_53y_female_ICU'], 'geo_accession': ['GSM4753128'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 53', 'Sex: female', 'icu: yes', 'apacheii: 22', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 24', 'dm: 0', 'hospital-free days post 45 day followup (days): 38', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 0.82', 'fibrinogen: 585', 'sofa: 5', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC8', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935024', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033906'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_08_53y_female_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753128']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 53', 'Sex: female', 'icu: yes', 'apacheii: 22', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 24', 'dm: 0', 'hospital-free days post 45 day followup (days): 38', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 0.82', 'fibrinogen: 585', 'sofa: 5', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['NC8', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935024', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033906']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753129 -- <SAMPLE: GSM4753129>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_09_49y_female_NonICU'], 'geo_accession': ['GSM4753129'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 49', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 44', 'ferritin (ng/ml): 411', 'crp (mg/l): 7.7', 'ddimer (mg/l_feu): 1.03', 'procalcitonin (ng/ml): 0.07', 'lactate (mmol/l): 0.79', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC9', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935023', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033907'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_09_49y_female_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753129']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 49', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 44', 'ferritin (ng/ml): 411', 'crp (mg/l): 7.7', 'ddimer (mg/l_feu): 1.03', 'procalcitonin (ng/ml): 0.07', 'lactate (mmol/l): 0.79', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['NC9', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935023', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033907']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753130 -- <SAMPLE: GSM4753130>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_10_67y_male_ICU'], 'geo_accession': ['GSM4753130'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 67', 'Sex: male', 'icu: yes', 'apacheii: 20', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 25', 'dm: 0', 'hospital-free days post 45 day followup (days): 38', 'ferritin (ng/ml): 79', 'crp (mg/l): 110.2', 'ddimer (mg/l_feu): 0.33', 'procalcitonin (ng/ml): 0.2', 'lactate (mmol/l): 0.61', 'fibrinogen: unknown', 'sofa: 10', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC10', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935022', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033908'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_10_67y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753130']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 67', 'Sex: male', 'icu: yes', 'apacheii: 20', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 25', 'dm: 0', 'hospital-free days post 45 day followup (days): 38', 'ferritin (ng/ml): 79', 'crp (mg/l): 110.2', 'ddimer (mg/l_feu): 0.33', 'procalcitonin (ng/ml): 0.2', 'lactate (mmol/l): 0.61', 'fibrinogen: unknown', 'sofa: 10', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['NC10', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935022', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033908']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753131 -- <SAMPLE: GSM4753131>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_11_58y_female_NonICU'], 'geo_accession': ['GSM4753131'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 58', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 6', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 41', 'ferritin (ng/ml): 630', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): 0.37', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: 304', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC11', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935021', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033909'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_11_58y_female_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753131']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 58', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 6', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 41', 'ferritin (ng/ml): 630', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): 0.37', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: 304', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['NC11', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935021', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033909']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753132 -- <SAMPLE: GSM4753132>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_12_82y_male_ICU'], 'geo_accession': ['GSM4753132'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 82', 'Sex: male', 'icu: yes', 'apacheii: 12', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 42', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 2.31', 'fibrinogen: unknown', 'sofa: 10', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC12', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935020', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033910'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_12_82y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753132']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 82', 'Sex: male', 'icu: yes', 'apacheii: 12', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 42', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 2.31', 'fibrinogen: unknown', 'sofa: 10', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['NC12', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935020', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033910']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753133 -- <SAMPLE: GSM4753133>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_13_65y_male_ICU'], 'geo_accession': ['GSM4753133'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 65', 'Sex: male', 'icu: yes', 'apacheii: 30', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 26', 'dm: 1', 'hospital-free days post 45 day followup (days): 35', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 3.35', 'fibrinogen: unknown', 'sofa: 11', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC13', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935019', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033911'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_13_65y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753133']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 65', 'Sex: male', 'icu: yes', 'apacheii: 30', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 26', 'dm: 1', 'hospital-free days post 45 day followup (days): 35', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 3.35', 'fibrinogen: unknown', 'sofa: 11', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['NC13', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935019', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033911']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753134 -- <SAMPLE: GSM4753134>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_14_75y_female_ICU'], 'geo_accession': ['GSM4753134'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 75', 'Sex: female', 'icu: yes', 'apacheii: 26', 'charlson score: 5', 'mechanical ventilation: yes', 'ventilator-free days: 24', 'dm: 0', 'hospital-free days post 45 day followup (days): 36', 'ferritin (ng/ml): 289', 'crp (mg/l): 41.5', 'ddimer (mg/l_feu): 2.29', 'procalcitonin (ng/ml): 0.34', 'lactate (mmol/l): 4.22', 'fibrinogen: 255', 'sofa: 11', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC14', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935018', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033912'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_14_75y_female_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753134']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 75', 'Sex: female', 'icu: yes', 'apacheii: 26', 'charlson score: 5', 'mechanical ventilation: yes', 'ventilator-free days: 24', 'dm: 0', 'hospital-free days post 45 day followup (days): 36', 'ferritin (ng/ml): 289', 'crp (mg/l): 41.5', 'ddimer (mg/l_feu): 2.29', 'procalcitonin (ng/ml): 0.34', 'lactate (mmol/l): 4.22', 'fibrinogen: 255', 'sofa: 11', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['NC14', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935018', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033912']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753135 -- <SAMPLE: GSM4753135>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_15_83y_unknown_ICU'], 'geo_accession': ['GSM4753135'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 83', 'Sex: unknown', 'icu: yes', 'apacheii: 11', 'charlson score: 8', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 30', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 0.59', 'fibrinogen: unknown', 'sofa: 1', 'associated controlled-vocabulary terms: DOID:4 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC15', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935017', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033913'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_15_83y_unknown_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753135']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 83', 'Sex: unknown', 'icu: yes', 'apacheii: 11', 'charlson score: 8', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 30', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 0.59', 'fibrinogen: unknown', 'sofa: 1', 'associated controlled-vocabulary terms: DOID:4 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['NC15', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935017', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033913']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753136 -- <SAMPLE: GSM4753136>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_16_40y_female_ICU'], 'geo_accession': ['GSM4753136'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 40', 'Sex: female', 'icu: yes', 'apacheii: 13', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 29', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): 4.56', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 1.49', 'fibrinogen: 583', 'sofa: 10', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC16', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935016', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033914'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_16_40y_female_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753136']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 40', 'Sex: female', 'icu: yes', 'apacheii: 13', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 29', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): 4.56', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 1.49', 'fibrinogen: 583', 'sofa: 10', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['NC16', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935016', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033914']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753137 -- <SAMPLE: GSM4753137>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_17_84y_female_ICU'], 'geo_accession': ['GSM4753137'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 84', 'Sex: female', 'icu: yes', 'apacheii: 11', 'charlson score: 8', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 41', 'ferritin (ng/ml): 105', 'crp (mg/l): 85.9', 'ddimer (mg/l_feu): 0.6', 'procalcitonin (ng/ml): 0.73', 'lactate (mmol/l): 0.95', 'fibrinogen: 539', 'sofa: 3', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC17', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935015', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033915'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_17_84y_female_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753137']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 84', 'Sex: female', 'icu: yes', 'apacheii: 11', 'charlson score: 8', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 41', 'ferritin (ng/ml): 105', 'crp (mg/l): 85.9', 'ddimer (mg/l_feu): 0.6', 'procalcitonin (ng/ml): 0.73', 'lactate (mmol/l): 0.95', 'fibrinogen: 539', 'sofa: 3', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['NC17', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935015', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033915']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753138 -- <SAMPLE: GSM4753138>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_18_88y_male_ICU'], 'geo_accession': ['GSM4753138'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 88', 'Sex: male', 'icu: yes', 'apacheii: 23', 'charlson score: 8', 'mechanical ventilation: yes', 'ventilator-free days: 25', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 211', 'crp (mg/l): 34', 'ddimer (mg/l_feu): 10.21', 'procalcitonin (ng/ml): 0.06', 'lactate (mmol/l): 1.15', 'fibrinogen: 300', 'sofa: 9', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC18', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935014', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033916'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_18_88y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753138']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 88', 'Sex: male', 'icu: yes', 'apacheii: 23', 'charlson score: 8', 'mechanical ventilation: yes', 'ventilator-free days: 25', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 211', 'crp (mg/l): 34', 'ddimer (mg/l_feu): 10.21', 'procalcitonin (ng/ml): 0.06', 'lactate (mmol/l): 1.15', 'fibrinogen: 300', 'sofa: 9', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['NC18', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935014', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033916']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753139 -- <SAMPLE: GSM4753139>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_19_66y_female_ICU'], 'geo_accession': ['GSM4753139'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 66', 'Sex: female', 'icu: yes', 'apacheii: 27', 'charlson score: 4', 'mechanical ventilation: yes', 'ventilator-free days: 21', 'dm: 0', 'hospital-free days post 45 day followup (days): 29', 'ferritin (ng/ml): 155', 'crp (mg/l): 150.2', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 0.35', 'lactate (mmol/l): 2.03', 'fibrinogen: 191', 'sofa: 15', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC19', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935013', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033917'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_19_66y_female_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753139']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 66', 'Sex: female', 'icu: yes', 'apacheii: 27', 'charlson score: 4', 'mechanical ventilation: yes', 'ventilator-free days: 21', 'dm: 0', 'hospital-free days post 45 day followup (days): 29', 'ferritin (ng/ml): 155', 'crp (mg/l): 150.2', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 0.35', 'lactate (mmol/l): 2.03', 'fibrinogen: 191', 'sofa: 15', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['NC19', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935013', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033917']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753140 -- <SAMPLE: GSM4753140>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_20_62y_female_ICU'], 'geo_accession': ['GSM4753140'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 62', 'Sex: female', 'icu: yes', 'apacheii: 18', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 0', 'dm: 1', 'hospital-free days post 45 day followup (days): 40', 'ferritin (ng/ml): 46', 'crp (mg/l): 1.8', 'ddimer (mg/l_feu): 3.43', 'procalcitonin (ng/ml): 0.5', 'lactate (mmol/l): 9.91', 'fibrinogen: 264', 'sofa: 3', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC20', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935012', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033918'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_20_62y_female_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753140']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 62', 'Sex: female', 'icu: yes', 'apacheii: 18', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 0', 'dm: 1', 'hospital-free days post 45 day followup (days): 40', 'ferritin (ng/ml): 46', 'crp (mg/l): 1.8', 'ddimer (mg/l_feu): 3.43', 'procalcitonin (ng/ml): 0.5', 'lactate (mmol/l): 9.91', 'fibrinogen: 264', 'sofa: 3', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['NC20', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935012', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033918']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753141 -- <SAMPLE: GSM4753141>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_21_71y_male_NonICU'], 'geo_accession': ['GSM4753141'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 71', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 36', 'ferritin (ng/ml): 106', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): 1.46', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 0.73', 'fibrinogen: 377', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC21', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934992', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033919'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_21_71y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753141']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 71', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 36', 'ferritin (ng/ml): 106', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): 1.46', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 0.73', 'fibrinogen: 377', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['NC21', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934992', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033919']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753142 -- <SAMPLE: GSM4753142>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_22_63y_male_NonICU'], 'geo_accession': ['GSM4753142'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 63', 'Sex: male', 'icu: no', 'apacheii: 4', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 43', 'ferritin (ng/ml): unknown', 'crp (mg/l): 26.1', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 0.05', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: 1', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC22', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934991', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033920'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_22_63y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753142']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 63', 'Sex: male', 'icu: no', 'apacheii: 4', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 43', 'ferritin (ng/ml): unknown', 'crp (mg/l): 26.1', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 0.05', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: 1', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['NC22', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934991', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033920']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753143 -- <SAMPLE: GSM4753143>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_23_42y_female_NonICU'], 'geo_accession': ['GSM4753143'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 42', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 40', 'ferritin (ng/ml): 85', 'crp (mg/l): 41', 'ddimer (mg/l_feu): 1.87', 'procalcitonin (ng/ml): 0.73', 'lactate (mmol/l): 1.99', 'fibrinogen: 635', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC23', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934990', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033921'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_23_42y_female_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753143']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 42', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 40', 'ferritin (ng/ml): 85', 'crp (mg/l): 41', 'ddimer (mg/l_feu): 1.87', 'procalcitonin (ng/ml): 0.73', 'lactate (mmol/l): 1.99', 'fibrinogen: 635', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['NC23', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934990', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033921']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753144 -- <SAMPLE: GSM4753144>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_24_32y_female_NonICU'], 'geo_accession': ['GSM4753144'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 32', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 43', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): 0.19', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 0.87', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC24', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934989', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033922'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_24_32y_female_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753144']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 32', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 43', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): 0.19', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 0.87', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['NC24', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934989', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033922']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753145 -- <SAMPLE: GSM4753145>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_25_62y_male_NonICU'], 'geo_accession': ['GSM4753145'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 62', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 8', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 44', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 0.99', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC25', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934988', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033923'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_25_62y_male_NonICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753145']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 62', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 8', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 44', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 0.99', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['NC25', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934988', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033923']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753146 -- <SAMPLE: GSM4753146>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_26_36y_male_ICU'], 'geo_accession': ['GSM4753146'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 36', 'Sex: male', 'icu: yes', 'apacheii: 40', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: 15', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC26', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934987', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033924'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_26_36y_male_ICU']\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753146']\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 36', 'Sex: male', 'icu: yes', 'apacheii: 40', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: 15', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "gsm.metadata.items \n",
      " description ['NC26', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934987', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033924']\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n"
     ]
    }
   ],
   "source": [
    "gse157103.name\n",
    "for gsm_name, gsm in gse157103.gsms.items():\n",
    "    print(gsm_name,\"--\", gsm)\n",
    "    print(\"metadata: \\n\",gsm.metadata)\n",
    "    for key, value in gsm.metadata.items():\n",
    "        print(\"gsm.metadata.items \\n\",key, value)\n",
    "    print(\"Table data:\",gsm.table)\n",
    "    print(gsm.columns)\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'GSM4753021': <SAMPLE: GSM4753021>,\n",
       " 'GSM4753022': <SAMPLE: GSM4753022>,\n",
       " 'GSM4753023': <SAMPLE: GSM4753023>,\n",
       " 'GSM4753024': <SAMPLE: GSM4753024>,\n",
       " 'GSM4753025': <SAMPLE: GSM4753025>,\n",
       " 'GSM4753026': <SAMPLE: GSM4753026>,\n",
       " 'GSM4753027': <SAMPLE: GSM4753027>,\n",
       " 'GSM4753028': <SAMPLE: GSM4753028>,\n",
       " 'GSM4753029': <SAMPLE: GSM4753029>,\n",
       " 'GSM4753030': <SAMPLE: GSM4753030>,\n",
       " 'GSM4753031': <SAMPLE: GSM4753031>,\n",
       " 'GSM4753032': <SAMPLE: GSM4753032>,\n",
       " 'GSM4753033': <SAMPLE: GSM4753033>,\n",
       " 'GSM4753034': <SAMPLE: GSM4753034>,\n",
       " 'GSM4753035': <SAMPLE: GSM4753035>,\n",
       " 'GSM4753036': <SAMPLE: GSM4753036>,\n",
       " 'GSM4753037': <SAMPLE: GSM4753037>,\n",
       " 'GSM4753038': <SAMPLE: GSM4753038>,\n",
       " 'GSM4753039': <SAMPLE: GSM4753039>,\n",
       " 'GSM4753040': <SAMPLE: GSM4753040>,\n",
       " 'GSM4753041': <SAMPLE: GSM4753041>,\n",
       " 'GSM4753042': <SAMPLE: GSM4753042>,\n",
       " 'GSM4753043': <SAMPLE: GSM4753043>,\n",
       " 'GSM4753044': <SAMPLE: GSM4753044>,\n",
       " 'GSM4753045': <SAMPLE: GSM4753045>,\n",
       " 'GSM4753046': <SAMPLE: GSM4753046>,\n",
       " 'GSM4753047': <SAMPLE: GSM4753047>,\n",
       " 'GSM4753048': <SAMPLE: GSM4753048>,\n",
       " 'GSM4753049': <SAMPLE: GSM4753049>,\n",
       " 'GSM4753050': <SAMPLE: GSM4753050>,\n",
       " 'GSM4753051': <SAMPLE: GSM4753051>,\n",
       " 'GSM4753052': <SAMPLE: GSM4753052>,\n",
       " 'GSM4753053': <SAMPLE: GSM4753053>,\n",
       " 'GSM4753054': <SAMPLE: GSM4753054>,\n",
       " 'GSM4753055': <SAMPLE: GSM4753055>,\n",
       " 'GSM4753056': <SAMPLE: GSM4753056>,\n",
       " 'GSM4753057': <SAMPLE: GSM4753057>,\n",
       " 'GSM4753058': <SAMPLE: GSM4753058>,\n",
       " 'GSM4753059': <SAMPLE: GSM4753059>,\n",
       " 'GSM4753060': <SAMPLE: GSM4753060>,\n",
       " 'GSM4753061': <SAMPLE: GSM4753061>,\n",
       " 'GSM4753062': <SAMPLE: GSM4753062>,\n",
       " 'GSM4753063': <SAMPLE: GSM4753063>,\n",
       " 'GSM4753064': <SAMPLE: GSM4753064>,\n",
       " 'GSM4753065': <SAMPLE: GSM4753065>,\n",
       " 'GSM4753066': <SAMPLE: GSM4753066>,\n",
       " 'GSM4753067': <SAMPLE: GSM4753067>,\n",
       " 'GSM4753068': <SAMPLE: GSM4753068>,\n",
       " 'GSM4753069': <SAMPLE: GSM4753069>,\n",
       " 'GSM4753070': <SAMPLE: GSM4753070>,\n",
       " 'GSM4753071': <SAMPLE: GSM4753071>,\n",
       " 'GSM4753072': <SAMPLE: GSM4753072>,\n",
       " 'GSM4753073': <SAMPLE: GSM4753073>,\n",
       " 'GSM4753074': <SAMPLE: GSM4753074>,\n",
       " 'GSM4753075': <SAMPLE: GSM4753075>,\n",
       " 'GSM4753076': <SAMPLE: GSM4753076>,\n",
       " 'GSM4753077': <SAMPLE: GSM4753077>,\n",
       " 'GSM4753078': <SAMPLE: GSM4753078>,\n",
       " 'GSM4753079': <SAMPLE: GSM4753079>,\n",
       " 'GSM4753080': <SAMPLE: GSM4753080>,\n",
       " 'GSM4753081': <SAMPLE: GSM4753081>,\n",
       " 'GSM4753082': <SAMPLE: GSM4753082>,\n",
       " 'GSM4753083': <SAMPLE: GSM4753083>,\n",
       " 'GSM4753084': <SAMPLE: GSM4753084>,\n",
       " 'GSM4753085': <SAMPLE: GSM4753085>,\n",
       " 'GSM4753086': <SAMPLE: GSM4753086>,\n",
       " 'GSM4753087': <SAMPLE: GSM4753087>,\n",
       " 'GSM4753088': <SAMPLE: GSM4753088>,\n",
       " 'GSM4753089': <SAMPLE: GSM4753089>,\n",
       " 'GSM4753090': <SAMPLE: GSM4753090>,\n",
       " 'GSM4753091': <SAMPLE: GSM4753091>,\n",
       " 'GSM4753092': <SAMPLE: GSM4753092>,\n",
       " 'GSM4753093': <SAMPLE: GSM4753093>,\n",
       " 'GSM4753094': <SAMPLE: GSM4753094>,\n",
       " 'GSM4753095': <SAMPLE: GSM4753095>,\n",
       " 'GSM4753096': <SAMPLE: GSM4753096>,\n",
       " 'GSM4753097': <SAMPLE: GSM4753097>,\n",
       " 'GSM4753098': <SAMPLE: GSM4753098>,\n",
       " 'GSM4753099': <SAMPLE: GSM4753099>,\n",
       " 'GSM4753100': <SAMPLE: GSM4753100>,\n",
       " 'GSM4753101': <SAMPLE: GSM4753101>,\n",
       " 'GSM4753102': <SAMPLE: GSM4753102>,\n",
       " 'GSM4753103': <SAMPLE: GSM4753103>,\n",
       " 'GSM4753104': <SAMPLE: GSM4753104>,\n",
       " 'GSM4753105': <SAMPLE: GSM4753105>,\n",
       " 'GSM4753106': <SAMPLE: GSM4753106>,\n",
       " 'GSM4753107': <SAMPLE: GSM4753107>,\n",
       " 'GSM4753108': <SAMPLE: GSM4753108>,\n",
       " 'GSM4753109': <SAMPLE: GSM4753109>,\n",
       " 'GSM4753110': <SAMPLE: GSM4753110>,\n",
       " 'GSM4753111': <SAMPLE: GSM4753111>,\n",
       " 'GSM4753112': <SAMPLE: GSM4753112>,\n",
       " 'GSM4753113': <SAMPLE: GSM4753113>,\n",
       " 'GSM4753114': <SAMPLE: GSM4753114>,\n",
       " 'GSM4753115': <SAMPLE: GSM4753115>,\n",
       " 'GSM4753116': <SAMPLE: GSM4753116>,\n",
       " 'GSM4753117': <SAMPLE: GSM4753117>,\n",
       " 'GSM4753118': <SAMPLE: GSM4753118>,\n",
       " 'GSM4753119': <SAMPLE: GSM4753119>,\n",
       " 'GSM4753120': <SAMPLE: GSM4753120>,\n",
       " 'GSM4753121': <SAMPLE: GSM4753121>,\n",
       " 'GSM4753122': <SAMPLE: GSM4753122>,\n",
       " 'GSM4753123': <SAMPLE: GSM4753123>,\n",
       " 'GSM4753124': <SAMPLE: GSM4753124>,\n",
       " 'GSM4753125': <SAMPLE: GSM4753125>,\n",
       " 'GSM4753126': <SAMPLE: GSM4753126>,\n",
       " 'GSM4753127': <SAMPLE: GSM4753127>,\n",
       " 'GSM4753128': <SAMPLE: GSM4753128>,\n",
       " 'GSM4753129': <SAMPLE: GSM4753129>,\n",
       " 'GSM4753130': <SAMPLE: GSM4753130>,\n",
       " 'GSM4753131': <SAMPLE: GSM4753131>,\n",
       " 'GSM4753132': <SAMPLE: GSM4753132>,\n",
       " 'GSM4753133': <SAMPLE: GSM4753133>,\n",
       " 'GSM4753134': <SAMPLE: GSM4753134>,\n",
       " 'GSM4753135': <SAMPLE: GSM4753135>,\n",
       " 'GSM4753136': <SAMPLE: GSM4753136>,\n",
       " 'GSM4753137': <SAMPLE: GSM4753137>,\n",
       " 'GSM4753138': <SAMPLE: GSM4753138>,\n",
       " 'GSM4753139': <SAMPLE: GSM4753139>,\n",
       " 'GSM4753140': <SAMPLE: GSM4753140>,\n",
       " 'GSM4753141': <SAMPLE: GSM4753141>,\n",
       " 'GSM4753142': <SAMPLE: GSM4753142>,\n",
       " 'GSM4753143': <SAMPLE: GSM4753143>,\n",
       " 'GSM4753144': <SAMPLE: GSM4753144>,\n",
       " 'GSM4753145': <SAMPLE: GSM4753145>,\n",
       " 'GSM4753146': <SAMPLE: GSM4753146>}"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gse157103.gsms\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "GSM4753021 -- <SAMPLE: GSM4753021>\n",
      "metadata: \n",
      " {'title': ['COVID_01_39y_male_NonICU'], 'geo_accession': ['GSM4753021'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 39', 'Sex: male', 'icu: no', 'apacheii: 15', 'charlson score: 0', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 946', 'crp (mg/l): 73.1', 'ddimer (mg/l_feu): 1.3', 'procalcitonin (ng/ml): 36', 'lactate (mmol/l): 0.9', 'fibrinogen: 513', 'sofa: 8', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C1', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934957', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033799'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_01_39y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753021']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 39', 'Sex: male', 'icu: no', 'apacheii: 15', 'charlson score: 0', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 946', 'crp (mg/l): 73.1', 'ddimer (mg/l_feu): 1.3', 'procalcitonin (ng/ml): 36', 'lactate (mmol/l): 0.9', 'fibrinogen: 513', 'sofa: 8', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C1', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934957', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033799']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753021>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753022 -- <SAMPLE: GSM4753022>\n",
      "metadata: \n",
      " {'title': ['COVID_02_63y_male_NonICU'], 'geo_accession': ['GSM4753022'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 63', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 39', 'ferritin (ng/ml): 1060', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): 1.03', 'procalcitonin (ng/ml): 0.37', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C2', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934956', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033800'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_02_63y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753022']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 63', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 39', 'ferritin (ng/ml): 1060', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): 1.03', 'procalcitonin (ng/ml): 0.37', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C2', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934956', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033800']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753022>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753023 -- <SAMPLE: GSM4753023>\n",
      "metadata: \n",
      " {'title': ['COVID_03_33y_male_NonICU'], 'geo_accession': ['GSM4753023'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 33', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 18', 'ferritin (ng/ml): 1335', 'crp (mg/l): 53.2', 'ddimer (mg/l_feu): 1.48', 'procalcitonin (ng/ml): 0.07', 'lactate (mmol/l): unknown', 'fibrinogen: 513', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C3', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934955', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033801'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_03_33y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753023']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 33', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 18', 'ferritin (ng/ml): 1335', 'crp (mg/l): 53.2', 'ddimer (mg/l_feu): 1.48', 'procalcitonin (ng/ml): 0.07', 'lactate (mmol/l): unknown', 'fibrinogen: 513', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C3', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934955', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033801']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753023>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753024 -- <SAMPLE: GSM4753024>\n",
      "metadata: \n",
      " {'title': ['COVID_04_49y_male_NonICU'], 'geo_accession': ['GSM4753024'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 49', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 39', 'ferritin (ng/ml): 583', 'crp (mg/l): 251.1', 'ddimer (mg/l_feu): 1.32', 'procalcitonin (ng/ml): 0.98', 'lactate (mmol/l): 0.87', 'fibrinogen: 949', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C4', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934954', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033802'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_04_49y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753024']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 49', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 39', 'ferritin (ng/ml): 583', 'crp (mg/l): 251.1', 'ddimer (mg/l_feu): 1.32', 'procalcitonin (ng/ml): 0.98', 'lactate (mmol/l): 0.87', 'fibrinogen: 949', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C4', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934954', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033802']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753024>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753025 -- <SAMPLE: GSM4753025>\n",
      "metadata: \n",
      " {'title': ['COVID_05_49y_male_NonICU'], 'geo_accession': ['GSM4753025'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 49', 'Sex: male', 'icu: no', 'apacheii: 19', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 23', 'dm: 0', 'hospital-free days post 45 day followup (days): 27', 'ferritin (ng/ml): 800', 'crp (mg/l): 355.8', 'ddimer (mg/l_feu): 0.69', 'procalcitonin (ng/ml): 4.92', 'lactate (mmol/l): 1.48', 'fibrinogen: 929', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C5', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934953', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033803'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_05_49y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753025']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 49', 'Sex: male', 'icu: no', 'apacheii: 19', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 23', 'dm: 0', 'hospital-free days post 45 day followup (days): 27', 'ferritin (ng/ml): 800', 'crp (mg/l): 355.8', 'ddimer (mg/l_feu): 0.69', 'procalcitonin (ng/ml): 4.92', 'lactate (mmol/l): 1.48', 'fibrinogen: 929', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C5', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934953', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033803']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753025>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753026 -- <SAMPLE: GSM4753026>\n",
      "metadata: \n",
      " {'title': ['COVID_06_:y_male_NonICU'], 'geo_accession': ['GSM4753026'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): :', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 36', 'ferritin (ng/ml): 563', 'crp (mg/l): 129.1', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 0.67', 'lactate (mmol/l): 0.86', 'fibrinogen: 769', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C6', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934952', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033804'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_06_:y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753026']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): :', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 36', 'ferritin (ng/ml): 563', 'crp (mg/l): 129.1', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 0.67', 'lactate (mmol/l): 0.86', 'fibrinogen: 769', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C6', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934952', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033804']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753026>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753027 -- <SAMPLE: GSM4753027>\n",
      "metadata: \n",
      " {'title': ['COVID_07_38y_female_NonICU'], 'geo_accession': ['GSM4753027'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 38', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 7', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 42', 'ferritin (ng/ml): 366', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): 0.87', 'procalcitonin (ng/ml): 0.06', 'lactate (mmol/l): 1.17', 'fibrinogen: 478', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C7', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934951', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033805'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_07_38y_female_NonICU']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753027']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 38', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 7', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 42', 'ferritin (ng/ml): 366', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): 0.87', 'procalcitonin (ng/ml): 0.06', 'lactate (mmol/l): 1.17', 'fibrinogen: 478', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C7', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934951', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033805']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753027>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753028 -- <SAMPLE: GSM4753028>\n",
      "metadata: \n",
      " {'title': ['COVID_08_78y_male_ICU'], 'geo_accession': ['GSM4753028'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 78', 'Sex: male', 'icu: yes', 'apacheii: 43', 'charlson score: 7', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1103', 'crp (mg/l): 79.5', 'ddimer (mg/l_feu): 12.16', 'procalcitonin (ng/ml): 4.2', 'lactate (mmol/l): 1.65', 'fibrinogen: 780', 'sofa: 16', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C8', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934950', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033806'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_08_78y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753028']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 78', 'Sex: male', 'icu: yes', 'apacheii: 43', 'charlson score: 7', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1103', 'crp (mg/l): 79.5', 'ddimer (mg/l_feu): 12.16', 'procalcitonin (ng/ml): 4.2', 'lactate (mmol/l): 1.65', 'fibrinogen: 780', 'sofa: 16', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C8', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934950', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033806']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753028>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753029 -- <SAMPLE: GSM4753029>\n",
      "metadata: \n",
      " {'title': ['COVID_09_64y_female_ICU'], 'geo_accession': ['GSM4753029'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 64', 'Sex: female', 'icu: yes', 'apacheii: 31', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 680', 'crp (mg/l): 203.5', 'ddimer (mg/l_feu): 102.21', 'procalcitonin (ng/ml): 0.25', 'lactate (mmol/l): 1.14', 'fibrinogen: 317', 'sofa: 12', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C9', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934949', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033807'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_09_64y_female_ICU']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753029']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 64', 'Sex: female', 'icu: yes', 'apacheii: 31', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 680', 'crp (mg/l): 203.5', 'ddimer (mg/l_feu): 102.21', 'procalcitonin (ng/ml): 0.25', 'lactate (mmol/l): 1.14', 'fibrinogen: 317', 'sofa: 12', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C9', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934949', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033807']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753029>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753030 -- <SAMPLE: GSM4753030>\n",
      "metadata: \n",
      " {'title': ['COVID_10_62y_male_ICU'], 'geo_accession': ['GSM4753030'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 62', 'Sex: male', 'icu: yes', 'apacheii: 34', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 2', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1746', 'crp (mg/l): 197.8', 'ddimer (mg/l_feu): 11.54', 'procalcitonin (ng/ml): 1.1', 'lactate (mmol/l): 1.45', 'fibrinogen: 893', 'sofa: 15', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C10', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934948', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033808'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_10_62y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753030']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 62', 'Sex: male', 'icu: yes', 'apacheii: 34', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 2', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1746', 'crp (mg/l): 197.8', 'ddimer (mg/l_feu): 11.54', 'procalcitonin (ng/ml): 1.1', 'lactate (mmol/l): 1.45', 'fibrinogen: 893', 'sofa: 15', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C10', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934948', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033808']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753030>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753031 -- <SAMPLE: GSM4753031>\n",
      "metadata: \n",
      " {'title': ['COVID_11_52y_female_NonICU'], 'geo_accession': ['GSM4753031'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 52', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 37', 'ferritin (ng/ml): 401', 'crp (mg/l): 64.1', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 0.21', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C11', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934947', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033809'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_11_52y_female_NonICU']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753031']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 52', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 37', 'ferritin (ng/ml): 401', 'crp (mg/l): 64.1', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 0.21', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C11', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934947', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033809']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753031>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753032 -- <SAMPLE: GSM4753032>\n",
      "metadata: \n",
      " {'title': ['COVID_12_50y_male_ICU'], 'geo_accession': ['GSM4753032'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 50', 'Sex: male', 'icu: yes', 'apacheii: 27', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 15', 'dm: 0', 'hospital-free days post 45 day followup (days): 22', 'ferritin (ng/ml): 2303', 'crp (mg/l): 199.8', 'ddimer (mg/l_feu): 2.21', 'procalcitonin (ng/ml): 2.61', 'lactate (mmol/l): 2.1', 'fibrinogen: 910', 'sofa: 10', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C12', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934946', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033810'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_12_50y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753032']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 50', 'Sex: male', 'icu: yes', 'apacheii: 27', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 15', 'dm: 0', 'hospital-free days post 45 day followup (days): 22', 'ferritin (ng/ml): 2303', 'crp (mg/l): 199.8', 'ddimer (mg/l_feu): 2.21', 'procalcitonin (ng/ml): 2.61', 'lactate (mmol/l): 2.1', 'fibrinogen: 910', 'sofa: 10', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C12', 'genes.ec.tsv', 'genes.tpm.tsv']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934946', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033810']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753032>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753033 -- <SAMPLE: GSM4753033>\n",
      "metadata: \n",
      " {'title': ['COVID_13_37y_male_NonICU'], 'geo_accession': ['GSM4753033'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 37', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 39', 'ferritin (ng/ml): 245', 'crp (mg/l): 122.5', 'ddimer (mg/l_feu): 0.58', 'procalcitonin (ng/ml): 0.15', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C13', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934945', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033811'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_13_37y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753033']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 37', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 39', 'ferritin (ng/ml): 245', 'crp (mg/l): 122.5', 'ddimer (mg/l_feu): 0.58', 'procalcitonin (ng/ml): 0.15', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C13', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934945', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033811']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753033>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753034 -- <SAMPLE: GSM4753034>\n",
      "metadata: \n",
      " {'title': ['COVID_14_55y_male_ICU'], 'geo_accession': ['GSM4753034'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 55', 'Sex: male', 'icu: yes', 'apacheii: 23', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 20', 'ferritin (ng/ml): 1632', 'crp (mg/l): 408.8', 'ddimer (mg/l_feu): 33.77', 'procalcitonin (ng/ml): 1.46', 'lactate (mmol/l): 1.35', 'fibrinogen: 654', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C14', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934944', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033812'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_14_55y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753034']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 55', 'Sex: male', 'icu: yes', 'apacheii: 23', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 20', 'ferritin (ng/ml): 1632', 'crp (mg/l): 408.8', 'ddimer (mg/l_feu): 33.77', 'procalcitonin (ng/ml): 1.46', 'lactate (mmol/l): 1.35', 'fibrinogen: 654', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C14', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934944', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033812']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753034>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753035 -- <SAMPLE: GSM4753035>\n",
      "metadata: \n",
      " {'title': ['COVID_15_68y_male_ICU'], 'geo_accession': ['GSM4753035'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 68', 'Sex: male', 'icu: yes', 'apacheii: 27', 'charlson score: 11', 'mechanical ventilation: yes', 'ventilator-free days: 2', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1205', 'crp (mg/l): 222.2', 'ddimer (mg/l_feu): 5.27', 'procalcitonin (ng/ml): 0.08', 'lactate (mmol/l): 1.35', 'fibrinogen: 675', 'sofa: 12', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C15', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934943', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033813'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_15_68y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753035']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 68', 'Sex: male', 'icu: yes', 'apacheii: 27', 'charlson score: 11', 'mechanical ventilation: yes', 'ventilator-free days: 2', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1205', 'crp (mg/l): 222.2', 'ddimer (mg/l_feu): 5.27', 'procalcitonin (ng/ml): 0.08', 'lactate (mmol/l): 1.35', 'fibrinogen: 675', 'sofa: 12', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C15', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934943', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033813']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753035>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753036 -- <SAMPLE: GSM4753036>\n",
      "metadata: \n",
      " {'title': ['COVID_16_48y_male_NonICU'], 'geo_accession': ['GSM4753036'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 48', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 42', 'ferritin (ng/ml): 923', 'crp (mg/l): 206.1', 'ddimer (mg/l_feu): 0.35', 'procalcitonin (ng/ml): 0.07', 'lactate (mmol/l): 1.23', 'fibrinogen: 540', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C16', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934942', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033814'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_16_48y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753036']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 48', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 42', 'ferritin (ng/ml): 923', 'crp (mg/l): 206.1', 'ddimer (mg/l_feu): 0.35', 'procalcitonin (ng/ml): 0.07', 'lactate (mmol/l): 1.23', 'fibrinogen: 540', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C16', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934942', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033814']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753036>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753037 -- <SAMPLE: GSM4753037>\n",
      "metadata: \n",
      " {'title': ['COVID_17_54y_male_NonICU'], 'geo_accession': ['GSM4753037'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 54', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 33', 'ferritin (ng/ml): 1249', 'crp (mg/l): 52.3', 'ddimer (mg/l_feu): 4.2', 'procalcitonin (ng/ml): 1.02', 'lactate (mmol/l): 1.47', 'fibrinogen: 583', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C17', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934940', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033815'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_17_54y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753037']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 54', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 33', 'ferritin (ng/ml): 1249', 'crp (mg/l): 52.3', 'ddimer (mg/l_feu): 4.2', 'procalcitonin (ng/ml): 1.02', 'lactate (mmol/l): 1.47', 'fibrinogen: 583', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C17', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934940', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033815']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753037>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753038 -- <SAMPLE: GSM4753038>\n",
      "metadata: \n",
      " {'title': ['COVID_18_70y_female_NonICU'], 'geo_accession': ['GSM4753038'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 70', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 34', 'ferritin (ng/ml): 77', 'crp (mg/l): 82.2', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 0.5', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C18', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934939', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033816'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_18_70y_female_NonICU']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753038']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 70', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 34', 'ferritin (ng/ml): 77', 'crp (mg/l): 82.2', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 0.5', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C18', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934939', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033816']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753038>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753039 -- <SAMPLE: GSM4753039>\n",
      "metadata: \n",
      " {'title': ['COVID_19_51y_male_NonICU'], 'geo_accession': ['GSM4753039'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 51', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 38', 'ferritin (ng/ml): 1116', 'crp (mg/l): 110.1', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 0.16', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C19', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934938', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033817'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_19_51y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753039']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 51', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 38', 'ferritin (ng/ml): 1116', 'crp (mg/l): 110.1', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 0.16', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C19', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934938', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033817']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753039>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753040 -- <SAMPLE: GSM4753040>\n",
      "metadata: \n",
      " {'title': ['COVID_20_62y_male_ICU'], 'geo_accession': ['GSM4753040'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 62', 'Sex: male', 'icu: yes', 'apacheii: 8', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 35', 'ferritin (ng/ml): 1219', 'crp (mg/l): 37.7', 'ddimer (mg/l_feu): 1.12', 'procalcitonin (ng/ml): 0.31', 'lactate (mmol/l): 1.05', 'fibrinogen: 301', 'sofa: 2', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C20', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934937', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033818'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_20_62y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753040']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 62', 'Sex: male', 'icu: yes', 'apacheii: 8', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 35', 'ferritin (ng/ml): 1219', 'crp (mg/l): 37.7', 'ddimer (mg/l_feu): 1.12', 'procalcitonin (ng/ml): 0.31', 'lactate (mmol/l): 1.05', 'fibrinogen: 301', 'sofa: 2', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C20', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934937', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033818']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753040>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753041 -- <SAMPLE: GSM4753041>\n",
      "metadata: \n",
      " {'title': ['COVID_21_66y_male_ICU'], 'geo_accession': ['GSM4753041'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 66', 'Sex: male', 'icu: yes', 'apacheii: 19', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 26', 'dm: 1', 'hospital-free days post 45 day followup (days): 32', 'ferritin (ng/ml): 107', 'crp (mg/l): 132.5', 'ddimer (mg/l_feu): 4.72', 'procalcitonin (ng/ml): 0.05', 'lactate (mmol/l): 1', 'fibrinogen: 457', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C21', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934936', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033819'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_21_66y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753041']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 66', 'Sex: male', 'icu: yes', 'apacheii: 19', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 26', 'dm: 1', 'hospital-free days post 45 day followup (days): 32', 'ferritin (ng/ml): 107', 'crp (mg/l): 132.5', 'ddimer (mg/l_feu): 4.72', 'procalcitonin (ng/ml): 0.05', 'lactate (mmol/l): 1', 'fibrinogen: 457', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C21', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934936', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033819']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753041>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753042 -- <SAMPLE: GSM4753042>\n",
      "metadata: \n",
      " {'title': ['COVID_22_43y_male_ICU'], 'geo_accession': ['GSM4753042'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 43', 'Sex: male', 'icu: yes', 'apacheii: 28', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 3438', 'crp (mg/l): 242.6', 'ddimer (mg/l_feu): 5.66', 'procalcitonin (ng/ml): 20.13', 'lactate (mmol/l): 0.92', 'fibrinogen: 949', 'sofa: 13', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C22', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934935', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033820'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_22_43y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753042']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 43', 'Sex: male', 'icu: yes', 'apacheii: 28', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 3438', 'crp (mg/l): 242.6', 'ddimer (mg/l_feu): 5.66', 'procalcitonin (ng/ml): 20.13', 'lactate (mmol/l): 0.92', 'fibrinogen: 949', 'sofa: 13', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C22', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934935', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033820']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753042>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753043 -- <SAMPLE: GSM4753043>\n",
      "metadata: \n",
      " {'title': ['COVID_23_76y_male_ICU'], 'geo_accession': ['GSM4753043'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 76', 'Sex: male', 'icu: yes', 'apacheii: 21', 'charlson score: 4', 'mechanical ventilation: yes', 'ventilator-free days: 20', 'dm: 1', 'hospital-free days post 45 day followup (days): 12', 'ferritin (ng/ml): 332', 'crp (mg/l): 107.6', 'ddimer (mg/l_feu): 21.55', 'procalcitonin (ng/ml): 0.33', 'lactate (mmol/l): 1.71', 'fibrinogen: 237', 'sofa: 12', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C23', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934934', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033821'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_23_76y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753043']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 76', 'Sex: male', 'icu: yes', 'apacheii: 21', 'charlson score: 4', 'mechanical ventilation: yes', 'ventilator-free days: 20', 'dm: 1', 'hospital-free days post 45 day followup (days): 12', 'ferritin (ng/ml): 332', 'crp (mg/l): 107.6', 'ddimer (mg/l_feu): 21.55', 'procalcitonin (ng/ml): 0.33', 'lactate (mmol/l): 1.71', 'fibrinogen: 237', 'sofa: 12', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C23', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934934', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033821']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753043>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753044 -- <SAMPLE: GSM4753044>\n",
      "metadata: \n",
      " {'title': ['COVID_24_55y_male_ICU'], 'geo_accession': ['GSM4753044'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 55', 'Sex: male', 'icu: yes', 'apacheii: 20', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 22', 'dm: 0', 'hospital-free days post 45 day followup (days): 23', 'ferritin (ng/ml): 1565', 'crp (mg/l): 64', 'ddimer (mg/l_feu): 1.76', 'procalcitonin (ng/ml): 0.73', 'lactate (mmol/l): unknown', 'fibrinogen: 365', 'sofa: 6', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C24', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934933', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033822'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_24_55y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753044']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 55', 'Sex: male', 'icu: yes', 'apacheii: 20', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 22', 'dm: 0', 'hospital-free days post 45 day followup (days): 23', 'ferritin (ng/ml): 1565', 'crp (mg/l): 64', 'ddimer (mg/l_feu): 1.76', 'procalcitonin (ng/ml): 0.73', 'lactate (mmol/l): unknown', 'fibrinogen: 365', 'sofa: 6', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C24', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934933', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033822']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753044>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753045 -- <SAMPLE: GSM4753045>\n",
      "metadata: \n",
      " {'title': ['COVID_25_55y_male_ICU'], 'geo_accession': ['GSM4753045'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 55', 'Sex: male', 'icu: yes', 'apacheii: 18', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 11', 'dm: 0', 'hospital-free days post 45 day followup (days): 26', 'ferritin (ng/ml): 355', 'crp (mg/l): 64.5', 'ddimer (mg/l_feu): 28.8', 'procalcitonin (ng/ml): 18', 'lactate (mmol/l): 0.94', 'fibrinogen: 719', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C25', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934932', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033823'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_25_55y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753045']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 55', 'Sex: male', 'icu: yes', 'apacheii: 18', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 11', 'dm: 0', 'hospital-free days post 45 day followup (days): 26', 'ferritin (ng/ml): 355', 'crp (mg/l): 64.5', 'ddimer (mg/l_feu): 28.8', 'procalcitonin (ng/ml): 18', 'lactate (mmol/l): 0.94', 'fibrinogen: 719', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C25', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934932', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033823']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753045>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753046 -- <SAMPLE: GSM4753046>\n",
      "metadata: \n",
      " {'title': ['COVID_26_41y_female_ICU'], 'geo_accession': ['GSM4753046'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 41', 'Sex: female', 'icu: yes', 'apacheii: 13', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 25', 'dm: 0', 'hospital-free days post 45 day followup (days): 27', 'ferritin (ng/ml): 318', 'crp (mg/l): 51.1', 'ddimer (mg/l_feu): 0.65', 'procalcitonin (ng/ml): 0.13', 'lactate (mmol/l): 0.8', 'fibrinogen: unknown', 'sofa: 9', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C26', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934931', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033824'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_26_41y_female_ICU']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753046']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 41', 'Sex: female', 'icu: yes', 'apacheii: 13', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 25', 'dm: 0', 'hospital-free days post 45 day followup (days): 27', 'ferritin (ng/ml): 318', 'crp (mg/l): 51.1', 'ddimer (mg/l_feu): 0.65', 'procalcitonin (ng/ml): 0.13', 'lactate (mmol/l): 0.8', 'fibrinogen: unknown', 'sofa: 9', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C26', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934931', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033824']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753046>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753047 -- <SAMPLE: GSM4753047>\n",
      "metadata: \n",
      " {'title': ['COVID_27_71y_female_ICU'], 'geo_accession': ['GSM4753047'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 71', 'Sex: female', 'icu: yes', 'apacheii: 29', 'charlson score: 6', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 882', 'crp (mg/l): 285', 'ddimer (mg/l_feu): 58.18', 'procalcitonin (ng/ml): 0.9', 'lactate (mmol/l): unknown', 'fibrinogen: 659', 'sofa: 11', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C27', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934930', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033825'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_27_71y_female_ICU']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753047']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 71', 'Sex: female', 'icu: yes', 'apacheii: 29', 'charlson score: 6', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 882', 'crp (mg/l): 285', 'ddimer (mg/l_feu): 58.18', 'procalcitonin (ng/ml): 0.9', 'lactate (mmol/l): unknown', 'fibrinogen: 659', 'sofa: 11', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C27', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934930', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033825']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753047>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753048 -- <SAMPLE: GSM4753048>\n",
      "metadata: \n",
      " {'title': ['COVID_28_63y_male_ICU'], 'geo_accession': ['GSM4753048'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 63', 'Sex: male', 'icu: yes', 'apacheii: 17', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 109', 'crp (mg/l): 40.1', 'ddimer (mg/l_feu): 2.03', 'procalcitonin (ng/ml): 0.93', 'lactate (mmol/l): unknown', 'fibrinogen: 554', 'sofa: 6', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C28', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934929', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033826'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_28_63y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753048']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 63', 'Sex: male', 'icu: yes', 'apacheii: 17', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 109', 'crp (mg/l): 40.1', 'ddimer (mg/l_feu): 2.03', 'procalcitonin (ng/ml): 0.93', 'lactate (mmol/l): unknown', 'fibrinogen: 554', 'sofa: 6', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C28', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934929', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033826']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753048>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753049 -- <SAMPLE: GSM4753049>\n",
      "metadata: \n",
      " {'title': ['COVID_29_63y_female_ICU'], 'geo_accession': ['GSM4753049'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 63', 'Sex: female', 'icu: yes', 'apacheii: 17', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 21', 'dm: 1', 'hospital-free days post 45 day followup (days): 10', 'ferritin (ng/ml): 376', 'crp (mg/l): 350.6', 'ddimer (mg/l_feu): 1.75', 'procalcitonin (ng/ml): 1.43', 'lactate (mmol/l): unknown', 'fibrinogen: 439', 'sofa: 6', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C29', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934928', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033827'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_29_63y_female_ICU']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753049']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 63', 'Sex: female', 'icu: yes', 'apacheii: 17', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 21', 'dm: 1', 'hospital-free days post 45 day followup (days): 10', 'ferritin (ng/ml): 376', 'crp (mg/l): 350.6', 'ddimer (mg/l_feu): 1.75', 'procalcitonin (ng/ml): 1.43', 'lactate (mmol/l): unknown', 'fibrinogen: 439', 'sofa: 6', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C29', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934928', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033827']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753049>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753050 -- <SAMPLE: GSM4753050>\n",
      "metadata: \n",
      " {'title': ['COVID_30_54y_male_ICU'], 'geo_accession': ['GSM4753050'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 54', 'Sex: male', 'icu: yes', 'apacheii: 25', 'charlson score: 7', 'mechanical ventilation: yes', 'ventilator-free days: 7', 'dm: 1', 'hospital-free days post 45 day followup (days): 3', 'ferritin (ng/ml): 686', 'crp (mg/l): 171.1', 'ddimer (mg/l_feu): 2.37', 'procalcitonin (ng/ml): 13', 'lactate (mmol/l): 0.87', 'fibrinogen: 635', 'sofa: 8', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C30', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934927', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033828'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_30_54y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753050']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 54', 'Sex: male', 'icu: yes', 'apacheii: 25', 'charlson score: 7', 'mechanical ventilation: yes', 'ventilator-free days: 7', 'dm: 1', 'hospital-free days post 45 day followup (days): 3', 'ferritin (ng/ml): 686', 'crp (mg/l): 171.1', 'ddimer (mg/l_feu): 2.37', 'procalcitonin (ng/ml): 13', 'lactate (mmol/l): 0.87', 'fibrinogen: 635', 'sofa: 8', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C30', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934927', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033828']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753050>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753051 -- <SAMPLE: GSM4753051>\n",
      "metadata: \n",
      " {'title': ['COVID_31_50y_male_ICU'], 'geo_accession': ['GSM4753051'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 50', 'Sex: male', 'icu: yes', 'apacheii: 24', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 374', 'crp (mg/l): 28.4', 'ddimer (mg/l_feu): 74.42', 'procalcitonin (ng/ml): 2.83', 'lactate (mmol/l): 2.85', 'fibrinogen: 140', 'sofa: 14', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C31', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934986', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033829'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_31_50y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753051']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 50', 'Sex: male', 'icu: yes', 'apacheii: 24', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 374', 'crp (mg/l): 28.4', 'ddimer (mg/l_feu): 74.42', 'procalcitonin (ng/ml): 2.83', 'lactate (mmol/l): 2.85', 'fibrinogen: 140', 'sofa: 14', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C31', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934986', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033829']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753051>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753052 -- <SAMPLE: GSM4753052>\n",
      "metadata: \n",
      " {'title': ['COVID_32_72y_male_ICU'], 'geo_accession': ['GSM4753052'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 72', 'Sex: male', 'icu: yes', 'apacheii: 21', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 7', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1294', 'crp (mg/l): 311.5', 'ddimer (mg/l_feu): 1.32', 'procalcitonin (ng/ml): 2.17', 'lactate (mmol/l): 1.05', 'fibrinogen: 513', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C32', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934985', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033830'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_32_72y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753052']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 72', 'Sex: male', 'icu: yes', 'apacheii: 21', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 7', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1294', 'crp (mg/l): 311.5', 'ddimer (mg/l_feu): 1.32', 'procalcitonin (ng/ml): 2.17', 'lactate (mmol/l): 1.05', 'fibrinogen: 513', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C32', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934985', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033830']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753052>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753053 -- <SAMPLE: GSM4753053>\n",
      "metadata: \n",
      " {'title': ['COVID_33_81y_male_NonICU'], 'geo_accession': ['GSM4753053'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 81', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 40', 'ferritin (ng/ml): 1013', 'crp (mg/l): 222.4', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 0.5', 'lactate (mmol/l): 1.39', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C33', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934984', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033831'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_33_81y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753053']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 81', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 40', 'ferritin (ng/ml): 1013', 'crp (mg/l): 222.4', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 0.5', 'lactate (mmol/l): 1.39', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C33', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934984', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033831']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753053>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753054 -- <SAMPLE: GSM4753054>\n",
      "metadata: \n",
      " {'title': ['COVID_34_64y_female_NonICU'], 'geo_accession': ['GSM4753054'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 64', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 36', 'ferritin (ng/ml): 563', 'crp (mg/l): 47.8', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 1.01', 'fibrinogen: 467', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C34', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934983', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033832'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_34_64y_female_NonICU']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753054']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 64', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 36', 'ferritin (ng/ml): 563', 'crp (mg/l): 47.8', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 1.01', 'fibrinogen: 467', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C34', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934983', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033832']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753054>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753055 -- <SAMPLE: GSM4753055>\n",
      "metadata: \n",
      " {'title': ['COVID_35_58y_female_NonICU'], 'geo_accession': ['GSM4753055'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 58', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 3', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 35', 'ferritin (ng/ml): 4859', 'crp (mg/l): 129.6', 'ddimer (mg/l_feu): 0.5', 'procalcitonin (ng/ml): 0.39', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C35', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934982', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033833'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_35_58y_female_NonICU']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753055']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 58', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 3', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 35', 'ferritin (ng/ml): 4859', 'crp (mg/l): 129.6', 'ddimer (mg/l_feu): 0.5', 'procalcitonin (ng/ml): 0.39', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C35', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934982', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033833']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753055>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753056 -- <SAMPLE: GSM4753056>\n",
      "metadata: \n",
      " {'title': ['COVID_36_68y_male_NonICU'], 'geo_accession': ['GSM4753056'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 68', 'Sex: male', 'icu: no', 'apacheii: 26', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1689', 'crp (mg/l): 203.7', 'ddimer (mg/l_feu): 13.37', 'procalcitonin (ng/ml): 1.2', 'lactate (mmol/l): 1.24', 'fibrinogen: 752', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C36', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934981', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033834'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_36_68y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753056']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 68', 'Sex: male', 'icu: no', 'apacheii: 26', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1689', 'crp (mg/l): 203.7', 'ddimer (mg/l_feu): 13.37', 'procalcitonin (ng/ml): 1.2', 'lactate (mmol/l): 1.24', 'fibrinogen: 752', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C36', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934981', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033834']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753056>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753057 -- <SAMPLE: GSM4753057>\n",
      "metadata: \n",
      " {'title': ['COVID_37_87y_male_NonICU'], 'geo_accession': ['GSM4753057'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 87', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 43', 'ferritin (ng/ml): 248', 'crp (mg/l): 158.7', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C37', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934980', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033835'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_37_87y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753057']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 87', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 43', 'ferritin (ng/ml): 248', 'crp (mg/l): 158.7', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C37', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934980', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033835']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753057>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753058 -- <SAMPLE: GSM4753058>\n",
      "metadata: \n",
      " {'title': ['COVID_38_68y_male_ICU'], 'geo_accession': ['GSM4753058'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 68', 'Sex: male', 'icu: yes', 'apacheii: 7', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 16', 'ferritin (ng/ml): 2212', 'crp (mg/l): 133.6', 'ddimer (mg/l_feu): 7.48', 'procalcitonin (ng/ml): 0.33', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: 2', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C38', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934979', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033836'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_38_68y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753058']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 68', 'Sex: male', 'icu: yes', 'apacheii: 7', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 16', 'ferritin (ng/ml): 2212', 'crp (mg/l): 133.6', 'ddimer (mg/l_feu): 7.48', 'procalcitonin (ng/ml): 0.33', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: 2', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C38', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753058>\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934979', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033836']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753058>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753059 -- <SAMPLE: GSM4753059>\n",
      "metadata: \n",
      " {'title': ['COVID_39_80y_female_ICU'], 'geo_accession': ['GSM4753059'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 80', 'Sex: female', 'icu: yes', 'apacheii: 33', 'charlson score: 7', 'mechanical ventilation: yes', 'ventilator-free days: 12', 'dm: 1', 'hospital-free days post 45 day followup (days): 3', 'ferritin (ng/ml): 747', 'crp (mg/l): 247', 'ddimer (mg/l_feu): 18.71', 'procalcitonin (ng/ml): 86.39', 'lactate (mmol/l): unknown', 'fibrinogen: 659', 'sofa: 10', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C39', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934978', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033837'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_39_80y_female_ICU']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753059']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 80', 'Sex: female', 'icu: yes', 'apacheii: 33', 'charlson score: 7', 'mechanical ventilation: yes', 'ventilator-free days: 12', 'dm: 1', 'hospital-free days post 45 day followup (days): 3', 'ferritin (ng/ml): 747', 'crp (mg/l): 247', 'ddimer (mg/l_feu): 18.71', 'procalcitonin (ng/ml): 86.39', 'lactate (mmol/l): unknown', 'fibrinogen: 659', 'sofa: 10', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C39', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934978', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033837']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753059>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753060 -- <SAMPLE: GSM4753060>\n",
      "metadata: \n",
      " {'title': ['COVID_40_66y_male_ICU'], 'geo_accession': ['GSM4753060'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 66', 'Sex: male', 'icu: yes', 'apacheii: 18', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 22', 'dm: 1', 'hospital-free days post 45 day followup (days): 29', 'ferritin (ng/ml): 566', 'crp (mg/l): 267.2', 'ddimer (mg/l_feu): 16.18', 'procalcitonin (ng/ml): 1.7', 'lactate (mmol/l): 1.65', 'fibrinogen: unknown', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C40', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934977', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033838'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_40_66y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753060']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 66', 'Sex: male', 'icu: yes', 'apacheii: 18', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 22', 'dm: 1', 'hospital-free days post 45 day followup (days): 29', 'ferritin (ng/ml): 566', 'crp (mg/l): 267.2', 'ddimer (mg/l_feu): 16.18', 'procalcitonin (ng/ml): 1.7', 'lactate (mmol/l): 1.65', 'fibrinogen: unknown', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C40', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934977', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033838']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753060>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753061 -- <SAMPLE: GSM4753061>\n",
      "metadata: \n",
      " {'title': ['COVID_41_74y_male_ICU'], 'geo_accession': ['GSM4753061'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 74', 'Sex: male', 'icu: yes', 'apacheii: 30', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 15', 'dm: 0', 'hospital-free days post 45 day followup (days): 20', 'ferritin (ng/ml): 755', 'crp (mg/l): 252', 'ddimer (mg/l_feu): 34.04', 'procalcitonin (ng/ml): 1.33', 'lactate (mmol/l): 1.69', 'fibrinogen: 268', 'sofa: 8', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C41', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934976', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033839'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_41_74y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753061']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 74', 'Sex: male', 'icu: yes', 'apacheii: 30', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 15', 'dm: 0', 'hospital-free days post 45 day followup (days): 20', 'ferritin (ng/ml): 755', 'crp (mg/l): 252', 'ddimer (mg/l_feu): 34.04', 'procalcitonin (ng/ml): 1.33', 'lactate (mmol/l): 1.69', 'fibrinogen: 268', 'sofa: 8', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C41', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753061>\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934976', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033839']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753061>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753062 -- <SAMPLE: GSM4753062>\n",
      "metadata: \n",
      " {'title': ['COVID_42_21y_female_ICU'], 'geo_accession': ['GSM4753062'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 21', 'Sex: female', 'icu: yes', 'apacheii: 22', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 11', 'dm: 1', 'hospital-free days post 45 day followup (days): 28', 'ferritin (ng/ml): 383', 'crp (mg/l): 33.6', 'ddimer (mg/l_feu): 1.37', 'procalcitonin (ng/ml): 2', 'lactate (mmol/l): unknown', 'fibrinogen: 513', 'sofa: 13', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C42', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934975', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033840'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_42_21y_female_ICU']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753062']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 21', 'Sex: female', 'icu: yes', 'apacheii: 22', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 11', 'dm: 1', 'hospital-free days post 45 day followup (days): 28', 'ferritin (ng/ml): 383', 'crp (mg/l): 33.6', 'ddimer (mg/l_feu): 1.37', 'procalcitonin (ng/ml): 2', 'lactate (mmol/l): unknown', 'fibrinogen: 513', 'sofa: 13', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C42', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934975', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033840']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753062>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753063 -- <SAMPLE: GSM4753063>\n",
      "metadata: \n",
      " {'title': ['COVID_43_83y_female_ICU'], 'geo_accession': ['GSM4753063'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 83', 'Sex: female', 'icu: yes', 'apacheii: 25', 'charlson score: 4', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1114', 'crp (mg/l): 164.3', 'ddimer (mg/l_feu): 18.83', 'procalcitonin (ng/ml): 0.6', 'lactate (mmol/l): unknown', 'fibrinogen: 526', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C43', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934974', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033841'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_43_83y_female_ICU']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753063']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 83', 'Sex: female', 'icu: yes', 'apacheii: 25', 'charlson score: 4', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1114', 'crp (mg/l): 164.3', 'ddimer (mg/l_feu): 18.83', 'procalcitonin (ng/ml): 0.6', 'lactate (mmol/l): unknown', 'fibrinogen: 526', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C43', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934974', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033841']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753063>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753064 -- <SAMPLE: GSM4753064>\n",
      "metadata: \n",
      " {'title': ['COVID_44_46y_male_ICU'], 'geo_accession': ['GSM4753064'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 46', 'Sex: male', 'icu: yes', 'apacheii: 23', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 822', 'crp (mg/l): 235', 'ddimer (mg/l_feu): 12.81', 'procalcitonin (ng/ml): 2.89', 'lactate (mmol/l): 0.58', 'fibrinogen: 703', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C44', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934973', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033842'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_44_46y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753064']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 46', 'Sex: male', 'icu: yes', 'apacheii: 23', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 822', 'crp (mg/l): 235', 'ddimer (mg/l_feu): 12.81', 'procalcitonin (ng/ml): 2.89', 'lactate (mmol/l): 0.58', 'fibrinogen: 703', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C44', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934973', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033842']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753064>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753065 -- <SAMPLE: GSM4753065>\n",
      "metadata: \n",
      " {'title': ['COVID_45_62y_female_ICU'], 'geo_accession': ['GSM4753065'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 62', 'Sex: female', 'icu: yes', 'apacheii: 14', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 21', 'dm: 0', 'hospital-free days post 45 day followup (days): 20', 'ferritin (ng/ml): 505', 'crp (mg/l): 147.3', 'ddimer (mg/l_feu): 104.42', 'procalcitonin (ng/ml): 0.19', 'lactate (mmol/l): 1.53', 'fibrinogen: 479', 'sofa: 6', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C45', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934972', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033843'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_45_62y_female_ICU']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753065']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 62', 'Sex: female', 'icu: yes', 'apacheii: 14', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 21', 'dm: 0', 'hospital-free days post 45 day followup (days): 20', 'ferritin (ng/ml): 505', 'crp (mg/l): 147.3', 'ddimer (mg/l_feu): 104.42', 'procalcitonin (ng/ml): 0.19', 'lactate (mmol/l): 1.53', 'fibrinogen: 479', 'sofa: 6', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C45', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934972', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033843']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753065>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753066 -- <SAMPLE: GSM4753066>\n",
      "metadata: \n",
      " {'title': ['COVID_46_62y_male_ICU'], 'geo_accession': ['GSM4753066'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 62', 'Sex: male', 'icu: yes', 'apacheii: 15', 'charlson score: 4', 'mechanical ventilation: yes', 'ventilator-free days: 26', 'dm: 1', 'hospital-free days post 45 day followup (days): 39', 'ferritin (ng/ml): 378', 'crp (mg/l): 15.9', 'ddimer (mg/l_feu): 19.99', 'procalcitonin (ng/ml): 2.29', 'lactate (mmol/l): 0.87', 'fibrinogen: 378', 'sofa: 4', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C46', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934971', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033844'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_46_62y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753066']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 62', 'Sex: male', 'icu: yes', 'apacheii: 15', 'charlson score: 4', 'mechanical ventilation: yes', 'ventilator-free days: 26', 'dm: 1', 'hospital-free days post 45 day followup (days): 39', 'ferritin (ng/ml): 378', 'crp (mg/l): 15.9', 'ddimer (mg/l_feu): 19.99', 'procalcitonin (ng/ml): 2.29', 'lactate (mmol/l): 0.87', 'fibrinogen: 378', 'sofa: 4', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C46', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934971', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033844']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753066>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753067 -- <SAMPLE: GSM4753067>\n",
      "metadata: \n",
      " {'title': ['COVID_47_78y_male_ICU'], 'geo_accession': ['GSM4753067'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 78', 'Sex: male', 'icu: yes', 'apacheii: 21', 'charlson score: 5', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1102', 'crp (mg/l): 166.3', 'ddimer (mg/l_feu): 56.27', 'procalcitonin (ng/ml): 0.43', 'lactate (mmol/l): 1.03', 'fibrinogen: 438', 'sofa: 8', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C47', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934970', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033845'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_47_78y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753067']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 78', 'Sex: male', 'icu: yes', 'apacheii: 21', 'charlson score: 5', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1102', 'crp (mg/l): 166.3', 'ddimer (mg/l_feu): 56.27', 'procalcitonin (ng/ml): 0.43', 'lactate (mmol/l): 1.03', 'fibrinogen: 438', 'sofa: 8', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C47', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934970', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033845']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753067>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753068 -- <SAMPLE: GSM4753068>\n",
      "metadata: \n",
      " {'title': ['COVID_48_72y_female_ICU'], 'geo_accession': ['GSM4753068'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 72', 'Sex: female', 'icu: yes', 'apacheii: 33', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 301', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): 51.7', 'procalcitonin (ng/ml): 1.01', 'lactate (mmol/l): 1.3', 'fibrinogen: unknown', 'sofa: 9', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C48', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934969', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033846'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_48_72y_female_ICU']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753068']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 72', 'Sex: female', 'icu: yes', 'apacheii: 33', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 301', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): 51.7', 'procalcitonin (ng/ml): 1.01', 'lactate (mmol/l): 1.3', 'fibrinogen: unknown', 'sofa: 9', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C48', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934969', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033846']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753068>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753069 -- <SAMPLE: GSM4753069>\n",
      "metadata: \n",
      " {'title': ['COVID_49_73y_male_ICU'], 'geo_accession': ['GSM4753069'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 73', 'Sex: male', 'icu: yes', 'apacheii: 18', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1320', 'crp (mg/l): 260.6', 'ddimer (mg/l_feu): 102.75', 'procalcitonin (ng/ml): 0.26', 'lactate (mmol/l): 1.3', 'fibrinogen: 404', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C49', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934968', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033847'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_49_73y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753069']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 73', 'Sex: male', 'icu: yes', 'apacheii: 18', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1320', 'crp (mg/l): 260.6', 'ddimer (mg/l_feu): 102.75', 'procalcitonin (ng/ml): 0.26', 'lactate (mmol/l): 1.3', 'fibrinogen: 404', 'sofa: 7', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C49', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934968', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033847']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753069>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753070 -- <SAMPLE: GSM4753070>\n",
      "metadata: \n",
      " {'title': ['COVID_50_37y_male_ICU'], 'geo_accession': ['GSM4753070'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 37', 'Sex: male', 'icu: yes', 'apacheii: 34', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 4609', 'crp (mg/l): 71', 'ddimer (mg/l_feu): 18.88', 'procalcitonin (ng/ml): 2.31', 'lactate (mmol/l): 1.32', 'fibrinogen: 305', 'sofa: 18', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C50', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934967', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033848'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_50_37y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753070']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 37', 'Sex: male', 'icu: yes', 'apacheii: 34', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 4609', 'crp (mg/l): 71', 'ddimer (mg/l_feu): 18.88', 'procalcitonin (ng/ml): 2.31', 'lactate (mmol/l): 1.32', 'fibrinogen: 305', 'sofa: 18', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C50', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934967', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033848']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753070>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753071 -- <SAMPLE: GSM4753071>\n",
      "metadata: \n",
      " {'title': ['COVID_51_58y_female_NonICU'], 'geo_accession': ['GSM4753071'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 58', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 6', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 38', 'ferritin (ng/ml): 1859', 'crp (mg/l): 299.8', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 1.46', 'lactate (mmol/l): 1.28', 'fibrinogen: 910', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C51', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934966', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033849'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_51_58y_female_NonICU']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753071']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 58', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 6', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 38', 'ferritin (ng/ml): 1859', 'crp (mg/l): 299.8', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 1.46', 'lactate (mmol/l): 1.28', 'fibrinogen: 910', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C51', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934966', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033849']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753071>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753072 -- <SAMPLE: GSM4753072>\n",
      "metadata: \n",
      " {'title': ['COVID_52_71y_male_NonICU'], 'geo_accession': ['GSM4753072'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 71', 'Sex: male', 'icu: no', 'apacheii: 27', 'charlson score: 7', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): 1.67', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: 458', 'sofa: 12', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C52', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934965', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033850'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_52_71y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753072']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 71', 'Sex: male', 'icu: no', 'apacheii: 27', 'charlson score: 7', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): 1.67', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: 458', 'sofa: 12', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C52', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934965', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033850']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753072>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753073 -- <SAMPLE: GSM4753073>\n",
      "metadata: \n",
      " {'title': ['COVID_53_35y_female_NonICU'], 'geo_accession': ['GSM4753073'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 35', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 41', 'ferritin (ng/ml): 61', 'crp (mg/l): 38', 'ddimer (mg/l_feu): 1.35', 'procalcitonin (ng/ml): 0.17', 'lactate (mmol/l): 0.9', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C53', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934964', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033851'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_53_35y_female_NonICU']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753073']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 35', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 41', 'ferritin (ng/ml): 61', 'crp (mg/l): 38', 'ddimer (mg/l_feu): 1.35', 'procalcitonin (ng/ml): 0.17', 'lactate (mmol/l): 0.9', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C53', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934964', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033851']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753073>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753074 -- <SAMPLE: GSM4753074>\n",
      "metadata: \n",
      " {'title': ['COVID_55_62y_female_ICU'], 'geo_accession': ['GSM4753074'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 62', 'Sex: female', 'icu: yes', 'apacheii: 39', 'charlson score: 7', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1035', 'crp (mg/l): 256.5', 'ddimer (mg/l_feu): 22.5', 'procalcitonin (ng/ml): 13.91', 'lactate (mmol/l): 1.52', 'fibrinogen: 654', 'sofa: 12', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C55', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934963', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033852'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_55_62y_female_ICU']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753074']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 62', 'Sex: female', 'icu: yes', 'apacheii: 39', 'charlson score: 7', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1035', 'crp (mg/l): 256.5', 'ddimer (mg/l_feu): 22.5', 'procalcitonin (ng/ml): 13.91', 'lactate (mmol/l): 1.52', 'fibrinogen: 654', 'sofa: 12', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C55', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934963', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033852']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753074>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753075 -- <SAMPLE: GSM4753075>\n",
      "metadata: \n",
      " {'title': ['COVID_56_33y_female_NonICU'], 'geo_accession': ['GSM4753075'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 33', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 39', 'ferritin (ng/ml): 91', 'crp (mg/l): 154.8', 'ddimer (mg/l_feu): 1.28', 'procalcitonin (ng/ml): 0.2', 'lactate (mmol/l): unknown', 'fibrinogen: 568', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C56', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934962', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033853'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_56_33y_female_NonICU']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753075']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 33', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 39', 'ferritin (ng/ml): 91', 'crp (mg/l): 154.8', 'ddimer (mg/l_feu): 1.28', 'procalcitonin (ng/ml): 0.2', 'lactate (mmol/l): unknown', 'fibrinogen: 568', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C56', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934962', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033853']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753075>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753076 -- <SAMPLE: GSM4753076>\n",
      "metadata: \n",
      " {'title': ['COVID_57_30y_female_NonICU'], 'geo_accession': ['GSM4753076'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 30', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 43', 'ferritin (ng/ml): 32', 'crp (mg/l): 19.4', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 0.66', 'fibrinogen: 420', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C57', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934958', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033854'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_57_30y_female_NonICU']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753076']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 30', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 43', 'ferritin (ng/ml): 32', 'crp (mg/l): 19.4', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 0.66', 'fibrinogen: 420', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C57', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934958', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033854']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753076>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753077 -- <SAMPLE: GSM4753077>\n",
      "metadata: \n",
      " {'title': ['COVID_58_62y_male_NonICU'], 'geo_accession': ['GSM4753077'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 62', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 6', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 44', 'ferritin (ng/ml): 228', 'crp (mg/l): 141.3', 'ddimer (mg/l_feu): 1.79', 'procalcitonin (ng/ml): 3.22', 'lactate (mmol/l): 0.77', 'fibrinogen: 703', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C58', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935042', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033855'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_58_62y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753077']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 62', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 6', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 44', 'ferritin (ng/ml): 228', 'crp (mg/l): 141.3', 'ddimer (mg/l_feu): 1.79', 'procalcitonin (ng/ml): 3.22', 'lactate (mmol/l): 0.77', 'fibrinogen: 703', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C58', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935042', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033855']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753077>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753078 -- <SAMPLE: GSM4753078>\n",
      "metadata: \n",
      " {'title': ['COVID_59_55y_male_NonICU'], 'geo_accession': ['GSM4753078'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 55', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 43', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C59', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934926', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033856'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_59_55y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753078']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 55', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 43', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C59', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934926', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033856']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753078>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753079 -- <SAMPLE: GSM4753079>\n",
      "metadata: \n",
      " {'title': ['COVID_60_49y_male_NonICU'], 'geo_accession': ['GSM4753079'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 49', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 43', 'ferritin (ng/ml): 825', 'crp (mg/l): 13.7', 'ddimer (mg/l_feu): 2.21', 'procalcitonin (ng/ml): 0.57', 'lactate (mmol/l): 0.65', 'fibrinogen: 874', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C60', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934925', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033857'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_60_49y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753079']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 49', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 43', 'ferritin (ng/ml): 825', 'crp (mg/l): 13.7', 'ddimer (mg/l_feu): 2.21', 'procalcitonin (ng/ml): 0.57', 'lactate (mmol/l): 0.65', 'fibrinogen: 874', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C60', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934925', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033857']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753079>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753080 -- <SAMPLE: GSM4753080>\n",
      "metadata: \n",
      " {'title': ['COVID_61_54y_female_NonICU'], 'geo_accession': ['GSM4753080'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 54', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 38', 'ferritin (ng/ml): 206', 'crp (mg/l): 114', 'ddimer (mg/l_feu): 0.63', 'procalcitonin (ng/ml): 0.07', 'lactate (mmol/l): unknown', 'fibrinogen: 617', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C61', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934924', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033858'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_61_54y_female_NonICU']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753080']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 54', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 38', 'ferritin (ng/ml): 206', 'crp (mg/l): 114', 'ddimer (mg/l_feu): 0.63', 'procalcitonin (ng/ml): 0.07', 'lactate (mmol/l): unknown', 'fibrinogen: 617', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C61', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934924', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033858']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753080>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753081 -- <SAMPLE: GSM4753081>\n",
      "metadata: \n",
      " {'title': ['COVID_62_78y_female_ICU'], 'geo_accession': ['GSM4753081'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 78', 'Sex: female', 'icu: yes', 'apacheii: 22', 'charlson score: 9', 'mechanical ventilation: no', 'ventilator-free days: 0', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 5508', 'crp (mg/l): 138.6', 'ddimer (mg/l_feu): 0.86', 'procalcitonin (ng/ml): 1.02', 'lactate (mmol/l): 1.17', 'fibrinogen: 449', 'sofa: 6', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C62', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934961', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033859'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_62_78y_female_ICU']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753081']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 78', 'Sex: female', 'icu: yes', 'apacheii: 22', 'charlson score: 9', 'mechanical ventilation: no', 'ventilator-free days: 0', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 5508', 'crp (mg/l): 138.6', 'ddimer (mg/l_feu): 0.86', 'procalcitonin (ng/ml): 1.02', 'lactate (mmol/l): 1.17', 'fibrinogen: 449', 'sofa: 6', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C62', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934961', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033859']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753081>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753082 -- <SAMPLE: GSM4753082>\n",
      "metadata: \n",
      " {'title': ['COVID_63_39y_female_ICU'], 'geo_accession': ['GSM4753082'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 39', 'Sex: female', 'icu: yes', 'apacheii: 14', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 18', 'dm: 0', 'hospital-free days post 45 day followup (days): 20', 'ferritin (ng/ml): 77', 'crp (mg/l): 340.6', 'ddimer (mg/l_feu): 10.48', 'procalcitonin (ng/ml): 0.08', 'lactate (mmol/l): 1.23', 'fibrinogen: 797', 'sofa: 8', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C63', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934960', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033860'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_63_39y_female_ICU']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753082']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 39', 'Sex: female', 'icu: yes', 'apacheii: 14', 'charlson score: 1', 'mechanical ventilation: yes', 'ventilator-free days: 18', 'dm: 0', 'hospital-free days post 45 day followup (days): 20', 'ferritin (ng/ml): 77', 'crp (mg/l): 340.6', 'ddimer (mg/l_feu): 10.48', 'procalcitonin (ng/ml): 0.08', 'lactate (mmol/l): 1.23', 'fibrinogen: 797', 'sofa: 8', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C63', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934960', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033860']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753082>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753083 -- <SAMPLE: GSM4753083>\n",
      "metadata: \n",
      " {'title': ['COVID_64_65y_male_ICU'], 'geo_accession': ['GSM4753083'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 65', 'Sex: male', 'icu: yes', 'apacheii: 23', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 9', 'dm: 0', 'hospital-free days post 45 day followup (days): 16', 'ferritin (ng/ml): 484', 'crp (mg/l): 8.7', 'ddimer (mg/l_feu): 1.3', 'procalcitonin (ng/ml): 0.79', 'lactate (mmol/l): 0.73', 'fibrinogen: 414', 'sofa: 6', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C64', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934959', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033861'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_64_65y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753083']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 65', 'Sex: male', 'icu: yes', 'apacheii: 23', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 9', 'dm: 0', 'hospital-free days post 45 day followup (days): 16', 'ferritin (ng/ml): 484', 'crp (mg/l): 8.7', 'ddimer (mg/l_feu): 1.3', 'procalcitonin (ng/ml): 0.79', 'lactate (mmol/l): 0.73', 'fibrinogen: 414', 'sofa: 6', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C64', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934959', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033861']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753083>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753084 -- <SAMPLE: GSM4753084>\n",
      "metadata: \n",
      " {'title': ['COVID_65_84y_male_NonICU'], 'geo_accession': ['GSM4753084'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 84', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 33', 'ferritin (ng/ml): 220', 'crp (mg/l): 128', 'ddimer (mg/l_feu): 0.4', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 1.47', 'fibrinogen: 797', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C65', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934923', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033862'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_65_84y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753084']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 84', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 33', 'ferritin (ng/ml): 220', 'crp (mg/l): 128', 'ddimer (mg/l_feu): 0.4', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 1.47', 'fibrinogen: 797', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C65', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934923', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033862']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753084>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753085 -- <SAMPLE: GSM4753085>\n",
      "metadata: \n",
      " {'title': ['COVID_66_66y_female_NonICU'], 'geo_accession': ['GSM4753085'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 66', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 6', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 22', 'ferritin (ng/ml): 1397', 'crp (mg/l): 8.5', 'ddimer (mg/l_feu): 2.22', 'procalcitonin (ng/ml): 0.16', 'lactate (mmol/l): unknown', 'fibrinogen: 478', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C66', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934922', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033863'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_66_66y_female_NonICU']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753085']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 66', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 6', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 22', 'ferritin (ng/ml): 1397', 'crp (mg/l): 8.5', 'ddimer (mg/l_feu): 2.22', 'procalcitonin (ng/ml): 0.16', 'lactate (mmol/l): unknown', 'fibrinogen: 478', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C66', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934922', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033863']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753085>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753086 -- <SAMPLE: GSM4753086>\n",
      "metadata: \n",
      " {'title': ['COVID_67_57y_male_ICU'], 'geo_accession': ['GSM4753086'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 57', 'Sex: male', 'icu: yes', 'apacheii: 20', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 24', 'dm: 0', 'hospital-free days post 45 day followup (days): 37', 'ferritin (ng/ml): 708', 'crp (mg/l): 191.7', 'ddimer (mg/l_feu): 6.03', 'procalcitonin (ng/ml): 0.19', 'lactate (mmol/l): 0.74', 'fibrinogen: 526', 'sofa: 8', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C67', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934921', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033864'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_67_57y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753086']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 57', 'Sex: male', 'icu: yes', 'apacheii: 20', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 24', 'dm: 0', 'hospital-free days post 45 day followup (days): 37', 'ferritin (ng/ml): 708', 'crp (mg/l): 191.7', 'ddimer (mg/l_feu): 6.03', 'procalcitonin (ng/ml): 0.19', 'lactate (mmol/l): 0.74', 'fibrinogen: 526', 'sofa: 8', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C67', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934921', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033864']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753086>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753087 -- <SAMPLE: GSM4753087>\n",
      "metadata: \n",
      " {'title': ['COVID_68_79y_male_ICU'], 'geo_accession': ['GSM4753087'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 79', 'Sex: male', 'icu: yes', 'apacheii: 10', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 17', 'ferritin (ng/ml): 1953', 'crp (mg/l): 269.1', 'ddimer (mg/l_feu): 0.59', 'procalcitonin (ng/ml): 1.31', 'lactate (mmol/l): 0.66', 'fibrinogen: 727', 'sofa: 3', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C68', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934920', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033865'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_68_79y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753087']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 79', 'Sex: male', 'icu: yes', 'apacheii: 10', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 17', 'ferritin (ng/ml): 1953', 'crp (mg/l): 269.1', 'ddimer (mg/l_feu): 0.59', 'procalcitonin (ng/ml): 1.31', 'lactate (mmol/l): 0.66', 'fibrinogen: 727', 'sofa: 3', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C68', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934920', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033865']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753087>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753088 -- <SAMPLE: GSM4753088>\n",
      "metadata: \n",
      " {'title': ['COVID_69_77y_female_NonICU'], 'geo_accession': ['GSM4753088'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 77', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 42', 'ferritin (ng/ml): 206', 'crp (mg/l): 124.2', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: 501', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C69', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934919', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033866'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_69_77y_female_NonICU']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753088']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 77', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 42', 'ferritin (ng/ml): 206', 'crp (mg/l): 124.2', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: 501', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C69', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934919', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033866']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753088>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753089 -- <SAMPLE: GSM4753089>\n",
      "metadata: \n",
      " {'title': ['COVID_70_81y_male_NonICU'], 'geo_accession': ['GSM4753089'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 81', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 29', 'ferritin (ng/ml): 900', 'crp (mg/l): 127.7', 'ddimer (mg/l_feu): 0.54', 'procalcitonin (ng/ml): 3.65', 'lactate (mmol/l): 1.05', 'fibrinogen: 489', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C70', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934918', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033867'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_70_81y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753089']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 81', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 29', 'ferritin (ng/ml): 900', 'crp (mg/l): 127.7', 'ddimer (mg/l_feu): 0.54', 'procalcitonin (ng/ml): 3.65', 'lactate (mmol/l): 1.05', 'fibrinogen: 489', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C70', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934918', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033867']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753089>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753090 -- <SAMPLE: GSM4753090>\n",
      "metadata: \n",
      " {'title': ['COVID_71_37y_male_ICU'], 'geo_accession': ['GSM4753090'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 37', 'Sex: male', 'icu: yes', 'apacheii: 28', 'charlson score: 0', 'mechanical ventilation: yes', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 21', 'ferritin (ng/ml): 1356', 'crp (mg/l): 128.4', 'ddimer (mg/l_feu): 2.21', 'procalcitonin (ng/ml): 4.97', 'lactate (mmol/l): 0.8', 'fibrinogen: 680', 'sofa: 11', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C71', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934917', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033868'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_71_37y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753090']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753090>\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 37', 'Sex: male', 'icu: yes', 'apacheii: 28', 'charlson score: 0', 'mechanical ventilation: yes', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 21', 'ferritin (ng/ml): 1356', 'crp (mg/l): 128.4', 'ddimer (mg/l_feu): 2.21', 'procalcitonin (ng/ml): 4.97', 'lactate (mmol/l): 0.8', 'fibrinogen: 680', 'sofa: 11', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C71', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934917', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033868']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753090>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753091 -- <SAMPLE: GSM4753091>\n",
      "metadata: \n",
      " {'title': ['COVID_72_50y_female_NonICU'], 'geo_accession': ['GSM4753091'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 50', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 39', 'ferritin (ng/ml): 14', 'crp (mg/l): 23.6', 'ddimer (mg/l_feu): 0.61', 'procalcitonin (ng/ml): 0.5', 'lactate (mmol/l): unknown', 'fibrinogen: 406', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C72', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934916', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033869'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_72_50y_female_NonICU']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753091']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 50', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 39', 'ferritin (ng/ml): 14', 'crp (mg/l): 23.6', 'ddimer (mg/l_feu): 0.61', 'procalcitonin (ng/ml): 0.5', 'lactate (mmol/l): unknown', 'fibrinogen: 406', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C72', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934916', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033869']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753091>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753092 -- <SAMPLE: GSM4753092>\n",
      "metadata: \n",
      " {'title': ['COVID_73_82y_male_NonICU'], 'geo_accession': ['GSM4753092'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 82', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 31', 'ferritin (ng/ml): 629', 'crp (mg/l): 14.3', 'ddimer (mg/l_feu): 0.92', 'procalcitonin (ng/ml): 0.05', 'lactate (mmol/l): unknown', 'fibrinogen: 362', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C73', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935011', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033870'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_73_82y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753092']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 82', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 31', 'ferritin (ng/ml): 629', 'crp (mg/l): 14.3', 'ddimer (mg/l_feu): 0.92', 'procalcitonin (ng/ml): 0.05', 'lactate (mmol/l): unknown', 'fibrinogen: 362', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C73', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935011', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033870']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753092>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753093 -- <SAMPLE: GSM4753093>\n",
      "metadata: \n",
      " {'title': ['COVID_74_55y_female_ICU'], 'geo_accession': ['GSM4753093'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 55', 'Sex: female', 'icu: yes', 'apacheii: 22', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 18', 'dm: 1', 'hospital-free days post 45 day followup (days): 30', 'ferritin (ng/ml): 186', 'crp (mg/l): 25.3', 'ddimer (mg/l_feu): 6.53', 'procalcitonin (ng/ml): 1.88', 'lactate (mmol/l): 1.33', 'fibrinogen: 312', 'sofa: 11', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C74', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935010', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033871'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_74_55y_female_ICU']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753093']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 55', 'Sex: female', 'icu: yes', 'apacheii: 22', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 18', 'dm: 1', 'hospital-free days post 45 day followup (days): 30', 'ferritin (ng/ml): 186', 'crp (mg/l): 25.3', 'ddimer (mg/l_feu): 6.53', 'procalcitonin (ng/ml): 1.88', 'lactate (mmol/l): 1.33', 'fibrinogen: 312', 'sofa: 11', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C74', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935010', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033871']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753093>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753094 -- <SAMPLE: GSM4753094>\n",
      "metadata: \n",
      " {'title': ['COVID_75_55y_male_NonICU'], 'geo_accession': ['GSM4753094'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 55', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 39', 'ferritin (ng/ml): 5971', 'crp (mg/l): 112.8', 'ddimer (mg/l_feu): 0.66', 'procalcitonin (ng/ml): 0.21', 'lactate (mmol/l): unknown', 'fibrinogen: 501', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C75', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935009', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033872'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_75_55y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753094']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 55', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 39', 'ferritin (ng/ml): 5971', 'crp (mg/l): 112.8', 'ddimer (mg/l_feu): 0.66', 'procalcitonin (ng/ml): 0.21', 'lactate (mmol/l): unknown', 'fibrinogen: 501', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C75', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935009', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033872']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753094>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753095 -- <SAMPLE: GSM4753095>\n",
      "metadata: \n",
      " {'title': ['COVID_76_73y_female_ICU'], 'geo_accession': ['GSM4753095'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 73', 'Sex: female', 'icu: yes', 'apacheii: 35', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 685', 'crp (mg/l): 326.6', 'ddimer (mg/l_feu): 21.78', 'procalcitonin (ng/ml): 16.51', 'lactate (mmol/l): unknown', 'fibrinogen: 490', 'sofa: 19', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C76', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935008', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033873'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_76_73y_female_ICU']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753095']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 73', 'Sex: female', 'icu: yes', 'apacheii: 35', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 685', 'crp (mg/l): 326.6', 'ddimer (mg/l_feu): 21.78', 'procalcitonin (ng/ml): 16.51', 'lactate (mmol/l): unknown', 'fibrinogen: 490', 'sofa: 19', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C76', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935008', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033873']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753095>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753096 -- <SAMPLE: GSM4753096>\n",
      "metadata: \n",
      " {'title': ['COVID_77_55y_female_ICU'], 'geo_accession': ['GSM4753096'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 55', 'Sex: female', 'icu: yes', 'apacheii: 6', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 26', 'ferritin (ng/ml): 1223', 'crp (mg/l): 84.5', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: 2', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C77', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935007', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033874'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_77_55y_female_ICU']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753096']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 55', 'Sex: female', 'icu: yes', 'apacheii: 6', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 26', 'ferritin (ng/ml): 1223', 'crp (mg/l): 84.5', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: 2', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C77', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935007', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033874']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753096>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753097 -- <SAMPLE: GSM4753097>\n",
      "metadata: \n",
      " {'title': ['COVID_78_80y_male_NonICU'], 'geo_accession': ['GSM4753097'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 80', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 31', 'ferritin (ng/ml): 635', 'crp (mg/l): 430.5', 'ddimer (mg/l_feu): 2.56', 'procalcitonin (ng/ml): 0.36', 'lactate (mmol/l): 1.26', 'fibrinogen: 639', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C78', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935006', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033875'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_78_80y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753097']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 80', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 31', 'ferritin (ng/ml): 635', 'crp (mg/l): 430.5', 'ddimer (mg/l_feu): 2.56', 'procalcitonin (ng/ml): 0.36', 'lactate (mmol/l): 1.26', 'fibrinogen: 639', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C78', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935006', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033875']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753097>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753098 -- <SAMPLE: GSM4753098>\n",
      "metadata: \n",
      " {'title': ['COVID_79_27y_male_NonICU'], 'geo_accession': ['GSM4753098'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 27', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 40', 'ferritin (ng/ml): 112', 'crp (mg/l): 1', 'ddimer (mg/l_feu): 0.8', 'procalcitonin (ng/ml): 0.13', 'lactate (mmol/l): unknown', 'fibrinogen: 215', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C79', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935005', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033876'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_79_27y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753098']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 27', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 40', 'ferritin (ng/ml): 112', 'crp (mg/l): 1', 'ddimer (mg/l_feu): 0.8', 'procalcitonin (ng/ml): 0.13', 'lactate (mmol/l): unknown', 'fibrinogen: 215', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C79', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935005', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033876']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753098>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753099 -- <SAMPLE: GSM4753099>\n",
      "metadata: \n",
      " {'title': ['COVID_80_71y_male_ICU'], 'geo_accession': ['GSM4753099'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 71', 'Sex: male', 'icu: yes', 'apacheii: 22', 'charlson score: 7', 'mechanical ventilation: no', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1200', 'crp (mg/l): 122.5', 'ddimer (mg/l_feu): 1.7', 'procalcitonin (ng/ml): 0.7', 'lactate (mmol/l): 1.88', 'fibrinogen: 489', 'sofa: 4', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C80', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935004', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033877'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_80_71y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753099']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 71', 'Sex: male', 'icu: yes', 'apacheii: 22', 'charlson score: 7', 'mechanical ventilation: no', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1200', 'crp (mg/l): 122.5', 'ddimer (mg/l_feu): 1.7', 'procalcitonin (ng/ml): 0.7', 'lactate (mmol/l): 1.88', 'fibrinogen: 489', 'sofa: 4', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C80', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935004', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033877']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753099>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753100 -- <SAMPLE: GSM4753100>\n",
      "metadata: \n",
      " {'title': ['COVID_82_67y_male_NonICU'], 'geo_accession': ['GSM4753100'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 67', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 41', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C82', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935003', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033878'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " title ['COVID_82_67y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753100']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 67', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 41', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C82', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935003', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033878']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753100>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753101 -- <SAMPLE: GSM4753101>\n",
      "metadata: \n",
      " {'title': ['COVID_83_85y_female_NonICU'], 'geo_accession': ['GSM4753101'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 85', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 133', 'crp (mg/l): 97.6', 'ddimer (mg/l_feu): 0.85', 'procalcitonin (ng/ml): 0.5', 'lactate (mmol/l): 0.84', 'fibrinogen: 296', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C83', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935002', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033879'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_83_85y_female_NonICU']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753101']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 85', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 133', 'crp (mg/l): 97.6', 'ddimer (mg/l_feu): 0.85', 'procalcitonin (ng/ml): 0.5', 'lactate (mmol/l): 0.84', 'fibrinogen: 296', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C83', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935002', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033879']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753101>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753102 -- <SAMPLE: GSM4753102>\n",
      "metadata: \n",
      " {'title': ['COVID_84_75y_female_NonICU'], 'geo_accession': ['GSM4753102'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 75', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 6', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 19', 'ferritin (ng/ml): 840', 'crp (mg/l): 153.5', 'ddimer (mg/l_feu): 4.2', 'procalcitonin (ng/ml): 2.66', 'lactate (mmol/l): 0.83', 'fibrinogen: 292', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C84', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935001', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033880'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_84_75y_female_NonICU']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753102']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 75', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 6', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 19', 'ferritin (ng/ml): 840', 'crp (mg/l): 153.5', 'ddimer (mg/l_feu): 4.2', 'procalcitonin (ng/ml): 2.66', 'lactate (mmol/l): 0.83', 'fibrinogen: 292', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753102>\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C84', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935001', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033880']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753102>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753103 -- <SAMPLE: GSM4753103>\n",
      "metadata: \n",
      " {'title': ['COVID_85_62y_male_ICU'], 'geo_accession': ['GSM4753103'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 62', 'Sex: male', 'icu: yes', 'apacheii: 12', 'charlson score: 3', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 22', 'ferritin (ng/ml): 423', 'crp (mg/l): 54.9', 'ddimer (mg/l_feu): 4.83', 'procalcitonin (ng/ml): 0.11', 'lactate (mmol/l): 0.5', 'fibrinogen: 396', 'sofa: 3', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C85', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935000', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033881'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_85_62y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753103']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 62', 'Sex: male', 'icu: yes', 'apacheii: 12', 'charlson score: 3', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 22', 'ferritin (ng/ml): 423', 'crp (mg/l): 54.9', 'ddimer (mg/l_feu): 4.83', 'procalcitonin (ng/ml): 0.11', 'lactate (mmol/l): 0.5', 'fibrinogen: 396', 'sofa: 3', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C85', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935000', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033881']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753103>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753104 -- <SAMPLE: GSM4753104>\n",
      "metadata: \n",
      " {'title': ['COVID_86_52y_female_NonICU'], 'geo_accession': ['GSM4753104'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 52', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 42', 'ferritin (ng/ml): 352', 'crp (mg/l): 145.7', 'ddimer (mg/l_feu): 0.22', 'procalcitonin (ng/ml): 0.05', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C86', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934999', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033882'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_86_52y_female_NonICU']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753104']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 52', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 42', 'ferritin (ng/ml): 352', 'crp (mg/l): 145.7', 'ddimer (mg/l_feu): 0.22', 'procalcitonin (ng/ml): 0.05', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C86', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934999', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033882']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753104>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753105 -- <SAMPLE: GSM4753105>\n",
      "metadata: \n",
      " {'title': ['COVID_87_61y_male_ICU'], 'geo_accession': ['GSM4753105'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 61', 'Sex: male', 'icu: yes', 'apacheii: 7', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 39', 'ferritin (ng/ml): 170', 'crp (mg/l): 2.8', 'ddimer (mg/l_feu): 1.55', 'procalcitonin (ng/ml): 0.5', 'lactate (mmol/l): 1.13', 'fibrinogen: 208', 'sofa: 2', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C87', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934998', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033883'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_87_61y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753105']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 61', 'Sex: male', 'icu: yes', 'apacheii: 7', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 39', 'ferritin (ng/ml): 170', 'crp (mg/l): 2.8', 'ddimer (mg/l_feu): 1.55', 'procalcitonin (ng/ml): 0.5', 'lactate (mmol/l): 1.13', 'fibrinogen: 208', 'sofa: 2', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C87', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934998', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033883']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753105>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753106 -- <SAMPLE: GSM4753106>\n",
      "metadata: \n",
      " {'title': ['COVID_89_90y_female_NonICU'], 'geo_accession': ['GSM4753106'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): >89', 'Sex: female', 'icu: no', 'apacheii: 17', 'charlson score: 7', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 33', 'ferritin (ng/ml): 38', 'crp (mg/l): 4.7', 'ddimer (mg/l_feu): 0.96', 'procalcitonin (ng/ml): 0.5', 'lactate (mmol/l): 1.09', 'fibrinogen: 292', 'sofa: 3', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C89', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934997', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033884'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_89_90y_female_NonICU']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753106']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): >89', 'Sex: female', 'icu: no', 'apacheii: 17', 'charlson score: 7', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 33', 'ferritin (ng/ml): 38', 'crp (mg/l): 4.7', 'ddimer (mg/l_feu): 0.96', 'procalcitonin (ng/ml): 0.5', 'lactate (mmol/l): 1.09', 'fibrinogen: 292', 'sofa: 3', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C89', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934997', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033884']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753106>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753107 -- <SAMPLE: GSM4753107>\n",
      "metadata: \n",
      " {'title': ['COVID_90_86y_female_NonICU'], 'geo_accession': ['GSM4753107'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 86', 'Sex: female', 'icu: no', 'apacheii: 11', 'charlson score: 8', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 24', 'ferritin (ng/ml): 288', 'crp (mg/l): 43.7', 'ddimer (mg/l_feu): 15.48', 'procalcitonin (ng/ml): 0.06', 'lactate (mmol/l): unknown', 'fibrinogen: 635', 'sofa: 2', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C90', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934996', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033885'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_90_86y_female_NonICU']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753107']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 86', 'Sex: female', 'icu: no', 'apacheii: 11', 'charlson score: 8', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 24', 'ferritin (ng/ml): 288', 'crp (mg/l): 43.7', 'ddimer (mg/l_feu): 15.48', 'procalcitonin (ng/ml): 0.06', 'lactate (mmol/l): unknown', 'fibrinogen: 635', 'sofa: 2', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C90', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934996', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033885']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753107>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753108 -- <SAMPLE: GSM4753108>\n",
      "metadata: \n",
      " {'title': ['COVID_91_29y_female_NonICU'], 'geo_accession': ['GSM4753108'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 29', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 40', 'ferritin (ng/ml): unknown', 'crp (mg/l): 1.8', 'ddimer (mg/l_feu): 0.28', 'procalcitonin (ng/ml): 2.5', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C91', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934995', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033886'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_91_29y_female_NonICU']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753108']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 29', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 1', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 40', 'ferritin (ng/ml): unknown', 'crp (mg/l): 1.8', 'ddimer (mg/l_feu): 0.28', 'procalcitonin (ng/ml): 2.5', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C91', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934995', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033886']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753108>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753109 -- <SAMPLE: GSM4753109>\n",
      "metadata: \n",
      " {'title': ['COVID_92_82y_female_ICU'], 'geo_accession': ['GSM4753109'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 82', 'Sex: female', 'icu: yes', 'apacheii: 23', 'charlson score: 7', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 111', 'crp (mg/l): 26.6', 'ddimer (mg/l_feu): 1.22', 'procalcitonin (ng/ml): 0.1', 'lactate (mmol/l): 0.97', 'fibrinogen: 489', 'sofa: 8', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C92', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934994', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033887'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_92_82y_female_ICU']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753109']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 82', 'Sex: female', 'icu: yes', 'apacheii: 23', 'charlson score: 7', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 111', 'crp (mg/l): 26.6', 'ddimer (mg/l_feu): 1.22', 'procalcitonin (ng/ml): 0.1', 'lactate (mmol/l): 0.97', 'fibrinogen: 489', 'sofa: 8', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C92', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934994', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033887']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753109>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753110 -- <SAMPLE: GSM4753110>\n",
      "metadata: \n",
      " {'title': ['COVID_93_81y_female_ICU'], 'geo_accession': ['GSM4753110'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 81', 'Sex: female', 'icu: yes', 'apacheii: 14', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 0', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 388', 'crp (mg/l): 17.2', 'ddimer (mg/l_feu): 1.25', 'procalcitonin (ng/ml): 0.07', 'lactate (mmol/l): 1.88', 'fibrinogen: 289', 'sofa: 6', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C93', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934993', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033888'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_93_81y_female_ICU']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753110']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 81', 'Sex: female', 'icu: yes', 'apacheii: 14', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 0', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 388', 'crp (mg/l): 17.2', 'ddimer (mg/l_feu): 1.25', 'procalcitonin (ng/ml): 0.07', 'lactate (mmol/l): 1.88', 'fibrinogen: 289', 'sofa: 6', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C93', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934993', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033888']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753110>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753111 -- <SAMPLE: GSM4753111>\n",
      "metadata: \n",
      " {'title': ['COVID_94_24y_female_NonICU'], 'geo_accession': ['GSM4753111'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 24', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 43', 'ferritin (ng/ml): 52', 'crp (mg/l): 39', 'ddimer (mg/l_feu): 0.67', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: 412', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C94', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935041', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033889'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_94_24y_female_NonICU']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753111']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 24', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 43', 'ferritin (ng/ml): 52', 'crp (mg/l): 39', 'ddimer (mg/l_feu): 0.67', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: 412', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C94', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935041', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033889']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753111>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753112 -- <SAMPLE: GSM4753112>\n",
      "metadata: \n",
      " {'title': ['COVID_95_49y_male_NonICU'], 'geo_accession': ['GSM4753112'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 49', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 33', 'ferritin (ng/ml): 386', 'crp (mg/l): 193.6', 'ddimer (mg/l_feu): 1.3', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: 752', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C95', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935040', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033890'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_95_49y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753112']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 49', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 33', 'ferritin (ng/ml): 386', 'crp (mg/l): 193.6', 'ddimer (mg/l_feu): 1.3', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: 752', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C95', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935040', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033890']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753112>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753113 -- <SAMPLE: GSM4753113>\n",
      "metadata: \n",
      " {'title': ['COVID_96_51y_male_NonICU'], 'geo_accession': ['GSM4753113'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 51', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 3', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 41', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C96', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935039', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033891'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_96_51y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753113']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 51', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 3', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 41', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C96', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935039', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033891']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753113>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753114 -- <SAMPLE: GSM4753114>\n",
      "metadata: \n",
      " {'title': ['COVID_97_76y_male_ICU'], 'geo_accession': ['GSM4753114'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 76', 'Sex: male', 'icu: yes', 'apacheii: 18', 'charlson score: 9', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 735', 'crp (mg/l): 185', 'ddimer (mg/l_feu): 2.06', 'procalcitonin (ng/ml): 1.06', 'lactate (mmol/l): 1.22', 'fibrinogen: 585', 'sofa: 10', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C97', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935038', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033892'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_97_76y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753114']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 76', 'Sex: male', 'icu: yes', 'apacheii: 18', 'charlson score: 9', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 735', 'crp (mg/l): 185', 'ddimer (mg/l_feu): 2.06', 'procalcitonin (ng/ml): 1.06', 'lactate (mmol/l): 1.22', 'fibrinogen: 585', 'sofa: 10', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C97', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935038', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033892']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753114>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753115 -- <SAMPLE: GSM4753115>\n",
      "metadata: \n",
      " {'title': ['COVID_98_81y_male_NonICU'], 'geo_accession': ['GSM4753115'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 81', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 8', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 15', 'ferritin (ng/ml): 582', 'crp (mg/l): 108.6', 'ddimer (mg/l_feu): 1.54', 'procalcitonin (ng/ml): 0.36', 'lactate (mmol/l): 3.28', 'fibrinogen: 468', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C98', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935037', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033893'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_98_81y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753115']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 81', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 8', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 15', 'ferritin (ng/ml): 582', 'crp (mg/l): 108.6', 'ddimer (mg/l_feu): 1.54', 'procalcitonin (ng/ml): 0.36', 'lactate (mmol/l): 3.28', 'fibrinogen: 468', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C98', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935037', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033893']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753115>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753116 -- <SAMPLE: GSM4753116>\n",
      "metadata: \n",
      " {'title': ['COVID_99_71y_male_ICU'], 'geo_accession': ['GSM4753116'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 71', 'Sex: male', 'icu: yes', 'apacheii: 12', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1231', 'crp (mg/l): 305.5', 'ddimer (mg/l_feu): 3.59', 'procalcitonin (ng/ml): 1.5', 'lactate (mmol/l): 1.18', 'fibrinogen: 949', 'sofa: 4', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C99', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935036', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033894'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_99_71y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753116']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 71', 'Sex: male', 'icu: yes', 'apacheii: 12', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 1231', 'crp (mg/l): 305.5', 'ddimer (mg/l_feu): 3.59', 'procalcitonin (ng/ml): 1.5', 'lactate (mmol/l): 1.18', 'fibrinogen: 949', 'sofa: 4', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C99', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935036', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033894']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753116>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753117 -- <SAMPLE: GSM4753117>\n",
      "metadata: \n",
      " {'title': ['COVID_100_74y_female_NonICU'], 'geo_accession': ['GSM4753117'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 74', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 6', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 35', 'ferritin (ng/ml): 304', 'crp (mg/l): 136.4', 'ddimer (mg/l_feu): 1.67', 'procalcitonin (ng/ml): 0.05', 'lactate (mmol/l): unknown', 'fibrinogen: 583', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C100', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935035', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033895'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_100_74y_female_NonICU']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753117']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 74', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 6', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 35', 'ferritin (ng/ml): 304', 'crp (mg/l): 136.4', 'ddimer (mg/l_feu): 1.67', 'procalcitonin (ng/ml): 0.05', 'lactate (mmol/l): unknown', 'fibrinogen: 583', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C100', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935035', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033895']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753117>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753118 -- <SAMPLE: GSM4753118>\n",
      "metadata: \n",
      " {'title': ['COVID_101_58y_male_ICU'], 'geo_accession': ['GSM4753118'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 58', 'Sex: male', 'icu: yes', 'apacheii: 14', 'charlson score: 3', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 37', 'ferritin (ng/ml): 602', 'crp (mg/l): 100.7', 'ddimer (mg/l_feu): 1.72', 'procalcitonin (ng/ml): 0.2', 'lactate (mmol/l): 2.43', 'fibrinogen: 479', 'sofa: 6', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C101', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935034', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033896'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_101_58y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753118']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 58', 'Sex: male', 'icu: yes', 'apacheii: 14', 'charlson score: 3', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 37', 'ferritin (ng/ml): 602', 'crp (mg/l): 100.7', 'ddimer (mg/l_feu): 1.72', 'procalcitonin (ng/ml): 0.2', 'lactate (mmol/l): 2.43', 'fibrinogen: 479', 'sofa: 6', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C101', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935034', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033896']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753118>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753119 -- <SAMPLE: GSM4753119>\n",
      "metadata: \n",
      " {'title': ['COVID_102_84y_male_NonICU'], 'geo_accession': ['GSM4753119'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 84', 'Sex: male', 'icu: no', 'apacheii: 23', 'charlson score: 7', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: unknown', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C102', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935033', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033897'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_102_84y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753119']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 84', 'Sex: male', 'icu: no', 'apacheii: 23', 'charlson score: 7', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: unknown', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C102', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935033', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033897']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753119>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753120 -- <SAMPLE: GSM4753120>\n",
      "metadata: \n",
      " {'title': ['COVID_103_83y_male_NonICU'], 'geo_accession': ['GSM4753120'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: COVID-19', 'age (years): 83', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 669', 'crp (mg/l): 214.2', 'ddimer (mg/l_feu): 0.75', 'procalcitonin (ng/ml): 0.17', 'lactate (mmol/l): 1.58', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['C103', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935032', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033898'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['COVID_103_83y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753120']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: COVID-19', 'age (years): 83', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 669', 'crp (mg/l): 214.2', 'ddimer (mg/l_feu): 0.75', 'procalcitonin (ng/ml): 0.17', 'lactate (mmol/l): 1.58', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:0080600 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753120>\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['C103', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935032', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033898']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753120>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753121 -- <SAMPLE: GSM4753121>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_01_54y_female_NonICU'], 'geo_accession': ['GSM4753121'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 54', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 35', 'ferritin (ng/ml): 18', 'crp (mg/l): 20', 'ddimer (mg/l_feu): 0.46', 'procalcitonin (ng/ml): 0.5', 'lactate (mmol/l): unknown', 'fibrinogen: 261', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC1', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935031', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033899'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_01_54y_female_NonICU']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753121']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 54', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 35', 'ferritin (ng/ml): 18', 'crp (mg/l): 20', 'ddimer (mg/l_feu): 0.46', 'procalcitonin (ng/ml): 0.5', 'lactate (mmol/l): unknown', 'fibrinogen: 261', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['NC1', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935031', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033899']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753121>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753122 -- <SAMPLE: GSM4753122>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_02_65y_male_ICU'], 'geo_accession': ['GSM4753122'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 65', 'Sex: male', 'icu: yes', 'apacheii: 15', 'charlson score: 6', 'mechanical ventilation: yes', 'ventilator-free days: 18', 'dm: 0', 'hospital-free days post 45 day followup (days): 29', 'ferritin (ng/ml): 297', 'crp (mg/l): 47.8', 'ddimer (mg/l_feu): 17.11', 'procalcitonin (ng/ml): 0.41', 'lactate (mmol/l): 3.68', 'fibrinogen: 267', 'sofa: 12', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC2', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935030', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033900'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_02_65y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753122']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 65', 'Sex: male', 'icu: yes', 'apacheii: 15', 'charlson score: 6', 'mechanical ventilation: yes', 'ventilator-free days: 18', 'dm: 0', 'hospital-free days post 45 day followup (days): 29', 'ferritin (ng/ml): 297', 'crp (mg/l): 47.8', 'ddimer (mg/l_feu): 17.11', 'procalcitonin (ng/ml): 0.41', 'lactate (mmol/l): 3.68', 'fibrinogen: 267', 'sofa: 12', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['NC2', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935030', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033900']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753122>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753123 -- <SAMPLE: GSM4753123>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_03_65y_male_ICU'], 'geo_accession': ['GSM4753123'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 65', 'Sex: male', 'icu: yes', 'apacheii: 24', 'charlson score: 7', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 809', 'crp (mg/l): 226.3', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 1.69', 'lactate (mmol/l): 1.88', 'fibrinogen: unknown', 'sofa: 9', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC3', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935029', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033901'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_03_65y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753123']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 65', 'Sex: male', 'icu: yes', 'apacheii: 24', 'charlson score: 7', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 1', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 809', 'crp (mg/l): 226.3', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 1.69', 'lactate (mmol/l): 1.88', 'fibrinogen: unknown', 'sofa: 9', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['NC3', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935029', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033901']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753123>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753124 -- <SAMPLE: GSM4753124>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_04_90y_male_NonICU'], 'geo_accession': ['GSM4753124'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): >89', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 41', 'ferritin (ng/ml): 58', 'crp (mg/l): 9.3', 'ddimer (mg/l_feu): 34.4', 'procalcitonin (ng/ml): 0.43', 'lactate (mmol/l): 1.51', 'fibrinogen: 258', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC4', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935028', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033902'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_04_90y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753124']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): >89', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 41', 'ferritin (ng/ml): 58', 'crp (mg/l): 9.3', 'ddimer (mg/l_feu): 34.4', 'procalcitonin (ng/ml): 0.43', 'lactate (mmol/l): 1.51', 'fibrinogen: 258', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['NC4', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753124>\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935028', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033902']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753124>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753125 -- <SAMPLE: GSM4753125>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_05_83y_female_NonICU'], 'geo_accession': ['GSM4753125'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 83', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 43', 'ferritin (ng/ml): 129', 'crp (mg/l): 104.3', 'ddimer (mg/l_feu): 1.92', 'procalcitonin (ng/ml): 11.37', 'lactate (mmol/l): 1.27', 'fibrinogen: 253', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC5', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935027', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033903'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_05_83y_female_NonICU']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753125']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 83', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 43', 'ferritin (ng/ml): 129', 'crp (mg/l): 104.3', 'ddimer (mg/l_feu): 1.92', 'procalcitonin (ng/ml): 11.37', 'lactate (mmol/l): 1.27', 'fibrinogen: 253', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['NC5', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935027', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033903']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753125>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753126 -- <SAMPLE: GSM4753126>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_06_75y_female_ICU'], 'geo_accession': ['GSM4753126'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 75', 'Sex: female', 'icu: yes', 'apacheii: 12', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 580', 'crp (mg/l): 200.2', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 14.36', 'lactate (mmol/l): 2', 'fibrinogen: unknown', 'sofa: 2', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC6', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935026', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033904'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_06_75y_female_ICU']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753126']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 75', 'Sex: female', 'icu: yes', 'apacheii: 12', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 580', 'crp (mg/l): 200.2', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 14.36', 'lactate (mmol/l): 2', 'fibrinogen: unknown', 'sofa: 2', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['NC6', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935026', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033904']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753126>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753127 -- <SAMPLE: GSM4753127>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_07_50y_male_ICU'], 'geo_accession': ['GSM4753127'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 50', 'Sex: male', 'icu: yes', 'apacheii: 26', 'charlson score: 7', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 42', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 3.02', 'fibrinogen: unknown', 'sofa: 11', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC7', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935025', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033905'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_07_50y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753127']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 50', 'Sex: male', 'icu: yes', 'apacheii: 26', 'charlson score: 7', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 42', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 3.02', 'fibrinogen: unknown', 'sofa: 11', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['NC7', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935025', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033905']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753127>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753128 -- <SAMPLE: GSM4753128>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_08_53y_female_ICU'], 'geo_accession': ['GSM4753128'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 53', 'Sex: female', 'icu: yes', 'apacheii: 22', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 24', 'dm: 0', 'hospital-free days post 45 day followup (days): 38', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 0.82', 'fibrinogen: 585', 'sofa: 5', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC8', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935024', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033906'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_08_53y_female_ICU']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753128']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 53', 'Sex: female', 'icu: yes', 'apacheii: 22', 'charlson score: 2', 'mechanical ventilation: yes', 'ventilator-free days: 24', 'dm: 0', 'hospital-free days post 45 day followup (days): 38', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 0.82', 'fibrinogen: 585', 'sofa: 5', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['NC8', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935024', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033906']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753128>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753129 -- <SAMPLE: GSM4753129>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_09_49y_female_NonICU'], 'geo_accession': ['GSM4753129'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 49', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 44', 'ferritin (ng/ml): 411', 'crp (mg/l): 7.7', 'ddimer (mg/l_feu): 1.03', 'procalcitonin (ng/ml): 0.07', 'lactate (mmol/l): 0.79', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC9', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935023', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033907'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_09_49y_female_NonICU']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753129']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 49', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 44', 'ferritin (ng/ml): 411', 'crp (mg/l): 7.7', 'ddimer (mg/l_feu): 1.03', 'procalcitonin (ng/ml): 0.07', 'lactate (mmol/l): 0.79', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['NC9', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753129>\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935023', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033907']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753129>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753130 -- <SAMPLE: GSM4753130>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_10_67y_male_ICU'], 'geo_accession': ['GSM4753130'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 67', 'Sex: male', 'icu: yes', 'apacheii: 20', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 25', 'dm: 0', 'hospital-free days post 45 day followup (days): 38', 'ferritin (ng/ml): 79', 'crp (mg/l): 110.2', 'ddimer (mg/l_feu): 0.33', 'procalcitonin (ng/ml): 0.2', 'lactate (mmol/l): 0.61', 'fibrinogen: unknown', 'sofa: 10', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC10', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935022', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033908'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_10_67y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753130']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 67', 'Sex: male', 'icu: yes', 'apacheii: 20', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 25', 'dm: 0', 'hospital-free days post 45 day followup (days): 38', 'ferritin (ng/ml): 79', 'crp (mg/l): 110.2', 'ddimer (mg/l_feu): 0.33', 'procalcitonin (ng/ml): 0.2', 'lactate (mmol/l): 0.61', 'fibrinogen: unknown', 'sofa: 10', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['NC10', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935022', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033908']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753130>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753131 -- <SAMPLE: GSM4753131>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_11_58y_female_NonICU'], 'geo_accession': ['GSM4753131'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 58', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 6', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 41', 'ferritin (ng/ml): 630', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): 0.37', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: 304', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC11', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935021', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033909'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_11_58y_female_NonICU']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753131']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 58', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 6', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 41', 'ferritin (ng/ml): 630', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): 0.37', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: 304', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['NC11', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935021', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033909']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753131>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753132 -- <SAMPLE: GSM4753132>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_12_82y_male_ICU'], 'geo_accession': ['GSM4753132'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 82', 'Sex: male', 'icu: yes', 'apacheii: 12', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 42', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 2.31', 'fibrinogen: unknown', 'sofa: 10', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC12', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935020', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033910'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_12_82y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753132']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 82', 'Sex: male', 'icu: yes', 'apacheii: 12', 'charlson score: 4', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 42', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 2.31', 'fibrinogen: unknown', 'sofa: 10', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['NC12', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935020', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033910']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753132>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753133 -- <SAMPLE: GSM4753133>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_13_65y_male_ICU'], 'geo_accession': ['GSM4753133'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 65', 'Sex: male', 'icu: yes', 'apacheii: 30', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 26', 'dm: 1', 'hospital-free days post 45 day followup (days): 35', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 3.35', 'fibrinogen: unknown', 'sofa: 11', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC13', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935019', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033911'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_13_65y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753133']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 65', 'Sex: male', 'icu: yes', 'apacheii: 30', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 26', 'dm: 1', 'hospital-free days post 45 day followup (days): 35', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 3.35', 'fibrinogen: unknown', 'sofa: 11', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['NC13', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935019', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033911']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753133>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753134 -- <SAMPLE: GSM4753134>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_14_75y_female_ICU'], 'geo_accession': ['GSM4753134'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 75', 'Sex: female', 'icu: yes', 'apacheii: 26', 'charlson score: 5', 'mechanical ventilation: yes', 'ventilator-free days: 24', 'dm: 0', 'hospital-free days post 45 day followup (days): 36', 'ferritin (ng/ml): 289', 'crp (mg/l): 41.5', 'ddimer (mg/l_feu): 2.29', 'procalcitonin (ng/ml): 0.34', 'lactate (mmol/l): 4.22', 'fibrinogen: 255', 'sofa: 11', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC14', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935018', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033912'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_14_75y_female_ICU']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753134']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 75', 'Sex: female', 'icu: yes', 'apacheii: 26', 'charlson score: 5', 'mechanical ventilation: yes', 'ventilator-free days: 24', 'dm: 0', 'hospital-free days post 45 day followup (days): 36', 'ferritin (ng/ml): 289', 'crp (mg/l): 41.5', 'ddimer (mg/l_feu): 2.29', 'procalcitonin (ng/ml): 0.34', 'lactate (mmol/l): 4.22', 'fibrinogen: 255', 'sofa: 11', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['NC14', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935018', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033912']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753134>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753135 -- <SAMPLE: GSM4753135>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_15_83y_unknown_ICU'], 'geo_accession': ['GSM4753135'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 83', 'Sex: unknown', 'icu: yes', 'apacheii: 11', 'charlson score: 8', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 30', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 0.59', 'fibrinogen: unknown', 'sofa: 1', 'associated controlled-vocabulary terms: DOID:4 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC15', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935017', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033913'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_15_83y_unknown_ICU']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753135']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 83', 'Sex: unknown', 'icu: yes', 'apacheii: 11', 'charlson score: 8', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 30', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 0.59', 'fibrinogen: unknown', 'sofa: 1', 'associated controlled-vocabulary terms: DOID:4 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['NC15', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935017', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033913']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753135>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753136 -- <SAMPLE: GSM4753136>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_16_40y_female_ICU'], 'geo_accession': ['GSM4753136'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 40', 'Sex: female', 'icu: yes', 'apacheii: 13', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 29', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): 4.56', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 1.49', 'fibrinogen: 583', 'sofa: 10', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC16', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935016', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033914'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_16_40y_female_ICU']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753136']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 40', 'Sex: female', 'icu: yes', 'apacheii: 13', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 29', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): 4.56', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 1.49', 'fibrinogen: 583', 'sofa: 10', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['NC16', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935016', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033914']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753136>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753137 -- <SAMPLE: GSM4753137>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_17_84y_female_ICU'], 'geo_accession': ['GSM4753137'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 84', 'Sex: female', 'icu: yes', 'apacheii: 11', 'charlson score: 8', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 41', 'ferritin (ng/ml): 105', 'crp (mg/l): 85.9', 'ddimer (mg/l_feu): 0.6', 'procalcitonin (ng/ml): 0.73', 'lactate (mmol/l): 0.95', 'fibrinogen: 539', 'sofa: 3', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC17', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935015', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033915'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_17_84y_female_ICU']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753137']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753137>\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 84', 'Sex: female', 'icu: yes', 'apacheii: 11', 'charlson score: 8', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 41', 'ferritin (ng/ml): 105', 'crp (mg/l): 85.9', 'ddimer (mg/l_feu): 0.6', 'procalcitonin (ng/ml): 0.73', 'lactate (mmol/l): 0.95', 'fibrinogen: 539', 'sofa: 3', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['NC17', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935015', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033915']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753137>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753138 -- <SAMPLE: GSM4753138>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_18_88y_male_ICU'], 'geo_accession': ['GSM4753138'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 88', 'Sex: male', 'icu: yes', 'apacheii: 23', 'charlson score: 8', 'mechanical ventilation: yes', 'ventilator-free days: 25', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 211', 'crp (mg/l): 34', 'ddimer (mg/l_feu): 10.21', 'procalcitonin (ng/ml): 0.06', 'lactate (mmol/l): 1.15', 'fibrinogen: 300', 'sofa: 9', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC18', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935014', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033916'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_18_88y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753138']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 88', 'Sex: male', 'icu: yes', 'apacheii: 23', 'charlson score: 8', 'mechanical ventilation: yes', 'ventilator-free days: 25', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): 211', 'crp (mg/l): 34', 'ddimer (mg/l_feu): 10.21', 'procalcitonin (ng/ml): 0.06', 'lactate (mmol/l): 1.15', 'fibrinogen: 300', 'sofa: 9', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['NC18', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935014', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033916']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753138>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753139 -- <SAMPLE: GSM4753139>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_19_66y_female_ICU'], 'geo_accession': ['GSM4753139'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 66', 'Sex: female', 'icu: yes', 'apacheii: 27', 'charlson score: 4', 'mechanical ventilation: yes', 'ventilator-free days: 21', 'dm: 0', 'hospital-free days post 45 day followup (days): 29', 'ferritin (ng/ml): 155', 'crp (mg/l): 150.2', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 0.35', 'lactate (mmol/l): 2.03', 'fibrinogen: 191', 'sofa: 15', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC19', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935013', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033917'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_19_66y_female_ICU']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753139']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 66', 'Sex: female', 'icu: yes', 'apacheii: 27', 'charlson score: 4', 'mechanical ventilation: yes', 'ventilator-free days: 21', 'dm: 0', 'hospital-free days post 45 day followup (days): 29', 'ferritin (ng/ml): 155', 'crp (mg/l): 150.2', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 0.35', 'lactate (mmol/l): 2.03', 'fibrinogen: 191', 'sofa: 15', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['NC19', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935013', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033917']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753139>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753140 -- <SAMPLE: GSM4753140>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_20_62y_female_ICU'], 'geo_accession': ['GSM4753140'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 62', 'Sex: female', 'icu: yes', 'apacheii: 18', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 0', 'dm: 1', 'hospital-free days post 45 day followup (days): 40', 'ferritin (ng/ml): 46', 'crp (mg/l): 1.8', 'ddimer (mg/l_feu): 3.43', 'procalcitonin (ng/ml): 0.5', 'lactate (mmol/l): 9.91', 'fibrinogen: 264', 'sofa: 3', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC20', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935012', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033918'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_20_62y_female_ICU']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753140']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 62', 'Sex: female', 'icu: yes', 'apacheii: 18', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 0', 'dm: 1', 'hospital-free days post 45 day followup (days): 40', 'ferritin (ng/ml): 46', 'crp (mg/l): 1.8', 'ddimer (mg/l_feu): 3.43', 'procalcitonin (ng/ml): 0.5', 'lactate (mmol/l): 9.91', 'fibrinogen: 264', 'sofa: 3', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 NCIT:C53511 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['NC20', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15935012', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033918']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753140>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753141 -- <SAMPLE: GSM4753141>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_21_71y_male_NonICU'], 'geo_accession': ['GSM4753141'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 71', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 36', 'ferritin (ng/ml): 106', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): 1.46', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 0.73', 'fibrinogen: 377', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC21', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934992', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033919'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_21_71y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753141']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 71', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 5', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 1', 'hospital-free days post 45 day followup (days): 36', 'ferritin (ng/ml): 106', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): 1.46', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 0.73', 'fibrinogen: 377', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " description ['NC21', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934992', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033919']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753141>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753142 -- <SAMPLE: GSM4753142>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_22_63y_male_NonICU'], 'geo_accession': ['GSM4753142'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 63', 'Sex: male', 'icu: no', 'apacheii: 4', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 43', 'ferritin (ng/ml): unknown', 'crp (mg/l): 26.1', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 0.05', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: 1', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC22', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934991', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033920'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_22_63y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753142']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 63', 'Sex: male', 'icu: no', 'apacheii: 4', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 43', 'ferritin (ng/ml): unknown', 'crp (mg/l): 26.1', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): 0.05', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: 1', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['NC22', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934991', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033920']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753142>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753143 -- <SAMPLE: GSM4753143>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_23_42y_female_NonICU'], 'geo_accession': ['GSM4753143'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 42', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 40', 'ferritin (ng/ml): 85', 'crp (mg/l): 41', 'ddimer (mg/l_feu): 1.87', 'procalcitonin (ng/ml): 0.73', 'lactate (mmol/l): 1.99', 'fibrinogen: 635', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC23', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934990', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033921'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_23_42y_female_NonICU']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753143']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 42', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 2', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 40', 'ferritin (ng/ml): 85', 'crp (mg/l): 41', 'ddimer (mg/l_feu): 1.87', 'procalcitonin (ng/ml): 0.73', 'lactate (mmol/l): 1.99', 'fibrinogen: 635', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['NC23', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753143>\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934990', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033921']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753143>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753144 -- <SAMPLE: GSM4753144>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_24_32y_female_NonICU'], 'geo_accession': ['GSM4753144'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 32', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 43', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): 0.19', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 0.87', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC24', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934989', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033922'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_24_32y_female_NonICU']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753144']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 32', 'Sex: female', 'icu: no', 'apacheii: unknown', 'charlson score: 0', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 43', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): 0.19', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 0.87', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003100 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['NC24', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934989', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033922']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753144>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753145 -- <SAMPLE: GSM4753145>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_25_62y_male_NonICU'], 'geo_accession': ['GSM4753145'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 62', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 8', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 44', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 0.99', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC25', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934988', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033923'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_25_62y_male_NonICU']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753145']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 62', 'Sex: male', 'icu: no', 'apacheii: unknown', 'charlson score: 8', 'mechanical ventilation: no', 'ventilator-free days: 28', 'dm: 0', 'hospital-free days post 45 day followup (days): 44', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): 0.99', 'fibrinogen: unknown', 'sofa: unknown', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['NC25', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934988', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033923']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753145>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "GSM4753146 -- <SAMPLE: GSM4753146>\n",
      "metadata: \n",
      " {'title': ['NONCOVID_26_36y_male_ICU'], 'geo_accession': ['GSM4753146'], 'status': ['Public on Aug 29 2020'], 'submission_date': ['Aug 28 2020'], 'last_update_date': ['Aug 29 2020'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['Leukocytes from whole blood'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: non-COVID-19', 'age (years): 36', 'Sex: male', 'icu: yes', 'apacheii: 40', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: 15', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells'], 'treatment_protocol_ch1': ['N/A'], 'growth_protocol_ch1': ['N/A'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.'], 'description': ['NC26', 'genes.ec.tsv', 'genes.tpm.tsv'], 'data_processing': ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)'], 'platform_id': ['GPL24676'], 'contact_name': ['Adolfo,Ariel,Jaitovich'], 'contact_email': ['jaitova@amc.edu'], 'contact_department': ['Molecular and Cellular Physiology'], 'contact_institute': ['Albany Medical Center'], 'contact_address': ['47 New Scotland Av'], 'contact_city': ['Albany'], 'contact_state': ['NY'], 'contact_zip/postal_code': ['12208'], 'contact_country': ['USA'], 'instrument_model': ['Illumina NovaSeq 6000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934987', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033924'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE157103'], 'data_row_count': ['0']}\n",
      "gsm.metadata.items \n",
      " title ['NONCOVID_26_36y_male_ICU']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " geo_accession ['GSM4753146']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " status ['Public on Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " submission_date ['Aug 28 2020']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " last_update_date ['Aug 29 2020']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " type ['SRA']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " channel_count ['1']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " source_name_ch1 ['Leukocytes from whole blood']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " organism_ch1 ['Homo sapiens']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " taxid_ch1 ['9606']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " characteristics_ch1 ['disease state: non-COVID-19', 'age (years): 36', 'Sex: male', 'icu: yes', 'apacheii: 40', 'charlson score: 3', 'mechanical ventilation: yes', 'ventilator-free days: 0', 'dm: 0', 'hospital-free days post 45 day followup (days): 0', 'ferritin (ng/ml): unknown', 'crp (mg/l): unknown', 'ddimer (mg/l_feu): unknown', 'procalcitonin (ng/ml): unknown', 'lactate (mmol/l): unknown', 'fibrinogen: unknown', 'sofa: 15', 'associated controlled-vocabulary terms: DOID:4 UBERON:0003101 NCIT:C53511 NCIT:C70909 CL:0000738 NCIT:C12529', 'cell type: leukocyte', 'tissue of origin: whole blood', 'sample type: primary cells']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " treatment_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " growth_protocol_ch1 ['N/A']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " molecule_ch1 ['total RNA']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " extract_protocol_ch1 ['10mL of whole blood collected from patients was filtered and RNA was isolated using LeukoLOCK Total RNA Isolation System.', 'TruSeq Stranded mRNA Library prep. Libraries were standardized to 2nM. Paired-end 2x50bp sequencing, using standard SBS chemistry (v3) on an Illumina NovaSeq6000 sequencer.']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " description ['NC26', 'genes.ec.tsv', 'genes.tpm.tsv']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_processing ['Basecalling via the standard Illumina Pipeline, version 1.8.2.', 'Trimming/Filtering for adapters and low quality via custom algorithm', 'Expression estimates via RSEM v1.3.0 under Python 2.7.15, using bowtie2-2.3.4.1. Executed command: rsem-calculate-expression --no-qualities --bowtie2 --bowtie2-k 200 --bowtie2-sensitivity-level sensitive --seed-length 20 --keep-intermediate-files --no-bam-output --paired-end [r1.fa.gz] [r2.fa.gz] [hg38 bowtie reference]', 'Genome_build: hg38', 'Supplementary_files_format_and_content: tab-delimited expression estimates, including non-normalized Expected Counts (EC), and normalized Transcripts Per Million (TPM)']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " platform_id ['GPL24676']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_email ['jaitova@amc.edu']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_department ['Molecular and Cellular Physiology']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_institute ['Albany Medical Center']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_address ['47 New Scotland Av']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_city ['Albany']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_state ['NY']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_zip/postal_code ['12208']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " contact_country ['USA']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " instrument_model ['Illumina NovaSeq 6000']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_selection ['cDNA']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_source ['transcriptomic']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " library_strategy ['RNA-Seq']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " relation ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN15934987', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX9033924']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " supplementary_file_1 ['NONE']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " series_id ['GSE157103']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n",
      "gsm.metadata.items \n",
      " data_row_count ['0']\n",
      "Table data: <SAMPLE: GSM4753146>\n",
      "Empty DataFrame\n",
      "Columns: [description]\n",
      "Index: []\n"
     ]
    }
   ],
   "source": [
    "gse157103.gsms.items\n",
    "\n",
    "for gsm_name, gsm in gse157103.gsms.items():\n",
    "    print(gsm_name,\"--\", gsm)\n",
    "    print(\"metadata: \\n\",gsm.metadata)\n",
    "    for key, value in gsm.metadata.items():\n",
    "        print(\"gsm.metadata.items \\n\",key, value)\n",
    "        print(\"Table data:\",gsm)\n",
    "        print(gsm.columns)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "k, v  title ['Large-scale Multi-omic Analysis of COVID-19 Severity']\n",
      "k, v  geo_accession ['GSE157103']\n",
      "k, v  status ['Public on Aug 29 2020']\n",
      "k, v  submission_date ['Aug 28 2020']\n",
      "k, v  last_update_date ['Nov 30 2020']\n",
      "k, v  pubmed_id ['33096026']\n",
      "k, v  summary ['Using RNA-seq and high-resolution mass spectrometry we performed a comprehensive systems analysis on 128 plasma and leukocyte samples from hospitalized patients with or without COVID-19 (n=102 and 26 respectively) and with differing degrees of disease severity. We generated abundance measurements for over 17,000 transcripts, proteins, metabolites, and lipids and compiled them with clinical data into a curated relational database. This resource offers the unique opportunity to perform systems analysis and cross-ome correlations to both molecules and patient outcomes. In total 219 molecular features were mapped with high significance to COVID-19 status and severity, including those involved in processes such as complement system activation, dysregulated lipid transport, and B cell activation. In one example, we detected a trio of covarying molecules â€“ citrate, plasmenyl-phosphatidylcholines, and gelsolin (GSN) â€“ that offer both pathophysiological insight and potential novel therapeutic targets. Further, our data revealed in some cases, and supported in others, that several biological processes were dysregulated in COVID-19 patients including vessel damage, platelet activation and degranulation, blood coagulation, and acute phase response. For example, we observed that the coagulation-related protein, cellular fibronectin (cFN), was highly increased within COVID-19 patients and provide new evidence that the upregulated proteoform stems from endothelial cells, consistent with endothelial injury as a major activator of the coagulation cascade. The abundance of prothrombin, which is cleaved to form thrombin during clotting, was significantly reduced and correlated with severity and might help to explain the hyper coagulative environment of SARS-CoV-2 infection. From transcriptomic analysis of leukocytes, we concluded that COVID-19 patients with acute respiratory distress syndrome (ARDS) demonstrated a phenotype that overlapped with, but was distinct from, that found in patients with non-COVID-19-ARDS. To aid in the global efforts toward elucidation of disease pathophysiology and therapeutic development, we created a web-based tool with interactive visualizations allowing for easy navigation of this systems-level compendium of biomolecule abundance in relation to COVID-19 status and severity. Finally, we leveraged these multi-omic data to predict COVID-19 patient outcomes with machine learning, which highlighted the predictive power of these expansive molecular measurements beyond the standardized clinical estimate of 10-year survival Charlson score.']\n",
      "k, v  overall_design ['126 samples were analyzed in total with 100 COVID-19 patients and 26 non-COVID-19']\n",
      "k, v  type ['Expression profiling by high throughput sequencing']\n",
      "k, v  contributor ['Ariel,,Jaitovitch', 'Joseph,,Balnis', 'Scott,A,Swanson']\n",
      "k, v  sample_id ['GSM4753021', 'GSM4753022', 'GSM4753023', 'GSM4753024', 'GSM4753025', 'GSM4753026', 'GSM4753027', 'GSM4753028', 'GSM4753029', 'GSM4753030', 'GSM4753031', 'GSM4753032', 'GSM4753033', 'GSM4753034', 'GSM4753035', 'GSM4753036', 'GSM4753037', 'GSM4753038', 'GSM4753039', 'GSM4753040', 'GSM4753041', 'GSM4753042', 'GSM4753043', 'GSM4753044', 'GSM4753045', 'GSM4753046', 'GSM4753047', 'GSM4753048', 'GSM4753049', 'GSM4753050', 'GSM4753051', 'GSM4753052', 'GSM4753053', 'GSM4753054', 'GSM4753055', 'GSM4753056', 'GSM4753057', 'GSM4753058', 'GSM4753059', 'GSM4753060', 'GSM4753061', 'GSM4753062', 'GSM4753063', 'GSM4753064', 'GSM4753065', 'GSM4753066', 'GSM4753067', 'GSM4753068', 'GSM4753069', 'GSM4753070', 'GSM4753071', 'GSM4753072', 'GSM4753073', 'GSM4753074', 'GSM4753075', 'GSM4753076', 'GSM4753077', 'GSM4753078', 'GSM4753079', 'GSM4753080', 'GSM4753081', 'GSM4753082', 'GSM4753083', 'GSM4753084', 'GSM4753085', 'GSM4753086', 'GSM4753087', 'GSM4753088', 'GSM4753089', 'GSM4753090', 'GSM4753091', 'GSM4753092', 'GSM4753093', 'GSM4753094', 'GSM4753095', 'GSM4753096', 'GSM4753097', 'GSM4753098', 'GSM4753099', 'GSM4753100', 'GSM4753101', 'GSM4753102', 'GSM4753103', 'GSM4753104', 'GSM4753105', 'GSM4753106', 'GSM4753107', 'GSM4753108', 'GSM4753109', 'GSM4753110', 'GSM4753111', 'GSM4753112', 'GSM4753113', 'GSM4753114', 'GSM4753115', 'GSM4753116', 'GSM4753117', 'GSM4753118', 'GSM4753119', 'GSM4753120', 'GSM4753121', 'GSM4753122', 'GSM4753123', 'GSM4753124', 'GSM4753125', 'GSM4753126', 'GSM4753127', 'GSM4753128', 'GSM4753129', 'GSM4753130', 'GSM4753131', 'GSM4753132', 'GSM4753133', 'GSM4753134', 'GSM4753135', 'GSM4753136', 'GSM4753137', 'GSM4753138', 'GSM4753139', 'GSM4753140', 'GSM4753141', 'GSM4753142', 'GSM4753143', 'GSM4753144', 'GSM4753145', 'GSM4753146']\n",
      "k, v  contact_name ['Adolfo,Ariel,Jaitovich']\n",
      "k, v  contact_email ['jaitova@amc.edu']\n",
      "k, v  contact_department ['Molecular and Cellular Physiology']\n",
      "k, v  contact_institute ['Albany Medical Center']\n",
      "k, v  contact_address ['47 New Scotland Av']\n",
      "k, v  contact_city ['Albany']\n",
      "k, v  contact_state ['NY']\n",
      "k, v  contact_zip/postal_code ['12208']\n",
      "k, v  contact_country ['USA']\n",
      "k, v  supplementary_file ['ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE157nnn/GSE157103/suppl/GSE157103_genes.ec.tsv.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE157nnn/GSE157103/suppl/GSE157103_genes.tpm.tsv.gz']\n",
      "k, v  platform_id ['GPL24676']\n",
      "k, v  platform_taxid ['9606']\n",
      "k, v  sample_taxid ['9606']\n",
      "k, v  relation ['BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA660067', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP279280']\n"
     ]
    }
   ],
   "source": [
    "for k,v in gse157103.metadata.items():\n",
    "    print(\"k, v \",k,v)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
